%\documentclass[english,12pt]{article}
\documentclass[doublespacing]{bmcart_dla}
\usepackage{fontspec}
\setmainfont{FreeSans}
\setsansfont{FreeSans}
\setmonofont{FreeMono}
\setmathrm{FreeSans}
\setmathsf{FreeSans}
\setmathtt{FreeSans}
%\usepackage[letterpaper,left=1in,right=1in,top=1in,bottom=1in,headheight=14pt]{geometry}
\usepackage[letterpaper]{geometry}
\usepackage{url}
% \usepackage{fancyhdr}
\usepackage{graphicx}
% \usepackage[bf]{caption}
\usepackage{rotating}
\usepackage{wrapfig}
\usepackage{booktabs}
\usepackage{float}
\usepackage{multirow}
\usepackage{setspace}
\usepackage{wrapfig}
% \usepackage{txfonts}
%\usepackage[plain]{fancyref}
\usepackage{acronym}
\usepackage{array}
\usepackage{dcolumn}
\usepackage{adjustbox}
\usepackage{longtable}
\usepackage{pdflscape}
\usepackage{amsmath}
\usepackage{sfmath}
\usepackage[backend=biber,natbib=true,hyperref=true,citestyle=numeric-comp,style=nature,autocite=inline]{biblatex}
\addbibresource{references.bib}
% \usepackage[noblocks,auth-sc]{authblk}
\usepackage[hyperfigures,bookmarks,colorlinks,linkcolor=black,urlcolor=black]{hyperref}
%\usepackage[hyperfigures,bookmarks,colorlinks,citecolor=black,filecolor=black,linkcolor=black,urlcolor=black]{hyperref}
% Floats at end for journals
% \usepackage[nolists,figuresfirst,nomarkers,heads,nofighead,notabhead]{endfloat}
% \AtBeginFigures{\newpage \section*{Figure Legends}\renewcommand{\efloatseparator}{\hbox{}}}
% \AtBeginTables{\newpage \section*{Tables}\renewcommand{\efloatseparator}{\mbox{}}}
\usepackage[nomargin,inline,draft]{fixme}
\newcommand{\DLA}[1]{\textcolor{red}{\fxnote{DLA: #1}}}
\newcommand{\DEW}[1]{\textcolor{purple}{\fxnote{DEW: #1}}}
% \usepackage[x11names,svgnames]{xcolor}
% \usepackage{texshade}
% \usepackage{tikz}
\usepackage{nameref}
\usepackage{xspace}
% this package will stretch the font slightly if necessary.
\usepackage{microtype}
\usepackage{zref-xr,zref-user}
\zxrsetup{toltxlabel}
\zexternaldocument*{supplemental_material}
\usepackage[capitalize]{cleveref}
% \newenvironment{narrow}[2]{%
%   \begin{list}{}{%
%       \setlength{\topsep}{0pt}%
%       \setlength{\leftmargin}{#1}%
%       \setlength{\rightmargin}{#2}%
%       \setlength{\listparindent}{\parindent}%
%       \setlength{\itemindent}{\parindent}%
%       \setlength{\parsep}{\parskip}}%
%   \item[]}{\end{list}}
% \def\newblock{\hskip}
% \newenvironment{paperquote}{%
%   \begin{quote}%
%      \it
%   }%
%   {\end{quote}}
% \renewcommand{\textfraction}{0.15}
% \renewcommand{\topfraction}{0.85}
% \renewcommand{\bottomfraction}{0.65}
% \renewcommand{\floatpagefraction}{0.60}
%\renewcommand{\baselinestretch}{1.8}
% \newenvironment{enumerate*}%
%   {\begin{enumerate}%
%     \setlength{\itemsep}{0pt}%
%     \setlength{\parskip}{0pt}}%
%   {\end{enumerate}}
% \newenvironment{itemize*}%
%   {\begin{itemize}%
%     \setlength{\itemsep}{0pt}%
%     \setlength{\parskip}{0pt}}%
%   {\end{itemize}}
% \setcounter{tocdepth}{3}
% \oddsidemargin 0.0in 
% \textwidth 6in
% \raggedbottom
% \clubpenalty = 10000
% \widowpenalty = 10000
% set a large emergency stretch to avoid issues with overfull boxes
% \setlength{\emergencystretch}{20em}
% make the afill smaller
% \renewcommand\Affilfont{\itshape\small}
\renewcommand\cite[1]{\autocite{#1}}
% \pagestyle{fancy}

% species names
\newcommand{\smonkey}[0]{\textit{A. fusciceps}\xspace}
\newcommand{\cow}[0]{\textit{B. taurus}\xspace}
\newcommand{\dog}[0]{\textit{C. familiaris}\xspace}
\newcommand{\armadillo}[0]{\textit{D. novemcinctus}\xspace}
\newcommand{\horse}[0]{\textit{E. caballus}\xspace}
\newcommand{\elephant}[0]{\textit{L. africana}\xspace}
\newcommand{\human}[0]{\textit{H. sapiens}\xspace}
\newcommand{\opossum}[0]{\textit{M. domestica}\xspace}
\newcommand{\mouse}[0]{\textit{M. musculus}\xspace}
\newcommand{\galili}[0]{\textit{N. galili}\xspace}
\newcommand{\sheep}[0]{\textit{O. aries}\xspace}
\newcommand{\bonobo}[0]{\textit{P. paniscus}\xspace}
\newcommand{\carmeli}[0]{\textit{S. carmeli}\xspace}
\newcommand{\pig}[0]{\textit{S. scrofa}\xspace}

% top 50 gene table explanation. Note that if you modify this, you
% should probably also modify the human table explanation, which is
% slightly different.
\newcommand{\tableexplanation}[2]{The top 50 genes by \ac{FPKM} in #1
  which are not annotated as Mt\_rRNA, Mt\_tRNA, rRNA or whose \human
  ortholog is a housekeeping gene. Gene symbol is the gene symbol or
  Ensembl gene id. Human Symbol is the 1:1 human ortholog or the best
  human alignment when enclosed in parenthesis. Human Name is the HUGO
  name corresponding to the human symbol. A complete list of all genes
  is in \cref{#2}.}

\newcommand{\genetreeexplanation}[1]{Gene tree and heatmap of
  expression of genes related to #1. The gene tree was obtained from
  Ensembl~\cite{Cunningham.ea2015:Ensembl2015} and genes from species
  not included in this study or having expression by \ac{FPKM} less
  than 1 were excluded from the plotted tree. Heatmap cells in grey
  are from species where there was no expression of the relevant
  transcript (or a species where that transcript does not apply).}



\begin{document}
\raggedright


<<load.libraries,echo=FALSE,results="hide",warning=FALSE,message=FALSE,error=FALSE,cache=FALSE>>=
opts_chunk$set(dev="cairo_pdf",out.width="\\textwidth",out.height="0.7\\textheight",out.extra="keepaspectratio")
opts_chunk$set(cache=TRUE, autodep=TRUE)
options(device = function(file, width = 8, height = 7, ...) {
  cairo_pdf(tempfile(), width = width, height = height, ...)
})
options(scipen = -2, digits = 0)
library("data.table")
library("ggplot2")
library("reshape2")
library("xtable")
library("grid")
library("gridExtra")
library("ape")
library("ggtree")
library("topGO")
library("org.Hs.eg.db")
library("scales")

@ 

<<source.to.latex,echo=FALSE,results="hide",cache.extra=tools::md5sum(paste0("../common_r_code/",c("to_latex.R","plot_tree_matrix.R","utility_functions.R")))>>=
for (a in paste0("../common_r_code/",c("to_latex.R","plot_tree_matrix.R","utility_functions.R"))) {
    source(a)
}
@ 

<<load.data,echo=FALSE,results="hide",cache.extra=tools::md5sum(paste0("../data/",c("combined_fpkm","gene_trees","housekeeping_genes_superset","placenta_core_transcriptome","placenta_go_results_bp","placenta_go_results_mf","hugo_nomenclature","gene_annotation")))>>=
for (a in paste0("../data/",c("combined_fpkm","gene_trees",
                              "housekeeping_genes_superset",
                              "placenta_core_transcriptome",
                              "placenta_go_results_bp",
                              "placenta_go_results_mf",
                              "hugo_nomenclature",
                              "gene_annotation"
                              ))) {
    load(a)
}
combined.fpkm[,species:=factor(species,levels=names(species.names))]
setkey(genes.to.tree,"gene_id")
star.logs[,value:=gsub("%$","",value)]
star.logs.wide <- 
    data.table(dcast(star.logs,species+file~field))
setnames(star.logs.wide,
         "Average input read length",
         "input.read.length")
setnames(star.logs.wide,
         "Number of input reads",
         "input.reads")
setnames(star.logs.wide,
         "Uniquely mapped reads number",
         "unique.map.reads")
star.logs.wide[,`% bad reads`:=(as.numeric(`% of reads mapped to too many loci`)+
                            as.numeric(`% of reads unmapped: other`)+
                            as.numeric(`% of reads unmapped: too many mismatches`)+
                            as.numeric(`% of reads unmapped: too short`))]
star.logs.wide[,`% good reads`:=100-`% bad reads`]
setkey(hugo.nomenclature,"Approved Symbol")
@ 

<<load.go.results,echo=FALSE,results="hide",cache.extra=tools::md5sum("../go_annotation/go_tissue_analysis")>>=
load("../go_annotation/go_tissue_analysis")

@ 

<<plot.defaults,echo=FALSE,results="hide">>=
grad.def <- list(low = "#132B43", high = "#56B1F7", space = "Lab", na.value = "gray50")

@ 

% Acronyms
\input{acronyms}

\begin{frontmatter}
\begin{fmbox}
\dochead{Research}
\title{The evolution of gene expression in the term placenta of viviparous mammals}%
%
\author[addressref={aff1},email={don@donarmstrong.com}]{\inits{DL}\fnm{Don L} \snm{Armstrong}}
\author[addressref={aff2}]{\inits{MR}\fnm{Michael R.} \snm{McGowen}}
\author[addressref={aff1},email={aweckle@illinois.edu}]{\inits{AL} \fnm{Amy} \snm{Weckle}}
\author[addressref={aff3}]{\inits{J}\fnm{Jason} \snm{Caravas}}
\author[addressref={aff4}]{\inits{D}\fnm{Dalen} \snm{Agnew}}
\author[addressref={aff5}]{\inits{K}\fnm{Kurt} \snm{Benirschke}}
\author[addressref={aff6}]{\inits{S}\fnm{Sue} \snm{Savage-Rumbaugh}}
\author[addressref={aff7,aff8}]{\inits{CJ}\fnm{Chong J} \snm{Kim}}
\author[addressref={aff9}]{\inits{GP}\fnm{GÃ¼nter P.} \snm{Wagner}}
\author[addressref={aff7}]{\inits{R}\fnm{Roberto} \snm{Romero}}
\author[addressref={aff1,aff6,aff10},corref={aff1},email={wildmand@illinois.edu}]{\inits{DE}\fnm{Derek E.} \snm{Wildman}}
% These are all wrong, but we'll change them later.
\address[id=aff1]{Institute for Genomic Biology, University of Illinois at Urbana-Champaign, Urbana, IL, USA}
\address[id=aff2]{School of Biological and Chemical Sciences, Queen Mary, University of London, London, UK}
\address[id=aff3]{Center for Molecular Medicine and Genetics, Wayne State University School of Medicine, Detroit, MI, USA}
\address[id=aff4]{Diagnostic Center for Population and Animal Health, College of Veterinary Medicine, Michigan State University, East Lansing, MI, USA}
\address[id=aff5]{Department of Pathology, University of California San Diego, La Jolla, CA, USA}
\address[id=aff6]{Great Ape Trust/Bonobo Hope Sanctuary, Des Moines, IA, USA}
\address[id=aff7]{Perinatology Research Branch, Eunice Kennedy Shriver National Institute of Child Health and Human Development (NICHD), National Institutes of Health (NIH), Detroit, MI, USA}
\address[id=aff8]{Department of Pathology, Wayne State University School of Medicine, Detroit, MI, USA}
\address[id=aff9]{Department of Ecology and Evolutionary Biology and Yale Systems Biology Institute, Yale University, New Haven, CT, USA}
\address[id=aff10]{Department of Molecular and Integrative Physiology, University of Illinois at Urbana-Champaign, Urbana, IL, USA}
%\address[id=affcor]{Derek Wildman <wildmand@illinois.edu>}

\begin{artnotes}
\end{artnotes}

\end{fmbox}
% \twocolumn

\begin{abstractbox}
\begin{abstract}
% \section{Abstract}
  \parttitle{Background} The placenta is arguably the most
  anatomically variable organ in mammals. Understanding gene
  expression patterns among placentas from multiple species can
  illuminate the molecular basis of placenta function as well as
  potentially enrich our knowledge of the biology of obstetrical
  syndromes.

  \parttitle{Results} We profiled the term placenta by using Illumina
  RNA-Seq technology to sequence the transcriptomes of
  \Sexpr{star.logs[,length(unique(species))]} species of therian (i.e.
  marsupials and âplacentalâ) mammals.


  \parttitle{Conclusions}


% We identified significant shifts in gene expression along specific
% evolutionary lineages using a model of character evolution. Genes that
% showed significant elevation in expression along the human lineage
% ($\mathrm{FDR}\le 0.01$) were enriched in GO categories such as
% angiogenesis, tube morphogenesis, cell migration, and regulation of
% inflammatory response. Three of these genes (KISS1, ADAM12, GPC3)
% showed exceptionally high expression in humans; these genes are
% implicated in pre-eclampsia and may indicate that the risk of
% pre-eclampsia recently emerged on the human lineage.

\textbf{Keywords:} mammal, gene expression, evolution, RNA-seq, placenta

\end{abstract}
\end{abstractbox}
\end{frontmatter}

\section{Background}

The main role of the placenta in viviparous mammals is the transfer of
nutrients, oxygen, and waste between mother and offspring. Although
these functions are conserved among mammals, the placenta is perhaps
the most variable organ in terms of structure and form
\cite{Mossman.1937:comparativemorphogenesisofthe}. Species differ in
the shape of attachment (i.e., discoid, diffuse, cotyledonary,
zonary), degree of invasiveness of fetal tissues into maternal tissues
(i.e. epitheliochorial, endotheliochorial, hemochorial), and degree of
interdigitation of maternal-fetal tissue (i.e., villous, folded,
trabecular, lamellar, labrynthine)
\cite{Mossman.1987:vertebratefetalmembranes}. In addition, therian
mammals differ drastically in the number and size of their offspring,
as well as the length of gestation, traits often associated with
specific aspects of placental morphology
\cite{Capellini.ea2011:Placentationandmaternalinvestment}
(\cref{tab:gestational_periods}).

The placenta arose at some point before the origin of therian mammals,
a group that includes both marsupials and eutherian or âplacentalâ
mammals. The term âplacental mammalsâ is misleading, as marsupials
also possess a placenta, although its direct apposition to maternal
tissues lasts for a proportionally shorter time during gestation
\cite{Renfree2010:ReviewMarsupialsplacentalmammals}. Marsupials also
differ from eutherian mammals in that most possess only a
choriovitelline placenta, produced from the fusion of the chorion and
yolk sac, during most of gestation. Although some eutherian mammals
may possess a choriovitelline placenta at some point in development,
the major site of nutrient exchange in placentals occurs via the
chorioallantoic placenta, produced from the fusion of the chorion and
allantois \cite{Mossman.1987:vertebratefetalmembranes}. It is unclear
whether the first placenta was more like that of marsupials or
eutherians; however, using phylogenetic methods, the ancestral
eutherian was reconstructed as possessing a relatively invasive
hemochorial placenta, in which the maternal and fetal blood supply is
separated by one tissue layer, as in humans and mouse
\cite{Elliot.ea2009:Phylogeneticevidenceforearly,Wildman.ea2006:Evolutionofmammalianplacenta}.

Although the morphology of the mammalian placenta has been well
characterized in a number of species
\cite{Mossman.1987:vertebratefetalmembranes}, the diversity of the
molecular environment is only beginning to be understood from a
genetic perspective \cite{Carter2012:Evolutionofplacentalfunction}.
Studies of certain model species (i.e., \human, \mouse, \cow) have
revealed a stunning divergence in the genetics of the placenta,
especially in later stages of pregnancy
\cite{Carter2012:Evolutionofplacentalfunction,Hou.ea2012:Elephanttranscriptomeprovidesinsights,Knox.ea2008:Genomicevolutionofplacenta,Su.ea2002:Large-scaleanalysisofhuman}.
For example, each lineage has evolved expansive placenta-specific gene
families with expression patterns that can vary in space and time
\cite{Green.ea2000:Pregnancy-associatedbovineandovine,Simmons.ea2008:Spatialandtemporalexpression}.
These include independent expansions of the prolactins in \mouse,
\carmeli, \galili, and \cow, \acp{PSG} in \human and \mouse, \acp{PAG}
in \cow, genes related to the thymus-specific serine protease
\textit{PRSS16} in \elephant, and the expansion of the growth hormone
and chorionic gonadotropin beta genes in hominoid primates
\cite{Haig2008:Placentalgrowthhormone-relatedproteins,Rawn.ea2008:evolutionregulationandfunction}.
Expression patterns of specific genes also can vary widely between
species. For example, the expression of orthologous genes in the
placenta have been found to vary substantially between \human and
\mouse \cite{Su.ea2002:Large-scaleanalysisofhuman}, and the
recruitment of \textit{SIGLEC6} to placental expression in \human
differs from both \textit{P. troglodytes} and \mouse
\cite{Linden.ea2007:Human-specificexpressionofSiglec-6}.

One explanation for the divergent paths of the placental interface is
the distinct evolutionary interplay between maternal and fetal
strategies during pregnancy \cite{Haig1993:Geneticconflictsinhuman}.
The maternal strategy is to requisition an adequate degree of
nutrients to the fetus without risking the health of the mother, while
the fetal strategy via paternal genes is to extract as many resources
from the mother as possible. Therefore, an interplay between these
extremes can set up an intricate âarms raceâ, involving the imprinting
of parental alleles, expansion of gene families, and enlistment of
existing genes to expression at the maternal-fetal interface
\cite{Haig1993:Geneticconflictsinhuman}.

One symptom of this maternal-fetal conflict is its potential affect on
\human health
\cite{Hollegaard.ea2013:Parent-offspringconflictandpersistence}. In
addition to having an invasive hemochorial placenta, \human possess a
deeply invasive trophoblast layer that can penetrate the myometrium of
the uterus \cite{Carter.ea2011:Evolutionofinvasiveplacentation}.
Defects in placental growth are known to be associated with the
devastating condition of pre-eclampsia, which can cause hypertension,
proteinuria, and has the potential to lead to death of the mother
and/or infant \cite{Chaiworapongsa.ea2014:Pre-eclampsiapart1current}.
Although cases have been documented in \textit{P. troglodytes}
\cite{Stout.ea1969:Glomerularcapillaryendothelialswelling}, and
\textit{Gorilla gorilla}
\cite{Thornton.ea1992:Convulsionsinpregnancyin}, pre-eclampsia has a
far higher incidence in \human pregnancies with as much as 4\% of
pregnancies being affected in the United States
\cite{Chaiworapongsa.ea2014:Pre-eclampsiapart1current}. \textit{P.
  troglodytes} and \textit{G. gorilla} also have deeply invasive
trophoblasts, although the progression of invasion in great apes may
be slower than \human
\cite{Carter.ea2011:Evolutionofinvasiveplacentation}; however,
large-scale expression studies of great ape placentas have not been
conducted. The increasing affordability of next-generation sequencing
technology has enabled the large-scale study of specific tissues from
a wide array of species to address broader phylogenetic questions in
the evolution of gene expression
\cite{Brawand.ea2011:evolutionofgeneexpression}. Here we use RNA-Seq
technology to sequence and quantify the mRNA environment of mammalian
placentas at term, including that of the \human, \bonobo (the sister
species to \textit{P. troglodytes}), \smonkey, \mouse, \galili,
\carmeli, \dog, \cow, \elephant, \armadillo, and \opossum. These
species represent three major branches of eutherian mammals
(Euarchontoglires, Laurasiatheria, Afrotheria) as well as marsupials.
These placentas also represent a wide array of placental morphologies
as well as traits such as number and size of offspring and length of
gestation (\cref{tab:gestational_periods}). We use this data, along with
existing RNA-Seq data from \mouse
\cite{Wang.ea2011:surveyfornovelimprinted}, \human
\cite{Hughes.ea2015:Evaluatingintra-andinter-individual}, \horse
\cite{Coleman.ea2010:Structuralannotationofequine}, \sheep
\cite{Archibald.ea2010:sheepgenomereferencesequence}, and \pig
\cite{Ahanda.ea2012:Predictionofaltered3-} to address the major
differences between expression in marsupial and eutherian mammals at
term, identify the evolution of expression in important placental
genes such as hormones and galectins, identify novel lineage-specific
expression patterns, and quantify \human specific differences in gene
expression that may be related to obstetrical disorders such as
pre-eclampsia.

\begin{table}
  \begin{tabular}{c c c c}
    \toprule
    Species                       & Gestation (days) & Litter Size  & Source \\
    \midrule
    \textit{Ateles fusciceps}     & 224 & 1 & \citet{Ernest2003:lifehistorycharacteristicsofplacental}\\
    \textit{Bos taurus}           & 278 & 1 & \citet{Kiltie1982:intraspecificvariationmammaliangestation} \\
    \textit{Canis familiaris}     & 61 & 4 & \citet{Kiltie1982:intraspecificvariationmammaliangestation} \\
    \textit{Dasypus novemcinctus} & 150 & 4 & \citet{Smith1984:amazingarmadillo} \\
    \textit{Equus caballus}       & 336 & 1 & \citet{Kiltie1982:intraspecificvariationmammaliangestation}\\
    \textit{Loxodonta africana}   & 644 & 1 & \citet{Ernest2003:lifehistorycharacteristicsofplacental}\\
    \textit{Homo sapiens}         & 270 & 1.1 & \citet{Kiltie1982:intraspecificvariationmammaliangestation} \\
    \textit{Monodelphis domestica}& 15 & 7.1 & \citet{Harder.ea1993:Gestationandplacentationin} \\
    \textit{Mus musculus}         & 20 & 6.1 & \citet{Kiltie1982:intraspecificvariationmammaliangestation} \\
    \textit{Nannospalax galili}   & 28-42 & 4 & \citet{Nevo1997:mosiacevolutionofsubterranianmammals} \\
    \textit{Ovis aries}           & 151 & 1.35 & \citet{Kiltie1982:intraspecificvariationmammaliangestation} \\
    \textit{Pan paniscus}         & 223 & 1 & \citet{Ernest2003:lifehistorycharacteristicsofplacental} \\
    \textit{Spalax carmeli}       & 28-42 & 4 & \citet{Nevo1997:mosiacevolutionofsubterranianmammals}\\
    \textit{Sus scrofa}           & 114 & 5.8 & \citet{Kiltie1982:intraspecificvariationmammaliangestation} \\
    \bottomrule
  \end{tabular}
  \caption{Gestational periods in days and average liter sizes for studied species.}
  \label{tab:gestational_periods}
\end{table}

\section{Methods}

\subsection{Collection of Placental Tissue and RNA Extraction}

Fetal tissue was collected from the placenta of eight mammalian
species (\human [human], \bonobo [bonobo], \dog [domestic dog], \cow
[domestic cow], \elephant [African savanna elephant], \opossum [gray
short-tailed opossum], \galili [blind mole rat from Upper Galilee],
\carmeli [blind mole rat from northern Israel]). Information on
specimen ID, place and time of collection, and collector is shown for
each sample in \DLA{Supplemental Table 1}. In the case of \bonobo,
\cow, and \elephant, the placenta was collected at term upon expulsion
from the mother following delivery. Placental tissue from \human was
collected upon delivery via a caesarian section. Near-term placental
tissue from \dog was collected upon the death of the mother from a car
accident. Placental tissue from \opossum was sampled at fetal stage
32, roughly one day prior to birth. Only the fetal portion of the
placenta was sampled at the maternal-fetal interface (i.e. villous
portion for \human, \bonobo, \dog, and \elephant; cotyledon for \cow;
diffuse placenta for \opossum). All tissues were collected and stored
in RNAlater (Life Technologies, Carlsbad, CA) until RNA isolation
could be performed. Tissues were homogenized in Trizol, and total RNA
was isolated following the manufacturer's instructions. RNA was
further purified using RNeasy Midi Kit (Qiagen, Valencia, CA), in
conjunction with the RNase-Free DNase Set (Qiagen, Valencia, CA),
following the manufacturerâs protocol to remove any contaminating DNA
from the sample. Total RNA was then evaluated on a Nanodrop 1000
(Thermo Scientific, Wilmington, DE) and an Agilent Bioanalyzer 2100
(Santa Clara, CA) to confirm concentration and quality of the
isolation. RNA quality was assessed using A260/A280 and A260/A230
absorbance values.

\DLA{Add details for new sequences and fix numbers}

\subsection{Sequencing}

All RNA-Seq mRNA isolation, library construction, and Illumina
sequencing was performed by the Applied Genomics Technology Center
(AGTC) at Wayne State University School of Medicine using 2.5-5 Âµg of
total RNA and the standard Illumina protocol. Paired-end sequencing
was performed using the Illumina Genome Analyzer II with an insert
size of 230 base pairs (bp). Read lengths for species are shown in
Supplemental \cref{tab:ref_genome_aligned_species}.

\subsubsection{Additional Sequences}

For more complete evolutionary comparisons additional sequences of
placenta in \mouse \cite{Wang.ea2011:surveyfornovelimprinted}, \human
\cite{Hughes.ea2015:Evaluatingintra-andinter-individual} \horse
\cite{Coleman.ea2010:Structuralannotationofequine}, \sheep
\cite{Archibald.ea2010:sheepgenomereferencesequence}, and \pig
\cite{Ahanda.ea2012:Predictionofaltered3-} were obtained from the
\ac{SRA} (Supplemental \cref{tab:srr_numbers}).

\subsection{Transcriptome Assembly, Alignment, and Quantification}

The quality of reads was evaluated using FASTQC v0.11.2
(\url{http://www.bioinformatics.babraham.ac.uk/projects/fastqc/}).
Transcriptomes without a suitable reference genome (\smonkey, \galili,
\carmeli; italic reference in Supplemental
\cref{tab:ref_genome_aligned_species}) were assembled 
using Trinity v2014-07-17 \cite{Haas.ea2013:Denovotranscriptsequence}
and annotated using Diamond
\cite{Buchfink.ea2015:Fastandsensitiveprotein} against either the
\human or \mouse transcriptome
(Supplemental \cref{tab:ref_genome_aligned_species}).
% 
All transcriptomes were then aligned to the appropriate reference (or
assembled) genome using STAR v2.4.2a
\cite{Dobin.ea2013:STARultrafastuniversalRNA-seq} and expression
values in \ac{FPKM}
\cite{Trapnell.ea2010:Transcriptassemblyandquantification} were
quantified using Cufflinks v2.2.1
\cite{Trapnell.ea2009:TopHatdiscoveringsplicejunctions}.
%
The genome of the closely-related \textit{Pan troglodytes} was used as
a reference for \bonobo. The separation between the two
recognized Pan species is generally perceived to be $\approx 1$ Mya
\cite{PrÃ¼fer.ea2012:bonobogenomecomparedwith}.


\begin{figure}
<<species.tree,fig.width=8,fig.height=6,echo=FALSE>>=
species.tree <- 
    read.tree(text="(Monodelphis_domestica,((Dasypus_novemcinctus,Loxodonta_africana)Atlantogenata,((Canis_familiaris,(Equus_caballus,(Sus_scrofa,(Ovis_aries,Bos_taurus))))Laurasiatheria,(((Spalax_carmeli,Nannospalax_galili),Mus_Musculus)Rodentia,((Pan_paniscus,Homo_sapiens),Ateles_fuciceps)Primates)Euarchontoglires)Boreoeutheria)Eutheria);")
pp <- ggtree(species.tree)+
    geom_text(aes(label=label),hjust=0,size=5,fontface="italic")+
    scale_x_continuous(limits=c(0,10))
print(pp %>%
      ## monodelphis
      phylopic("91324e57-b3f1-42e0-abe3-43e5bc8aa4c6", color="#86B875",alpha=0.5,node=1,width=0.2)
      ## Armadillo
      %>% phylopic("5d59b5ce-c1dd-40f6-b295-8d2629b9775e", color="#44EEBB",alpha=0.5,node=2,width=0.18)
      ## elephant
      %>% phylopic("80db1004-bc9f-4318-84e7-cdd9639a1f3e",color="#BB1111",alpha=0.5,node=3,width=0.18)
      ## dog
      %>% phylopic("4d83a0cd-cf06-4a32-9a5a-0a6b644158c1",color="#FFAACC",alpha=0.5,node=4,width=0.19)
      ## horse
      %>% phylopic("071ee517-a0f1-4d19-aa29-812b9f86cb53",color="#22BBBB",alpha=0.5,node=5,width=0.18)
      ## pig
      %>% phylopic("008d6d88-d1be-470a-8c70-73625c3fb4fb",color="#33AA11",alpha=0.5,node=6,width=0.15)
      ## sheep
      %>% phylopic("cbea1964-89bf-4ce1-99b0-58fe0f9170b3",color="#004411",alpha=0.5,node=7,width=0.2)
      ## cow
      %>% phylopic("415714b4-859c-4d1c-9ce0-9e1081613df7",color="#881111",alpha=0.5,node=8,width=0.15)
      ## s carmeli
      %>% phylopic("287c58ad-009b-4878-9a9e-59fd7e0fe66f",color="#881188",alpha=0.5,node=9,width=0.15)
      ## n galili
      %>% phylopic("287c58ad-009b-4878-9a9e-59fd7e0fe66f",color="#888888",alpha=0.5,node=10,width=0.15)
      ## mouse
      %>% phylopic("6b2b98f6-f879-445f-9ac2-2c2563157025",color="#BB11CC",alpha=0.5,node=11,width=0.2)
      ## bonobo
      %>% phylopic("f06b4dd0-094e-4f37-9105-25fdb4eb1b02",color="#1111FF",alpha=0.5,node=14,width=0.18)
      ## spider monkey
      %>% phylopic("aceb287d-84cf-46f1-868c-4797c4ac54a8",color="#FF11FF",alpha=0.5,node=12,width=0.25)
      ## human
      %>% phylopic("a9f4ebd5-53d7-4de9-9e66-85ff6c2d513e",color="#CCCC44",alpha=0.5,node=13,width=0.18)
      )
@ 
\caption{Species tree for organisms studied}
\label{fig:species_tree}
\end{figure}

\subsection{Exploration of Individual Transcriptome Data}

To determine enrichment levels of functional categories of genes in
highly-expressed transcripts in each transcriptome, we performed
functional annotation cluster analyses per transcriptome using topGO
\cite{topGO:2.20.0}. Gene Ontology clusters were considered
significantly enriched if the \ac{FDR} as estimated by the Benjamini
and Hochberg procedure \cite{Benjamini1995:bh_procedure} on from the
$p$ values obtained from the Fisher Exact Test were less than or equal
to 0.05. Enrichment tests were considered with two gene lists per
transcriptome: annotated genes with expression levels of the top
5\% of the frequency distribution of \ac{FPKM} (Supplemental
\cref{tab:go_terms_enriched_top_one_dasypus_novemcinctus,tab:go_terms_enriched_top_one_equus_caballus,tab:go_terms_enriched_top_one_homo_sapiens,tab:go_terms_enriched_top_one_loxodonta_africana,tab:go_terms_enriched_top_one_monodelphis_domestica,tab:go_terms_enriched_top_one_mus_musculus,tab:go_terms_enriched_top_one_ovis_aries,tab:go_terms_enriched_top_one_pan_paniscus,tab:go_terms_enriched_top_one_sus_scrofa,tab:go_terms_enriched_top_one_ateles_fusciceps,tab:go_terms_enriched_top_one_nannospalax_galili,tab:go_terms_enriched_top_one_spalax_carmeli,tab:go_terms_enriched_top_one_bos_taurus,tab:go_terms_enriched_top_one_canis_familiaris});
the top 100 genes with the highest \ac{FPKM} (Supplemental
\cref{tab:go_terms_enriched_dasypus_novemcinctus,tab:go_terms_enriched_equus_caballus,tab:go_terms_enriched_homo_sapiens,tab:go_terms_enriched_loxodonta_africana,tab:go_terms_enriched_monodelphis_domestica,tab:go_terms_enriched_mus_musculus,tab:go_terms_enriched_ovis_aries,tab:go_terms_enriched_pan_paniscus,tab:go_terms_enriched_sus_scrofa,tab:go_terms_enriched_ateles_fusciceps,tab:go_terms_enriched_nannospalax_galili,tab:go_terms_enriched_spalax_carmeli,tab:go_terms_enriched_bos_taurus,tab:go_terms_enriched_canis_familiaris}).
Due to the paucity of annotated gene ontology terms for some species,
gene lists were compared using the \human Gene Ontology
using the 1:1 \human ortholog obtained

\subsection{Comparative Analysis of Gene Expression}

All 1:1 orthologues present among \opossum, \textit{Pan troglodytes},
\cow, \dog, \elephant, \mouse, and \human were determined using
Ensembl v.80 gene trees via Biomart
\cite{Kasprzyk2011:BioMartdrivingparadigmchange} using the biomaRt R
package
\cite{Durinck.ea2009:Mappingidentifiersforintegration,Durinck.ea2005:BioMartandBioconductorpowerful}.
A total of \Sexpr{combined.fpkm[species!="homo sapiens" & mean_fpkm >=
  1,length(unique(human_name))]} 1:1 orthologous genes were expressed
in at least one species. Of these, \Sexpr{combined.fpkm[species!="homo
  sapiens" & mean_fpkm >= 10,length(unique(human_name))]} 1:1
orthologous genes with \ac{FPKM} $\ge 10$ were retained for future
analyses of gene expression evolution described below.

% We used the software \ac{OUCH} in the R statistical environment to
% investigate significant shifts in gene expression levels over
% time\cite{Butler.2004:phlogeneticcomparativeanalysis}. \ac{OUCH}
% institutes an \ac{OU} model of evolution, which allows for different
% optima for quantitative trait values along user-specified branches of
% a phylogenetic tree. The \ac{OU} model explicitly accounts for change
% in quantitative trait values by introducing two parameters into the
% model of stochastic Brownian motion of change in trait value along a
% phylogenetic tree through time. These parameters are $\theta$, an
% optimum trait value, and $\alpha$, the strength of selection
% (interpreted as âpullâ toward the optimum). In addition, the \ac{OU}
% model can also integrate evolutionary change by assuming that each
% user-specified set of braches has its own separate \ac{OU} parameters
% \cite{Butler.2004:phlogeneticcomparativeanalysis}.
% 
% \DLA{The description of the OU process is terrible; it needs to be
%   completely redone}
% 
% Various \ac{OU} branch models were tested against a model in which no
% variation in optima ($\theta$) among branches was allowed. The
% following branches were specified in separate models to have a
% different optimum than all other branches: human, bonobo, mouse, dog,
% cow, elephant, opossum, human+bonobo, Euarchontoglires, and
% Boreoeutheria. Likelihood values were computed for each branch model
% and compared to an \ac{OU} model with one optimum using likelihood
% ratio tests with df=1. We filtered out genes with universally low
% expression levels, and included only genes in which expression in at
% least one species had an $\mathrm{FPKM}\ge10$ as described above. We
% corrected for multiple testing using the \ac{FDR} and using the
% Benjamini-Hochberg correction\cite{Benjamini1995:bh_procedure}. We
% used the time-calibrated phylogenetic tree of
% \cite{Meredith.ea2011:ImpactsofCretaceousTerrestrial} with all species
% not used in this analyses pruned.
% 
% We also calculated the enrichment of functional categories in the
% lists of genes which displayed a significant shift in expression on
% each branch using the same methodology as in the per-species gene
% lists \DLA{reference the tables once this analysis is complete}.

\subsection{Correlation of Tissue Specific Index with Lineage Specific Genes}

To evaluate the extent of tissue specificity of each gene, we
calculated the \ac{TSI} based on human RNASeq data in 3267 samples in
126 tissues the Roadmap Epigenomics
\cite{Bernstein.ea2010:NIHRoadmapEpigenomicsMapping} and GTEx projects
\cite{Gtexconsortium.2015:Thegenotypetissueexpression} as well
additional placenta
\cite{Hughes.ea2015:Evaluatingintra-andinter-individual} and uterus
\cite{Chan.ea2014:Assessmentofmyometrialtranscriptome} samples from
SRA. The \ac{TSI} measures the extent to which a specific gene has
predominant expression in only one tissue
\cite{Yanai.ea2005:Genome-widemidrangetranscriptionprofiles}. A gene
expressed in only one tissue would have a \ac{TSI} of 1, whereas a
gene expressed in all tissues would have a \ac{TSI} of 0.

% \subsection{Validation of Primate Expression using Spider Monkey Placenta Transcriptome}
% 
% To verify expression levels in human and bonobo, we also generated
% RNA-Seq data for the placenta transcriptome of the black-headed spider
% monkey (\textit{Ateles fusciceps}) using similar protocols discussed above.
% Ateles is outside the clade, which includes hominids and can be used
% to polarize expression values of highly expressed genes in Homo and
% Pan. We sampled placental tissue from an Ateles placenta expelled from
% the mother shortly after birth (see Supplemental Table X). We used
% similar procedures discussed above to extract and sequence this
% sample. [DEREKâIâm unsure how to proceed on describing this so as to
% not step on Natalieâs toes]
% 
% \DLA{Complete discussion of the ateles placenta}

\subsection{Sequence Deposition and Code}

All of the code used to produce every analysis presented in this paper
is available at
\url{https://github.com/uiuc-cgm/placenta-viviparous.git}. All of the
sequences used are deposited in SRA, and can be obtained using
\DLA{add GSE label for the experiment}.

\section{Results}

\subsection{Transcriptome Statistics}

Supplemental \cref{fig:transcriptome_statistics} shows the RNAseq read and
alignment statistics for all species. For transcriptomes sequenced
with 75 bp %
%
(\Sexpr{species(star.logs.wide[input.read.length > 70 & input.read.length < 80,unique(species)])})
%
,
the number of trimmed reads ranged from %
\Sexpr{star.logs.wide[input.read.length > 70 & input.read.length < 80,min(as.numeric(input.reads))]} 
%
to %
\Sexpr{star.logs.wide[input.read.length > 70 & input.read.length < 80,max(as.numeric(input.reads))]}
with an average of \Sexpr{star.logs.wide[input.read.length > 70 & input.read.length < 80,mean(`% good reads`)]}\% of reads 
mapping to the respective reference genome (\cref{tab:ref_genome_aligned_species}). 
%
For transcriptomes sequenced with 150 bp, %
% The mismatch here is because we're reporting trimmed reads here, not
% full length reads
(\Sexpr{species(star.logs.wide[input.read.length > 120 & input.read.length < 160,unique(species)])})
%
,
the number of trimmed reads ranged from %
\Sexpr{star.logs.wide[input.read.length > 120 & input.read.length < 160,min(as.numeric(input.reads))]} 
%
to %
\Sexpr{star.logs.wide[input.read.length  > 120 & input.read.length < 160,max(as.numeric(input.reads))]}
%
with an average of \Sexpr{star.logs.wide[input.read.length > 120 & input.read.length < 160,mean(`% good reads`)]}\% %
of reads mapping to the
respective reference genome (\cref{tab:ref_genome_aligned_species}).
%
%
For transcriptomes sequenced with 35 bp, %
% The mismatch here is because we're reporting trimmed reads here, not
% full length reads
(\Sexpr{species(star.logs.wide[input.read.length < 40 ,unique(species)])})
%
,
the number of trimmed reads ranged from %
\Sexpr{star.logs.wide[input.read.length < 40 ,min(as.numeric(input.reads))]} 
%
to %
\Sexpr{star.logs.wide[input.read.length  < 40  ,max(as.numeric(input.reads))]}
%
with an average of \Sexpr{star.logs.wide[input.read.length < 40, mean(`% good reads`)]}\%
of reads mapping to the
respective reference genome (\cref{tab:ref_genome_aligned_species}).
The top 50 expressed genes in each genome analyzed are shown in
\cref{tab:top50_human_placenta,tab:top50_bonobo_placenta,tab:top50_spider_monkey_placenta,tab:top50_dog_placenta,tab:top50_cow_placenta,tab:top50_elephant_placenta,tab:top50_opossum_placenta,tab:top50_mouse_placenta,tab:top50_n_galili_placenta,tab:top50_s_carmeli_placenta,tab:top50_pig_placenta,tab:top50_horse_placenta,tab:top50_sheep_placenta}
and all genes in Supplemental \cref{tab:all_ateles_fusciceps_placenta,tab:all_homo_sapiens_placenta,tab:all_pan_paniscus_placenta,tab:all_mus_musculus_placenta,tab:all_nannospalax_galili_placenta,tab:all_spalax_carmeli_placenta,tab:all_bos_taurus_placenta,tab:all_ovis_aries_placenta,tab:all_sus_scrofa_placenta,tab:all_equus_caballus_placenta,tab:all_canis_familiaris_placenta,tab:all_loxodonta_africana_placenta,tab:all_dasypus_novemcinctus_placenta,tab:all_monodelphis_domestica_placenta}.

% The percentage of bases that mapped to Ensembl annotated mRNA ranged
% from ~35\% in elephant to ~85\% in human, with percentages of bases
% mapping to annotated protein-coding segments of genes ranging from
% ~34\% (elephant) to ~54\% (human), untranslated regions (UTRs) ranging
% from ~0.6\% (elephant) to ~31\% (human), and known introns ranging
% from ~5.5\% (dog) to 11\% (bonobo). Median percent genomic coverage
% ranged from ~43\% in elephant to 80\% in bonobo. The number of
% annotated transcripts to which reads mapped in each respective genomes
% ranged from 4119 (bonobo) to 6506 (human) for transcripts with \ac{FPKM}>10
% and 11689 (dog) to 30733 (human) for transcripts with \ac{FPKM}>1. The
% greater degree of mapping to annotated protein-coding sequences in
% humans corresponds to the greater number of Ensembl annotated
% sequences in the human genome compared to the other five reference
% genomes (Flicek, 2012).


<<top.genes.function,echo=FALSE,results="hide">>=
top.genes.table <- function(species.name,n=50) {
    alignment.species <- species.name
    if (alignment.species=="pan paniscus") {
        alignment.species <- "pan troglodytes"
    } else if (alignment.species=="ateles fusciceps") {
        alignment.species <- "homo sapiens"
    } else if (grepl("spalax",alignment.species)) {
        alignment.species <- "mus musculus"
    }
    temp <- 
        combined.fpkm[species==species.name &
                      !(!is.na(human_name) & 
                        human_name %in% housekeeping.genes.superset[,gene_short_name]) &
                      ! (gene_biotype %in% c("Mt_tRNA","Mt_rRNA","rRNA")) &
                      !grepl("((RNA\\d|MT)-|^RP[SL]P?\\d+[ABCD]?[L]?$)",human_name) &
                      !grepl("((RNA\\d|MT)-|^RP[SL]P?\\d+[ABCD]?[L]?$)",human_alignment_symbol) &
                      !grepl("(^[Mm][Tt]-|^R[pP][lsLS]\\d+[abcdABCD]?[lL]?(?:-.+)?$|rRNA$)",
                             gene_short_name)
                      ][order(-mean_fpkm)][,list(name_or_id,
                                                 human_alignment_symbol,
                                                 human_name,mean_fpkm,sd_fpkm,n_fpkm)]
    ## remove any genes which have a null name_or_id; they are probably from unassembled contigs
    temp <- temp[!(is.na(name_or_id)|name_or_id==""),]
    temp <- temp[!duplicated(name_or_id)][1:n]
    temp[,used_alignment:=FALSE]
    temp[is.na(human_name) & !is.na(human_alignment_symbol),
                    used_alignment:=TRUE]
    temp[is.na(human_name),human_name:=human_alignment_symbol]
    setkey(temp,"human_name")
    temp <- 
        hugo.nomenclature[temp][order(-mean_fpkm),
                                list(name_or_id,used_alignment,`Approved Symbol`,`Approved Name`,
                                     mean_fpkm,sd_fpkm,n_fpkm)]
    temp[!is.na(name_or_id),
         name_or_id:=paste0("\\href{http://www.ensembl.org/",
                             capfirst(gsub(" ","_",alignment.species)),
                             "/Gene/Summary?g=",name_or_id,"}{",
                             xtable.sanitize.text(name_or_id),"}")]
    temp[!is.na(`Approved Name`),
         `Approved Name`:=paste0("\\href{http://www.ensembl.org/Homo_sapiens/Gene/Summary?g=",`Approved Symbol`,"}{",xtable.sanitize.text(`Approved Name`),"}")]
    temp[!is.na(`Approved Symbol`),
         `Approved Symbol`:=paste0("\\href{http://www.ensembl.org/Homo_sapiens/Gene/Summary?g=",`Approved Symbol`,"}{",xtable.sanitize.text(`Approved Symbol`),"}")]
    temp[used_alignment==TRUE,`Approved Symbol`:=paste0("(",`Approved Symbol`,")")]
    temp[,used_alignment:=NULL]
    temp[,sem_fpkm:=as.numeric(NA)]
    temp[n_fpkm>1,sem_fpkm:=sd_fpkm/sqrt(n_fpkm)]
    temp[,sd_fpkm:=NULL]
    temp[,n_fpkm:=NULL]
    setnames(temp,c("Gene symbol","Human Symbol","Human Name","FPKM","FPKM SEM"))
    if (temp[,all(is.na(`FPKM SEM`))]) {
        temp[,`FPKM SEM`:=NULL]
    }
    if (species.name=="homo sapiens" || species.name=="ateles fusciceps") {
        temp[,`Human Symbol`:=NULL]
    }
    temp[,`FPKM`:=round(`FPKM`)]
    return(temp)
}
gene.expression <- function(species.name,gene,complete=FALSE,units=TRUE) {
    species.name <- gsub("_"," ",species.name)
    fpkm <- combined.fpkm[species==species.name &
                              (gene_short_name==gene | 
                               gene_id==gene),
                          mean_fpkm][1]
    if (units || complete)
        fpkm <- formatC(fpkm,digits=0,format="f")
    if (units) {
        fpkm <- paste0(fpkm,
                       "\\ \\text{FPKM}")
    }
    if (complete) {
        if (any(grepl("^ENS.+G\\d+$",gene))) {
            fpkm <- paste0("\\text{",gene,"} = ",
                           fpkm)
        } else {
            fpkm <- paste0("\\text{\\textit{",gene,"}} = ",
                           fpkm)
        }
    }
    if (units || complete) {
        fpkm <- paste0("$",fpkm,"$")
    }
    return(fpkm)
}
    
@ 

<<load.placenta.lineage.specific,echo=FALSE,results="hide",cache.extra=tools::md5sum(paste0("../data/",c("placenta_lineage_specific")))>>=
load("../data/placenta_lineage_specific")
source("/home/don/projects/xtable/pkg/R/print.xtable.R")

placenta.lineage.specific.table <- function(analysis,output=TRUE) {
    temp.table <- 
        head(placenta.lineage.specific.analysis[analysis.name==analysis &
                                                p.value <= 0.05][order(p.value),
                                                                 list(human_name,p.value,fdr,fc),],
             n=5)
    temp.table <- data.table(temp.table)
    setkey(temp.table,"human_name")
    temp.table <- 
        hugo.nomenclature[temp.table][,list(`Approved Symbol`,
                                            `Approved Name`,p.value,fdr,fc)]
    setnames(temp.table,
              c("Human Symbol","Human Name","p (t-test)","FDR","Fold Change")
              )
    setkey(temp.table,"Human Symbol")
    if (output) {
        print(xtable(temp.table[order(`p (t-test)`),],
                     display=c("s","s","s","e","g","f"),
                     ),
              booktabs=TRUE,
              floating=FALSE,
              math.style.exponents=TRUE,
              sanitize.text.function=genes.table.sanitize,
              include.rownames=FALSE)
        
    }
    return(invisible(temp.table[order(`p (t-test)`),]))
}
output.top.genes.table <- function(data,output=TRUE) {
    if (output) {
        table.display <- c("s","s","s","d")
        table.align <- c("c","c","p{3in}","r")
        if (any(colnames(data) %in% "FPKM SEM"))  {
            table.display <- c(table.display,"d")
            table.align <- c(table.align,"r")
        }
        if (any(colnames(data) %in% "Human Symbol"))  {
            table.align <- c("c",table.align)
            table.display <- c("s",table.display)
        }
        print(xtable(data,
                     display=table.display,
                     align=table.align
                     ),
              booktabs=TRUE,
              floating=FALSE,
              sanitize.text.function=genes.table.sanitize,
              include.rownames=FALSE)
    }
}

@ 

\subsection{Primate Placenta Transcriptomes}

\begin{table}
  \footnotesize
<<primate.lineage.specific,echo=FALSE,results="asis">>=
pri.all.lin.spec <- placenta.lineage.specific.table("pri.all")
setkey(pri.all.lin.spec,"Human Symbol")
@ 
\caption{Genes whose expression changed significantly in the primate
  order as compared to all other analyzed species.}
\label{tab:primate_lineage_specific}
\end{table}

We examined the expression of three primates, \human, \bonobo, and
\smonkey. The genes whose expressions are different on the primate
lineage in comparison to all other analyzed species are listed in
\cref{tab:primate_lineage_specific}. \textit{FSTL1} has significantly
higher expression in the primate lineage
(Supplemental \cref{fig:fstl1_evolution_tree},
$\mathrm{FDR}=\Sexpr{pri.all.lin.spec["FSTL1",`FDR`]}$), is implicated
in BMP signaling, and has previously been implicated in inflammation
and invasiveness
\cite{Mouillet.ea2015:expressionandpost-transcriptionalregulation}. Of
non-primates, \horse shows the highest expression
%
\Sexpr{gene.expression(species.name="equus caballus",gene="FSTL1",complete=TRUE)}, 
%
which is still lower than the lowest primate, \smonkey
%
(\Sexpr{gene.expression(species.name="ateles fusciceps",gene="FSTL1",complete=TRUE)}). 
%
No other genes were significantly differentially expressed on this
lineage by t-test.

\subsubsection{\human Placenta Transcriptome}
\label{sec:human_placenta}

\begin{table}
  \footnotesize
<<top.50.human.expressed.genes,echo=FALSE,results="asis">>=
output.top.genes.table(top.genes.table(species.name="homo sapiens"))
@ 
\caption{The top 50 genes by \ac{FPKM} in \human which are not
  annotated as Mt\_rRNA, Mt\_tRNA, rRNA or a housekeeping gene. Gene
  symbol is the gene symbol or Ensembl gene id. Human Name is the HUGO
  name corresponding to the human symbol. A complete list of all genes
  is in \cref{tab:all_homo_sapiens_placenta}.}
\label{tab:top50_human_placenta}
\end{table}

<<cgb.min.expression,results="hide",echo=FALSE,warnings=FALSE>>=
cgb.min.expression <- 
    min(sapply(paste0("CGB",c("","1","2","5","7","8")),
               function(x){gene.expression(species.name="homo sapiens",gene=x,units=FALSE)}))
cgb.sum.expression <- 
    sum(sapply(paste0("CGB",c("","1","2","5","7","8")),
               function(x){gene.expression(species.name="homo sapiens",gene=x,units=FALSE)}))
@ 

The top 50 expressed genes as measured by \ac{FPKM} in the term
placenta of \human are shown in
\cref{tab:top50_human_placenta} with all genes with \ac{FPKM} $\ge 50$
in Supplemental \cref{tab:all_homo_sapiens_placenta}. The gene with by far
the highest \ac{FPKM} value
%
(\Sexpr{gene.expression(species.name="homo sapiens",gene="CSH1")}) %
%
in \human is \textit{CSH1}, followed by \textit{CSH2}, and \textit{CSHL1}, all
three of which are members of a distinct expansion of the growth
hormone gene family within hominoids (\cref{fig:galectins_evolution_tree}). Another member of the growth
hormone gene family, \ac{GH2} (\ac{FPKM}=%
\Sexpr{gene.expression(species.name="homo sapiens",gene="GH2")}), %
is also expressed to a high degree in the \human placenta
(\cref{tab:top50_human_placenta}).
% 
Eleven \acp{PSG} are highly-expressed
(\ac{FPKM}$\ge 100$) in the term placenta of \human, with \textit{PSG1} showing the
greatest degree of expression %
(\Sexpr{gene.expression(species.name="homo sapiens",gene="PSG1")}) % 
and the pseudogene \textit{PSG10P} showing the least %
(\Sexpr{gene.expression(species.name="homo sapiens",gene="PSG10P")}, \cref{fig:pag2_prss16_ctsl_psg_evolution_tree}~A) . %
These
genes are members of a hominoid-specific expansion of the \ac{CEA} family
and known to be expressed by the syncytiotrophoblast
\cite{Rawn.ea2008:evolutionregulationandfunction,Lei.ea1992:Cloningandexpressionof}. %
%
Genes involved in the production of estrogens (%
\Sexpr{gene.expression(species.name="homo sapiens",gene="CYP19A1",complete=TRUE)},
% 
\Sexpr{gene.expression(species.name="homo sapiens",gene="HSD17B1",complete=TRUE)}, and
% 
\Sexpr{gene.expression(species.name="homo sapiens",gene="HSD3B1",complete=TRUE)}%
% 
) and progesterone (%
\Sexpr{gene.expression(species.name="homo sapiens",gene="HSD3B1",complete=TRUE)},
\Sexpr{gene.expression(species.name="homo sapiens",gene="CYP11A1",complete=TRUE)}, and
\Sexpr{gene.expression(species.name="homo sapiens",gene="ADM",complete=TRUE)})
%
show exceptionally high expression in the human placenta, as well as %
%
\textit{CGA} (FPKM=\Sexpr{gene.expression(species.name="homo sapiens",gene="CGA")}, %
%
which generates the alpha subnit of multiple human chorionic
gonadotropin hormones (\cref{tab:top50_human_placenta}). The genes
producing the beta subunits of the chorionic gonadotropins
(\textit{CGB}, \textit{CGB1}, \textit{CGB2}, \textit{CGB5},
\textit{CGB7}, \textit{CGB8}) have reasonable expression
($\mathrm{FPKM} \ge \Sexpr{cgb.min.expression}$), and the sum of their
expression is on the order of the expression of expression of
\textit{CGA} ($\sum \mathit{CGBs}=\Sexpr{cgb.sum.expression}$ FPKM) .
%
%
Other highly expressed genes in the top 50 include members of the
hemoglobin family (e.g. \textit{HBA1}, \textit{HBB}, \textit{HBG2},
\textit{HBA2}), the maternally imprinted long noncoding RNA gene
\textit{H19}, genes coding for proteins required for growth and/or
cell proliferation (\textit{ADAM12}, \textit{KISS1}, \textit{IGF2},
\textit{GPC3}, \textit{CSF3R}), anticoagulation (\textit{TFPI2}), and
glucose transport (\textit{SLC2A1}) among others
(\cref{tab:top50_human_placenta}, Supplemental
\cref{tab:all_homo_sapiens_placenta}).



\begin{figure}
<<psg.ctsl.evolution.tree,echo=FALSE,fig.width=16,fig.height=16,warning=FALSE,message=FALSE>>=
plot.label.size <- 3

pushViewport(viewport(layout=grid.layout(nrow=2,ncol=2)))
tree.id <-
    genes.to.tree[combined.fpkm[gene_short_name=="PSG1" & species=="homo sapiens",gene_id],tree]
gene.tree.table <- trees[[tree.id]]$table
gene.tree.expression <- combined.fpkm[gene_id %in% gene.tree.table[,gene_id]]
gene.tree <- read.tree(text=trees[[tree.id]]$tree.dat)
plot.tree.matrix(read.tree(text=trees[[tree.id]]$tree.dat),gene.tree.table,gene.tree.expression,min.fpkm=1,
                 vp=viewport(layout.pos.col=1,layout.pos.row=1),
                 fontsize=plot.label.size
                 )

tree.id <-
    genes.to.tree[combined.fpkm[gene_short_name=="CTSL" & species=="homo sapiens",gene_id],tree]
gene.tree.table <- trees[[tree.id]]$table
gene.tree.expression <- combined.fpkm[gene_id %in% gene.tree.table[,gene_id]]
gene.tree <- read.tree(text=trees[[tree.id]]$tree.dat)
plot.tree.matrix(gene.tree,gene.tree.table,gene.tree.expression,min.fpkm=1,
                 subtree=c("ENSMODG00000002844","ENSMUSG00000055679"),
                 vp=viewport(layout.pos.col=2,layout.pos.row=1),
                 fontsize=plot.label.size
                 )

tree.id <-
    genes.to.tree[combined.fpkm[gene_short_name=="PRSS16" & species=="homo sapiens",gene_id],tree]
gene.tree.table <- trees[[tree.id]]$table
gene.tree.expression <- combined.fpkm[gene_id %in% gene.tree.table[,gene_id]]
gene.tree <- read.tree(text=trees[[tree.id]]$tree.dat)
plot.tree.matrix(read.tree(text=trees[[tree.id]]$tree.dat),gene.tree.table,gene.tree.expression,
                 vp=viewport(layout.pos.col=1,layout.pos.row=2),
                 fontsize=plot.label.size
                 )

tree.id <-
    genes.to.tree[combined.fpkm[gene_short_name=="PAG2" & species=="bos taurus",gene_id],tree]
gene.tree.table <- trees[[tree.id]]$table
gene.tree.expression <- combined.fpkm[gene_id %in% gene.tree.table[,gene_id]]
gene.tree <- read.tree(text=trees[[tree.id]]$tree.dat)
plot.tree.matrix(read.tree(text=trees[[tree.id]]$tree.dat),
                 gene.tree.table,gene.tree.expression,min.fpkm=1,
                 subtree=c("ENSSSCG00000006792",
                           "ENSBTAG00000046085"),
                 vp=viewport(layout.pos.col=2,layout.pos.row=2),
                 fontsize=plot.label.size
                 )

popViewport(1)
grid.text(c("A","B","C","D"),
          x=unit(c(0,0.5,0,0.5),"npc"),y=unit(c(1,1,0.5,0.5),"npc"),
          just=c("left","top"),
          gp=gpar(fontsize="48")
          )
@ 
\caption{\genetreeexplanation{(A) \textit{PSG1}, (B) \textit{CTSL}, (C) \textit{PRSS16}, and (D) \textit{PAG2}}}
\label{fig:pag2_prss16_ctsl_psg_evolution_tree}
\end{figure}


\subsubsection{\bonobo Placenta Transcriptome}

\begin{table}
  \footnotesize
<<top.50.bonobo.expressed.genes,echo=FALSE,results="asis">>=
top.bonobo.genes <- top.genes.table(species.name="pan paniscus")
output.top.genes.table(top.bonobo.genes)
setkey(top.bonobo.genes,"Gene symbol")
@ 
\caption{\tableexplanation{\bonobo}{tab:all_pan_paniscus_placenta}}
\label{tab:top50_bonobo_placenta}
\end{table}

The top 50 expressed genes as measured by \ac{FPKM} in the term
placenta of \bonobo are shown in \cref{tab:top50_bonobo_placenta}
with a list of all genes in Supplemental
\cref{tab:all_pan_paniscus_placenta}. Expression in the \bonobo
placenta differs somewhat from that of \human, as evidenced by the
makeup of the top 50 genes (\cref{tab:top50_bonobo_placenta}). The highest expression
 in \bonobo is %
\Sexpr{gene.expression("pan paniscus","PRG2",complete=TRUE)}, %
%
which is involved in insulin growth factor signaling and angiogenesis
during pregnancy \cite{Weyer.ea2011:Placentalregulationofpeptide}.
\textit{PRG2} is also expressed at relatively high levels (though
comparatively much lower) in the human placenta
(\Sexpr{gene.expression("homo sapiens","PRG2")};
Supplemental \cref{tab:all_homo_sapiens_placenta}). Four \bonobo genes
related to the chorionic somatomammotropins
are also expressed at relatively high levels (GH1 
\Sexpr{gene.expression("pan paniscus","ENSPTRG00000034560",complete=TRUE)}, 
\Sexpr{gene.expression("pan paniscus","ENSPTRG00000034212",complete=TRUE)}, 
GH2
\Sexpr{gene.expression("pan paniscus","ENSPTRG00000030805",complete=TRUE)}, 
\Sexpr{gene.expression("pan paniscus","ENSPTRG00000034209",complete=TRUE)}, 
%
\cref{tab:all_pan_paniscus_placenta}, \cref{fig:csh1_evolution_tree}).
Placental expression of the genes involved in estrogen and
progesterone production (\textit{CYP19A1}, \textit{HSD3B1}
respectively) show lower expression in \bonobo as compared to \human %
(\Sexpr{gene.expression("pan paniscus","CYP19A1",complete=TRUE)},
\cref{fig:estrogen_evolution_tree}; %
\Sexpr{gene.expression("pan paniscus","HSD3B1",complete=TRUE)},
\cref{fig:progesterone_evolution_tree} and
\cref{tab:top50_human_placenta}) although the expression of
\textit{HSD3B1} is fairly similar to that of \smonkey %
(\Sexpr{gene.expression("ateles fusciceps","HSD3B1",complete=TRUE)}. %
In addition, \textit{CGA} is expressed in the bonobo placenta %
(\Sexpr{gene.expression("pan paniscus","CGA")}), but at much
lower levels than \human as well (\cref{tab:top50_human_placenta}).
%
Two Ensembl-annotated beta subunits of the chorionic gonadotropin
genes are expressed in \bonobo, one related to \textit{CGB} and
another related to \textit{CGB7}; both are not highly expressed
(\cref{fig:cgb_evolution_tree}). Among the other top 50 expressed
genes, there is high expression of keratins (e.g., \textit{KRT19},
\textit{KRT18}), heat-shock proteins (e.g., \textit{HSP90B1},
\textit{HSPA1A}), serpins (\textit{SERPINE1}, \textit{SERPINE2},
Supplemental \cref{fig:serpine_evolution_tree}), insulin growth factor
2 (\textit{IGF2}) and one of its binding proteins \textit{IGFBP3}, and
the hormone adrenomedullin (\textit{ADM}).

\subsubsection{\smonkey Transcriptome}

\begin{table}
  \footnotesize
<<top.50.spider.monkey.expressed.genes,echo=FALSE,results="asis">>=
top.spider.monkey.genes <- top.genes.table(species.name="ateles fusciceps")
output.top.genes.table(top.spider.monkey.genes)
setkey(top.spider.monkey.genes,"Gene symbol")
@ 
\caption{\tableexplanation{\smonkey}{tab:all_ateles_fusciceps_placenta}}
\label{tab:top50_spider_monkey_placenta}
\end{table}

For the term placenta of \smonkey, assembled contigs from RNA-seq were
aligned using against \human and the best alignment was chosen as the
annotation for each assembled contig. In the term placenta of
\smonkey, the top 50 non-housekeeping genes
(\cref{tab:top50_spider_monkey_placenta}) include a possible regulator
of androgen receptor signaling
\cite{Sampson.ea2012:PAGE4positivityisassociated}, \textit{PAGE4} %
(\Sexpr{gene.expression("ateles_fusciceps","PAGE4")}), %
the pregnancy specific glycoprotein \textit{PSG8}
(\Sexpr{gene.expression("ateles_fusciceps","PSG8")}), %
a member of the chorionic somatotropins, \textit{GH1} %
(\Sexpr{gene.expression("ateles_fusciceps","GH1")}), %
a few HLA genes (\textit{HLA-A}, \textit{HLA-F},
\cref{tab:top50_spider_monkey_placenta}), and multiple components of
annexin complexes
(\Sexpr{gene.expression("ateles_fusciceps","ANXA2",complete=TRUE)}, %
\Sexpr{gene.expression("ateles_fusciceps","S100P",complete=TRUE)}, %
\Sexpr{gene.expression("ateles_fusciceps","S100A11",complete=TRUE)}) %
which may play a role in angiogenesis, and certainly regulate
cell-cell interaction in the developing placenta
\cite{Liu.ea2015:AnnexinA2complexeswith}.
\Sexpr{gene.expression("ateles_fusciceps","AL365202.1",complete=TRUE)}
is also highly expressed, has no known function, and is duplicated
multiple times in \textit{Macaca mulatta}. All genes with expression
greater than 50 \ac{FPKM} are listed in Supplementary
\cref{tab:all_ateles_fusciceps_placenta}.


\subsection{Rodentia Placenta Transcriptomes}

We examined the expression of three rodentia, \mouse, \carmeli, and
\galili{}. We lacked sufficient power to detect any significant
changes on the Rodentia lineage in comparison to all other analyzed
species (Supplemental \cref{tab:rodentia_lineage_specific}).

\subsubsection{\mouse Placenta Transcriptome}

\begin{table}[h]
  \footnotesize
<<top.50.mouse.expressed.genes,echo=FALSE,results="asis">>=
output.top.genes.table(top.genes.table(species.name="mus musculus"))
@ 
\caption{\tableexplanation{\mouse}{tab:all_mus_musculus_placenta}}
\label{tab:top50_mouse_placenta}
\end{table}

The 50 non-housekeeping genes with highest expression (by \ac{FPKM})
in the term placenta of \mouse are shown in
\cref{tab:top50_mouse_placenta} with a list of all genes with
$\mathrm{FPKM} \ge 50$ in Supplemental
\cref{tab:all_mus_musculus_placenta}. The top expressed genes include
\Sexpr{gene.expression("mus_musculus","Mir6236",complete=TRUE)}, whose
target \textit{in silico} is the orphan G-coupled receptor
\textit{Mrgprg}
\cite{Agarwal.ea2015:PredictingeffectivemicroRNAtarget},
\Sexpr{gene.expression("mus_musculus","H19",complete=TRUE)} and
\Sexpr{gene.expression("mus_musculus","Gm27786",complete=TRUE)}, both
of which are maternally imprinted (the later of which is a homolog of
human \textit{H19\_2}),
\Sexpr{gene.expression("mus_musculus","Igf2",complete=TRUE)}, numerous
cathepsins and a rodentia specific expansion of prolactin related
genes. There are also two miRNA which overlap with the of the 7S
signal recognition particle
(\Sexpr{gene.expression("mus_musculus","AC157822.1",complete=TRUE)}
and
\Sexpr{gene.expression("mus_musculus","AC099934.1",complete=TRUE)})
and numerous pseudogenes
(\Sexpr{gene.expression("mus_musculus","Gm11966",complete=TRUE)}
\Sexpr{gene.expression("mus_musculus","Gm10736",complete=TRUE)}
\Sexpr{gene.expression("mus_musculus","Gm10180",complete=TRUE)}
\Sexpr{gene.expression("mus_musculus","Gm29216",complete=TRUE)}) which
appear to be highly expressed, however, the short read length (34--44)
in many samples is likely preventing unambiguous assignment of all
reads to the more lengthy complete transcript, so these shorter
transcripts should be treated with suspicion, and we will ignore them
in further discussion. The prolactin-related genes
(\Sexpr{gene.expression("mus_musculus","Prl3b1",complete=TRUE)},
\Sexpr{gene.expression("mus_musculus","Prl2b1",complete=TRUE)},
\Sexpr{gene.expression("mus_musculus","Prl8a9",complete=TRUE)}, and
\Sexpr{gene.expression("mus_musculus","Prl8a6",complete=TRUE)}) are
very highly expressed in \mouse in, with %
\Sexpr{combined.fpkm[species=="mus musculus" &
  grepl("Prl(\\d.\\d)?",gene_short_name), sum(mean_fpkm >= 1)]}
prolactin related genes having $\mathrm{FPKM} \ge 1$
(\cref{fig:prolactin_evolution_tree}, Supplemental
\cref{fig:prolactin_receptor_evolution_tree}). The cathespins have
also expanded in rodents, with six showing high expression
(\Sexpr{gene.expression("mus_musculus","Ctsj",complete=TRUE)},
\Sexpr{gene.expression("mus_musculus","Cts6",complete=TRUE)},
\Sexpr{gene.expression("mus_musculus","Ctsr",complete=TRUE)},
\Sexpr{gene.expression("mus_musculus","Ctsq",complete=TRUE)},
\Sexpr{gene.expression("mus_musculus","Cts3",complete=TRUE)}, and
\Sexpr{gene.expression("mus_musculus","Ctsm",complete=TRUE)},
\cref{tab:top50_mouse_placenta,fig:cathespin_evolution_tree}).
Finally, \textit{Peg10}, an imprinted gene required for proper
formation of the placenta in mice is also highly expressed
(\Sexpr{gene.expression("mus_musculus","Peg10")}), as previously
reported \cite{Ono.ea2006:DeletionofPeg10imprinted}.

\subsubsection{\galili Placenta Transcriptome}

\begin{table}[h]
  \footnotesize
<<top.50.n.galili.expressed.genes,echo=FALSE,results="asis">>=
output.top.genes.table(top.genes.table(species.name="nannospalax galili"))
@ 
\caption{\tableexplanation{\galili}{tab:all_nannospalax_galili_placenta}}
\label{tab:top50_n_galili_placenta}
\end{table}

The 50 non-housekeeping genes with highest expression (by \ac{FPKM})
in the term placenta of \galili are shown in
\cref{tab:top50_n_galili_placenta} with a list of all highly expressed
genes in Supplemental \cref{tab:all_nannospalax_galili_placenta}.
Components of annexin complexes are highly expressed
(\Sexpr{gene.expression("nannospalax_galili","S100a6",complete=TRUE)},
\Sexpr{gene.expression("nannospalax_galili","S100a11",complete=TRUE)},
\Sexpr{gene.expression("nannospalax_galili","S100a10",complete=TRUE)},
and
\Sexpr{gene.expression("nannospalax_galili","Anxa2",complete=TRUE)}),
as are members of the thymosin family
(\Sexpr{gene.expression("nannospalax_galili","Tmsb4x",complete=TRUE)}
and
\Sexpr{gene.expression("nannospalax_galili","Tmsb10",complete=TRUE)}),
various cytoskeleton related proteins
(\Sexpr{gene.expression("nannospalax_galili","Actc1",complete=TRUE)},
\Sexpr{gene.expression("nannospalax_galili","Actg2",complete=TRUE)},
\Sexpr{gene.expression("nannospalax_galili","Acta2",complete=TRUE)},
\Sexpr{gene.expression("nannospalax_galili","Myl9",complete=TRUE)},
\Sexpr{gene.expression("nannospalax_galili","Vim",complete=TRUE)}), and
transmembrane protein of currently unknown function, 
\Sexpr{gene.expression("nannospalax_galili","Gm10134",complete=TRUE)}.


\subsubsection{\carmeli Placenta Transcriptome}

\begin{table}[h]
  \footnotesize
<<top.50.s.carmeli.expressed.genes,echo=FALSE,results="asis">>=
output.top.genes.table(top.genes.table(species.name="spalax carmeli"))
@ 
\caption{\tableexplanation{\carmeli}{tab:all_spalax_carmeli_placenta}}
\label{tab:top50_s_carmeli_placenta}
\end{table}

The 50 non-housekeeping genes with highest expression (by \ac{FPKM})
in the term placenta of \carmeli are shown in
\cref{tab:top50_s_carmeli_placenta} with a list of all highly
expressed genes in Supplemental
\cref{tab:all_spalax_carmeli_placenta}. Numerous hemoglobin subunits
ware highly expressed
(\Sexpr{gene.expression("spalax_carmeli","Hbb-bh1",complete=TRUE)},
\Sexpr{gene.expression("spalax_carmeli","Hbb-a1",complete=TRUE)}, and
\Sexpr{gene.expression("spalax_carmeli","Hbb-bs",complete=TRUE)}), as
well as the relaxin
\Sexpr{gene.expression("spalax_carmeli","Rln1",complete=TRUE)},
multiple cathespins
(\Sexpr{gene.expression("spalax_carmeli","Ctsm",complete=TRUE)},
\Sexpr{gene.expression("spalax_carmeli","BC051665",complete=TRUE)},
and \Sexpr{gene.expression("spalax_carmeli","Ctsl",complete=TRUE)},
\cref{fig:pag2_prss16_ctsl_psg_evolution_tree}~B), and many prolactin-related genes
(\Sexpr{gene.expression("spalax_carmeli","Prl4a1",complete=TRUE)},
\Sexpr{gene.expression("spalax_carmeli","Prl6a1",complete=TRUE)},
\Sexpr{gene.expression("spalax_carmeli","Prl3b1",complete=TRUE)}, and
\Sexpr{gene.expression("spalax_carmeli","Prl7c1",complete=TRUE)},
\cref{fig:prolactin_evolution_tree}) which shows a similar pattern of
expansion seen in \mouse.

\subsection{Laurasiatheria}

We examined the expression of five Laurasiatherians: \cow, \sheep,
\pig, \horse, and \dog. We were unable to detect any significant
differences after correcting for multiple testing between
Laurasiatherians and all other analyzed species
(Supplemental \cref{tab:laurasiatheria_lineage_specific}) or between
Laurasiatherians and Euasiatherians
(Supplemental \cref{tab:laurasiatheria_eurasiatherian_lineage_specific}).

\subsubsection{\dog Placenta Transcriptome}

\begin{table}
  \footnotesize
<<top.50.dog.expressed.genes,echo=FALSE,results="asis">>=
output.top.genes.table(top.genes.table(species.name="canis familiaris"))
@ 
\caption{\tableexplanation{\dog}{tab:all_canis_familiaris_placenta}}
\label{tab:top50_dog_placenta}
\end{table}

The top 50 non-housekeeping expressed genes as measured by \ac{FPKM}
in the term placenta of \dog are shown in
\cref{tab:top50_dog_placenta} with a list of all genes in Supplemental
\cref{tab:all_canis_familiaris_placenta}. The gene with the highest
expression %
(\Sexpr{gene.expression("canis familiaris","TMSB10")}) is the thymosin 
\textit{TMSB10} %
(\Sexpr{gene.expression("canis familiaris","ENSCAFG00000002115")}) %
is the hormone relaxin (\textit{RLN2}). Other highly expressed genes
among the top 50 (\cref{tab:top50_dog_placenta}) include genes
involved in the cytoskeleton (\textit{TMSB10}, \textit{TMSB4X}),
anticoagulation (\textit{TFPI2}, \textit{PLA2G7}), innate immunity
(\textit{LCN2}, \textit{GM2A}), lipid metabolism (\textit{APOE}),
inhibition of matrix metalloproteinases (\textit{TIMP2},
\textit{TIMP3}), and growth factors (\textit{PGF}, \textit{EGFR},
\textit{HTRA1}). The gene \textit{LGALS1}, which shows evidence of
immunosupression during pregnancy
\cite{Than.ea2008:Emergenceofhormonaland}, is highly expressed %
(\Sexpr{gene.expression("canis familiaris","LGALS1")}). %
In addition, an ortholog of \textit{SSLP1} (ENSCAFG00000010638) a gene
expressed in the placentas of \cow (%
\Sexpr{gene.expression("bos taurus","SSLP1",complete=TRUE)}, %
\cref{tab:top50_cow_placenta}) and \sheep
(\Sexpr{gene.expression("ovis aries","SOLD1",complete=TRUE)},
\cref{tab:top50_sheep_placenta}) but is not present in primates
\cite{Ushizawa.ea2009:Characterizationandexpressionanalysis}
(\cref{fig:sslp1_evolution_tree}), is also expressed at high levels
%
(\Sexpr{gene.expression("canis familiaris","ENSCAFG00000010638")}). %
There is no placental expression of the estrogen synthase gene
(\textit{CYP19A1}) in \dog 
(\Sexpr{gene.expression("canis familiaris","CYP19A1")}
\cref{fig:estrogen_evolution_tree}), 
and expression of \textit{HSD3B7} is also low
(\Sexpr{gene.expression("canis familiaris","HSD3B7")}, 
\cref{fig:progesterone_evolution_tree}).

\subsubsection{\cow Placenta Transcriptome}

\begin{table}
  \footnotesize
<<top.50.cow.expressed.genes,echo=FALSE,results="asis">>=
output.top.genes.table(top.genes.table(species.name="bos taurus"))
@ 
\caption{\tableexplanation{\cow}{tab:all_bos_taurus_placenta}}
\label{tab:top50_cow_placenta}
\end{table}

The top 50 expressed genes as measured by \ac{FPKM} in the term
placenta of the \cow are shown in \cref{tab:top50_cow_placenta} with a
list of all genes in Supplemental \cref{tab:all_bos_taurus_placenta}.
The gene with the highest expression was \textit{TMSB10} %
(\Sexpr{gene.expression("bos taurus","TMSB10")}), and
\textit{TMSB4} was also highly expressed %
(\Sexpr{gene.expression("bos taurus","TMSB4")}). As in
\dog{}, members of annexin complexes were also highly expressed
(\Sexpr{gene.expression("bos taurus","S100G",complete=TRUE)}, %
\Sexpr{gene.expression("bos taurus","ANXA2",complete=TRUE)}, %
\Sexpr{gene.expression("bos taurus","S100A2",complete=TRUE)}, and %
\Sexpr{gene.expression("bos taurus","S100A11",complete=TRUE)}), as well as %
at least one of a set of four miRNA with 100\% identity
(\Sexpr{gene.expression("bos taurus","ENSBTAG00000047046",complete=TRUE)}, %
\Sexpr{gene.expression("bos taurus","ENSBTAG00000047154",complete=TRUE)}, %
\Sexpr{gene.expression("bos taurus","ENSBTAG00000045710",complete=TRUE)}, and %
\Sexpr{gene.expression("bos taurus","ENSBTAG00000046351",complete=TRUE)}). 
%
As these are novel miRNA, there is a possibility that the assignment
of the short reads to these four miRNA is due to miss-annotation of
the \cow genome.
% 
Several members of the ruminant-specific pregnancy-associated
glycoprotein family show especially high expression in \cow placenta
(\cref{tab:top50_cow_placenta}, \cref{fig:pag2_prss16_ctsl_psg_evolution_tree}~D, and
Supplemental \cref{tab:all_bos_taurus_placenta}) including
%
\Sexpr{gene.expression("bos taurus","PAG2",complete=TRUE)}, %
\Sexpr{gene.expression("bos taurus","PAG10",complete=TRUE)}, %
\Sexpr{gene.expression("bos taurus","ENSBTAG00000022759",complete=TRUE)}, %
\Sexpr{gene.expression("bos taurus","ENSBTAG00000013214",complete=TRUE)}, %
\Sexpr{gene.expression("bos taurus","ENSBTAG00000019226",complete=TRUE)}, %
\Sexpr{gene.expression("bos taurus","ENSBTAG00000024611",complete=TRUE)}, %
\Sexpr{gene.expression("bos taurus","PAG12",complete=TRUE)}. %
%
28 members of this family have $FPKM \ge 1$, and this family of
genes has greatly expanded in ruminants (\cow and \sheep{}) although
the expression levels of this family are higher in \cow than in \sheep
\cref{fig:pag2_prss16_ctsl_psg_evolution_tree}~D. Other genes with high expression
include genes involved with regulation of growth (\textit{IGF2},
\textit{PHLDA2}), imprinted genes (\textit{DLK1}, \textit{PHLDA2}),
and \textit{CSH1}, a gene that is a member of a ruminant specific
expansion of placental prolactins and not directly related to the gene
of the same name in hominids \ref{fig:prolactin_evolution_tree}. Other
ruminant-specific placental prolactins also show high expression in
the cow placenta including %
\Sexpr{gene.expression("bos taurus","PRP1",complete=TRUE)}, %
\Sexpr{gene.expression("bos taurus","PRP5",complete=TRUE)}, %
\Sexpr{gene.expression("bos taurus","PRP6",complete=TRUE)}, %
\Sexpr{gene.expression("bos taurus","PRP8",complete=TRUE)}, and %
\Sexpr{gene.expression("bos taurus","PRP7",complete=TRUE)}, %
with a total of 14 with $\mathrm{FPKM} \ge 1$.
%
\textit{SSLP1} (\textit{SOLD1} in \sheep{}), a gene expressed in the
cotyledonary villi of the ruminiant placenta and involved in cell
proliferation
\cite{Ushizawa.ea2009:Characterizationandexpressionanalysis}, also
shows evidence of high expression %
(\Sexpr{gene.expression("bos taurus","SSLP1")}). %
There is some placental expression of the estrogen synthase gene
(\textit{CYP19A1}) in cows %
(\Sexpr{gene.expression("bos taurus","CYP19A1")}), and
comparatively greater expression of \textit{HSD3B} %
(\Sexpr{gene.expression("bos taurus","HSD3B")}).

\subsubsection{\pig Placenta Transcriptome}

\begin{table}[h]
  \footnotesize
<<top.50.pig.expressed.genes,echo=FALSE,results="asis">>=
output.top.genes.table(top.genes.table(species.name="sus scrofa"))
@ 
\caption{\tableexplanation{\pig}{tab:all_sus_scrofa_placenta}}
\label{tab:top50_pig_placenta}
\end{table}

The 50 non-housekeeping genes with highest expression (by \ac{FPKM})
in the term placenta of \textit{Sus scrofa} are shown in
\cref{tab:top50_pig_placenta} with a list of all highly expressed
genes in Supplemental \cref{tab:all_sus_scrofa_placenta}. The top two
expressed transcripts,
\Sexpr{gene.expression("sus_scrofa","ENSSSCG00000025162",complete=TRUE)}
and
\Sexpr{gene.expression("sus_scrofa","ENSSSCG00000021356",complete=TRUE)},
are miRNA which correspond to the \cow miRNA
\href{http://www.mirbase.org/cgi-bin/mirna_entry.pl?acc=MI0013080}{bta-mir-2904-1}
and
\href{http://www.mirbase.org/cgi-bin/mirna_entry.pl?acc=MIMAT0013845}{bta-mir-2887-1},
respectively, both of which are also expressed in \cow ovary
\cite{Hossain.ea2009:Identificationandcharacterizationof}, although
neither of which have an obvious target gene on the basis of
\textit{in silico} analysis
\cite{Agarwal.ea2015:PredictingeffectivemicroRNAtarget}. Other highly
expressed genes include members of the insulin growth factor family
(\Sexpr{gene.expression("sus_scrofa","IGF2",complete=TRUE)} and
\Sexpr{gene.expression("sus_scrofa","IGFBP3",complete=TRUE)}), members
of annexin complexes
(\Sexpr{gene.expression("sus_scrofa","S100A1",complete=TRUE)},
\Sexpr{gene.expression("sus_scrofa","ANXA2",complete=TRUE)},
\Sexpr{gene.expression("sus_scrofa","S100A12",complete=TRUE)}, and
\Sexpr{gene.expression("sus_scrofa","S100A10",complete=TRUE)}), the
galectin \Sexpr{gene.expression("sus_scrofa","LGALS3",complete=TRUE)}
and multiple cytoskeleton and extracellular matrix proteins
(\Sexpr{gene.expression("sus_scrofa","KRT19",complete=TRUE)},
\Sexpr{gene.expression("sus_scrofa","COL1A1",complete=TRUE)},
\Sexpr{gene.expression("sus_scrofa","BETA-ACTIN",complete=TRUE)},
\Sexpr{gene.expression("sus_scrofa","KRT9",complete=TRUE)},
\Sexpr{gene.expression("sus_scrofa","CLDN4",complete=TRUE)}, and
\Sexpr{gene.expression("sus_scrofa","KRT9",complete=TRUE)}).

\subsubsection{\horse Placenta Transcriptome}

\begin{table}[h]
  \footnotesize
<<top.50.horse.expressed.genes,echo=FALSE,results="asis">>=
top.horse.genes <- top.genes.table(species.name="equus caballus")
print(xtable(top.horse.genes),
      booktabs=TRUE,
      floating=FALSE,
      sanitize.text.function=genes.table.sanitize,include.rownames=FALSE)
setkey(top.horse.genes,"Gene symbol")
@ 
\caption{\tableexplanation{\horse}{tab:all_equus_caballus_placenta}}
\label{tab:top50_horse_placenta}
\end{table}

The 50 non-housekeeping genes with highest expression (by \ac{FPKM})
in the term placenta of \textit{Equus Caballus} are shown in
\cref{tab:top50_horse_placenta} with a list of all highly expressed
genes in Supplemental \cref{tab:all_equus_caballus_placenta}. The gene
with the highest expression was the relaxin,
\Sexpr{gene.expression("equus_caballus","RLN",complete=TRUE)},
followed by two miRNA
(\Sexpr{gene.expression("equus_caballus","ENSECAG00000025328",complete=TRUE)}
and
\Sexpr{gene.expression("equus_caballus","ENSECAG00000027056",complete=TRUE)})
with no known \textit{in silico} targets
\cite{Agarwal.ea2015:PredictingeffectivemicroRNAtarget}, the thymosin
\Sexpr{gene.expression("equus_caballus","TMSB4",complete=TRUE)},
members of the insulin growth factor family
(\Sexpr{gene.expression("equus_caballus","IGF2",complete=TRUE)} and
\Sexpr{gene.expression("equus_caballus","IGFBP3",complete=TRUE)}) and
multiple members of annexin complexes
(\Sexpr{gene.expression("equus_caballus","S100A6",complete=TRUE)},
\Sexpr{gene.expression("equus_caballus","S100A1",complete=TRUE)}, and
\Sexpr{gene.expression("equus_caballus","ANXA2",complete=TRUE)}),
similar to the pattern of expression seen in \pig and \dog.

\subsubsection{\sheep Placenta Transcriptome}

\begin{table}[h]
  \footnotesize
<<top.50.sheep.expressed.genes,echo=FALSE,results="asis">>=
top.sheep.genes <- top.genes.table(species.name="ovis aries",n=200)
top.sheep.genes <-
    top.sheep.genes[!grepl("\\{(7SK|.+_([SM]RP|nuc)|SNOR[AD]\\d+|U\\d+|HIST.*)\\}",
                           `Gene symbol`) &
                    !grepl("\\((7SK|.+_([SM]RP|nuc)|SNOR[AD]\\d+|U\\d+|HIST.*)\\)",
                           `Human Symbol`)
                   ,][1:50,]
output.top.genes.table(top.sheep.genes)
@ 
\caption{\tableexplanation{\sheep}{tab:all_ovis_aries_placenta}}
\label{tab:top50_sheep_placenta}
\end{table}

The 50 non-housekeeping genes with highest expression (by \ac{FPKM})
in the term placenta of \sheep are shown in
\cref{tab:top50_sheep_placenta} with a list of all highly expressed
genes in Supplemental \cref{tab:all_ovis_aries_placenta}. The complete
list of highly expressed genes includes many snRNAs and rRNAs likely
due to the RNA extraction and library preparation procedure utilized
for this sample. Ignoring these, notable highly expressed genes
include members of annexin complexes
(\Sexpr{gene.expression("ovis_aries","S100G",complete=TRUE)},
\Sexpr{gene.expression("ovis_aries","ANXA2",complete=TRUE)}),
hemoglobin subunits
(\Sexpr{gene.expression("ovis_aries","ENSOARG00000010944",complete=TRUE)}),
an ortholog of \textit{SSLP1},
\Sexpr{gene.expression("ovis_aries","SORD1",complete=TRUE)}, two
cathespins (\Sexpr{gene.expression("ovis_aries","CTSL",complete=TRUE)}
and \Sexpr{gene.expression("ovis_aries","CTSB",complete=TRUE)}), and
placentally expressed transcript 1
\Sexpr{gene.expression("ovis_aries","PLET1",complete=TRUE)} which may
play a role in cellular migration
\cite{Raymond.ea2010:Expressionoforphanprotein}.


\subsection{Atlantogenata}
\subsubsection{\elephant Placenta Transcriptome}

\begin{table}
\footnotesize
<<top.50.elephant.expressed.genes,echo=FALSE,results="asis">>=
output.top.genes.table(top.genes.table(species.name="loxodonta africana"))
@ 
\caption{\tableexplanation{\elephant}{tab:top50_elephant_placenta}}
\label{tab:top50_elephant_placenta}
\end{table}


The top 50 expressed genes as measured by \ac{FPKM} in the term
placenta of \elephant are shown in \cref{tab:top50_elephant_placenta}
with a list of all genes in
\cref{tab:all_loxodonta_africana_placenta}. The gene with the highest
expression was the natriuretic peptide precursor B
(\Sexpr{gene.expression("loxodonta_africana","NPPB",complete=TRUE)}).
Two thymosins,
\Sexpr{gene.expression("loxodonta_africana","TMSB10",complete=TRUE)}
and
\Sexpr{gene.expression("loxodonta_africana","TMSB4Y",complete=TRUE)}
were also also was highly expressed. The miRNA mir-675
(\Sexpr{gene.expression("loxodonta_africana","ENSLAFG00000027406",complete=TRUE)}).
which is imbedded in the first exon of the maternally imprinted long
non-coding RNA gene \textit{H19}
\cite{Keniry.ea2012:H19lincRNAisdevelopmental}, a gene which is
currently un-annotated in \elephant{}. \DLA{Probably consider fixing
  the unannotation of H19; it seems pretty obvious to me, though it
  would require fixing the GTF file for elephant with the right
  \href{http://useast.ensembl.org/Loxodonta_africana/Transcript/Summary?db=core;pt=GENSCAN00000034833;r=scaffold_145:855987-899506}{genscan
    prediction} for this region} Five uncharacterized genes are
expressed to a high degree in the \elephant placenta
(\cref{tab:top50_elephant_placenta}), three of which
(\Sexpr{gene.expression("loxodonta_africana","ENSLAFG00000028493",complete=TRUE)},
\Sexpr{gene.expression("loxodonta_africana","ENSLAFG00000032011",complete=TRUE)},
\Sexpr{gene.expression("loxodonta_africana","ENSLAFG00000031933",complete=TRUE)})
are members of an \elephant{}-specific expansion of 11 genes closely
related to the thymus-specific serine protease gene \textit{PRSS16}
\cref{fig:pag2_prss16_ctsl_psg_evolution_tree}~C The other 8 \textit{PRSS16}-like
genes are also expressed to a lesser degree
(\Sexpr{gene.expression("loxodonta_africana","ENSLAFG00000026505",complete=TRUE)}).
Two other novel genes with high expression are related to the gene
\textit{SSLP1}
(\Sexpr{gene.expression("loxodonta_africana","ENSLAFG00000032659",complete=TRUE)}
and
\Sexpr{gene.expression("loxodonta_africana","ENSLAFG00000029551",complete=TRUE)},
\cref{fig:sslp1_evolution_tree}), which is also expressed in \cow and
\dog placentas (\cref{tab:top50_dog_placenta,tab:top50_cow_placenta}).
Placental expression of the hormones relaxin (\textit{RLN}, %
\Sexpr{gene.expression("loxodonta_africana","ENSLAFG00000036090")})
and prolactin (\textit{PRL}) are exceptionally high in the elephant
when compared to other mammals
(\cref{fig:rln2_evolution_tree,fig:prolactin_evolution_tree}). There
is little to no placental expression of the estrogen synthase gene,
\textit{CYP19A1}
(\Sexpr{gene.expression("loxodonta_africana","CYP19A1")}) and the
enzyme that produces progesterone HSD3B1
(\Sexpr{gene.expression("loxodonta_africana","HSD3B")}). Other genes
with high expression in the elephant placenta include genes involved
in oxidative stress
(\Sexpr{gene.expression("loxodonta_africana","TXN",complete=TRUE)}),
growth
(\Sexpr{gene.expression("loxodonta_africana","IGF2",complete=TRUE)})
and
\Sexpr{gene.expression("loxodonta_africana","PAEP",complete=TRUE)}, a
gene usually expressed in the endometrium of the uterus in humans
\cite{Rutanen.ea1987:Progesterone-associatedproteinsPP12and}.

\subsubsection{\armadillo Placenta Transcriptome}

\begin{table}
\footnotesize
<<top.50.armadillo.expressed.genes,echo=FALSE,results="asis">>=
output.top.genes.table(top.genes.table(species.name="dasypus novemcinctus"))
@ 
\caption{\tableexplanation{\armadillo}{tab:all_dasypus_novemcinctus_placenta}}
\label{tab:top50_armadillo_placenta}
\end{table}

The top 50 expressed genes in the term placenta of \armadillo are
shown in \cref{tab:top50_armadillo_placenta} with a list of all genes
with $\mathrm{FPKM}\ge 50$ in Supplemental
\cref{tab:all_dasypus_novemcinctus_placenta}. The gene with the
highest expression is
\Sexpr{gene.expression("dasypus_novemcinctus","ENSDNOG00000045029",complete=TRUE)}
which is most similar to the \mouse \textit{Pga5} is expanded in
\opossum into three members which are expressed in placenta (the other
two being
\Sexpr{gene.expression("dasypus_novemcinctus","ENSDNOG00000033885",complete=TRUE)}
and
\Sexpr{gene.expression("dasypus_novemcinctus","ENSDNOG00000040431",complete=TRUE)}),
though the expansion is less than that seen \cow and \sheep
(\cref{fig:pag2_prss16_ctsl_psg_evolution_tree}~D). There is also high expression of
hemoglobin subunit beta,
\Sexpr{gene.expression("dasypus_novemcinctus","HBB",complete=TRUE)},
an un-annotated gene with high similarity to \human and \cow
\textit{OVOS2}
\Sexpr{gene.expression("dasypus_novemcinctus","ENSDNOG00000033320",complete=TRUE)}
(Supplemental \cref{fig:ovos2_evolution_tree}), the growth factors
\Sexpr{gene.expression("dasypus_novemcinctus","IGF2",complete=TRUE)}
and
\Sexpr{gene.expression("dasypus_novemcinctus","IGFBP4",complete=TRUE)},
and the thymosin
\Sexpr{gene.expression("dasypus_novemcinctus","TMSB4X",complete=TRUE)}.
As has been seen previously, multiple components of the annexin
complex are also highly expressed, including
\Sexpr{gene.expression("dasypus_novemcinctus","ENSDNOG00000045634",complete=TRUE)}
(\textit{ANXA8L1}),
\Sexpr{gene.expression("dasypus_novemcinctus","ENSDNOG00000017399",complete=TRUE)}
(\textit{S100B}),
\Sexpr{gene.expression("dasypus_novemcinctus","ENSDNOG00000048504",complete=TRUE)}
(\textit{S100A10}), and
\Sexpr{gene.expression("dasypus_novemcinctus","ENSDNOG00000004237",complete=TRUE)}
(\textit{ANXA2}).




\subsection{Metatherians}

\subsubsection{\opossum Placenta Transcriptome}

\begin{table}
  \footnotesize
<<top.50.opossum.expressed.genes,echo=FALSE,results="asis">>=
output.top.genes.table(top.genes.table(species.name="monodelphis domestica"))
@ 
\caption{\tableexplanation{\opossum}{tab:all_monodelphis_domestica_placenta}}
\label{tab:top50_opossum_placenta}
\end{table}

The top 50 expressed genes as measured by \ac{FPKM} in the term
placenta of the \opossum are shown in
\cref{tab:top50_opossum_placenta} with a list of all genes in
Supplemental \cref{tab:all_monodelphis_domestica_placenta}. The gene with the
second highest expression is superoxide dismutase 3, extracellular
(\Sexpr{gene.expression("monodelphis_domestica","SOD3",complete=TRUE)}),
which functions as an antioxidant. The gene with the second highest
expression is the hemoglobin subunit epsilon
(\Sexpr{gene.expression("monodelphis_domestica","ENSMODG00000011325",complete=TRUE)}),
which is expressed at the embryonic stage of development in
marsupials, as in other mammals
\cite{Holland.ea1997:Characterizationofembryonicglobin}. \textit{HBE}
subunits link up with protein products of two hemoglobin, zeta
(\textit{HBZ}) genes, which are also expressed to a high degree in the
opossum placenta
(\Sexpr{gene.expression("monodelphis_domestica","ENSMODG00000016635",complete=TRUE)}
and
\Sexpr{gene.expression("monodelphis_domestica","ENSMODG00000016631",complete=TRUE)}).
In addition, there is high expression of the gene hemoglobin, omega
(\textit{HBO},
\Sexpr{gene.expression("monodelphis_domestica","ENSMODG00000016637",complete=TRUE)}),
% this is LOC_1000016060 which I figured out by tblastn'ing AAK11486.1
% against Monodelphis domestica
a hemoglobin found only in marsupials and monotremes
\cite{Opazo.ea2008:Genomicevidenceforindependent} (Supplemental
\cref{fig:hbe1_evolution_tree}). The hemoglobin gene \textit{HBA} is
also expressed in the placenta
(\Sexpr{gene.expression("monodelphis_domestica","ENSMODG00000016633",complete=TRUE)}).
Two cystatin genes
(\Sexpr{gene.expression("monodelphis_domestica","ENSMODG00000018295",complete=TRUE)}
and
\Sexpr{gene.expression("monodelphis_domestica","ENSMODG00000018296",complete=TRUE)})
are expressed to a high degree, along with one \textit{CTSL} related
gene
(\Sexpr{gene.expression("monodelphis_domestica","ENSMODG00000002844",complete=TRUE)},
\cref{fig:pag2_prss16_ctsl_psg_evolution_tree}~B), which are known to interact with
cystatins \cite{Turk.ea2001:Lysosomalcysteineproteasesfacts}. The
uncharacterized gene \textit{LOC10001195}
(\Sexpr{gene.expression("monodelphis_domestica","ENSMODG00000006547",complete=TRUE)})
also shows high expression in \opossum placenta. The gene
\textit{WFDC2}, a member of the whey acidic protein (WAP) family is
also expressed to a high degree in \opossum placenta
(\Sexpr{gene.expression("monodelphis_domestica","WFDC2",complete=TRUE)}).
Other genes with high expression include members of the annexin
complex
(\Sexpr{gene.expression("monodelphis_domestica","S100A6",complete=TRUE)},
\Sexpr{gene.expression("monodelphis_domestica","ANXA2",complete=TRUE)}).
In addition, there was little production of CYP19A1 and HSD3B1
(\Sexpr{gene.expression("monodelphis_domestica","CYP19A1",complete=TRUE)}
and
\Sexpr{gene.expression("monodelphis_domestica","ENSMODG00000005385",complete=TRUE)},
respectively).



\subsection{Core Placenta Transcriptome}


\begin{figure}
<<core.placenta.transcriptome,fig.width=8,fig.height=12,out.height="0.8\\textheight",echo=FALSE>>=
print(ggplot(core.placenta.transcriptome.long[!grepl("(min|median|max)_expression",Species),],
             aes(y=human_name,x=capfirst(Species)))
      + geom_raster(aes(fill=log10(FPKM)))
      + scale_x_discrete(expand=c(0,0))
      + scale_y_discrete(expand=c(0,0))
      + scale_fill_gradient(low = grad.def[["low"]],
                            high = grad.def[["high"]],
                            space = grad.def[["space"]],
                            na.value = grad.def[["na.value"]],
                            limits=c(0,4),
                            guide=guide_colorbar(title=expression(log[10](FPKM)))
                            )
      + ylab("Human Ortholog Name")
      + xlab("Species")
      + theme_grey()
      + theme(legend.position="top",
              axis.ticks = element_blank(), 
              axis.text.x = element_text(angle = 340, hjust = 0.1, vjust=1,face="italic"))
      )
@ 
\caption{Core Placenta Transcriptome.
  \Sexpr{core.placenta.transcriptome.long[,length(unique(human_name))]}
  genes expressed (\ac{FPKM}$ \ge 10$) in the placenta of all organisms
  studied with housekeeping
  genes \cite{Eisenberg.ea2013:Humanhousekeepinggenesrevisited}
  removed.}
\label{fig:core_placenta_transcriptome}
\end{figure}

\begin{table}
\footnotesize
<<go.analysis,warning=FALSE,message=FALSE,echo=FALSE,results="asis">>=
placenta.go.results <- rbindlist(list(head(n=20,placenta_go_results_mf[classic.fdr < 0.05,]),
                             head(n=20,placenta_go_results_bp[classic.fdr < 0.05,])))
placenta.go.results <- placenta.go.results[,Annotated:=NULL]
setnames(placenta.go.results,"Significant","# Core")
setnames(placenta.go.results,"classic","p value")
setnames(placenta.go.results,"classic.fdr","FDR")
setnames(placenta.go.results,"GO.ID","GO ID")
setnames(placenta.go.results,"Term","GO Term")
print(xtable(placenta.go.results,
             digits=2,
             display=c("d","s","s","d","f","e","e","s")),
      include.rownames=FALSE,
      booktabs=TRUE,
      floating=FALSE
      )
@ 
\caption{\ac{GO} terms most significantly enriched
  (\Sexpr{placenta.go.results[,sum(Ontology=="MF")]} MF,
  \Sexpr{placenta.go.results[,sum(Ontology=="BP")]} BP)
  in the core placenta transcriptome as compared to all genes
  with 1:1 human orthologs in all species. â\# Coreâ is the
  number of genes in the core transcriptome for each \ac{GO} term;
  âExpectedâ is the number of genes expected in the core
  transcriptome for each term; âp-valueâ are the odds of this
  occurring by chance calculated by the Fisher exact test for
  each term; âFDRâ is the false discovery rate as estimated by
  the Benjamini and Hochberg procedure \cite{Benjamini1995:bh_procedure}.}
\label{tab:core_placenta_go_terms}
\end{table}

The
\Sexpr{core.placenta.transcriptome.long[,length(unique(human_name))]}
genes comprising the core placenta transcriptome are shown in
\cref{fig:core_placenta_transcriptome} ordered by their median
transcription level in \ac{FPKM}. We considered a gene to be core to
the placenta transcriptome if a 1:1 ortholog of that gene was
expressed with $\mathrm{FPKM} \ge 10$ in all species studied and was
not a human housekeeping gene according to
\citet{Eisenberg.ea2013:Humanhousekeepinggenesrevisited}.
Supplemental \cref{fig:core_placenta_transcriptome_shape} shows the number of genes
which would be included in the core placenta set if the \ac{FPKM}
cutoff was set differently.
%
Multiple components of annexin complexes are included in the core
transcriptome, including \textit{ANXA2}, \textit{ANXA1},
\textit{S100A11}, \textit{S100A10}. ANXA1 is involved in the
resolution of the inflammation response through inhibition of
phospholipase A2 activity and signaling through the formyl peptide
receptor family, and has previously been hypothesized to be important
for the maintenance of the anti-inflammatory during pregnancy
\cite{Hutchinson.ea2011:Molecularregulatorsofresolution}. ANXA2 is
known to interact with S100A10, S100A6, S100A11, and S100A4
\cite{Liu.ea2015:AnnexinA2complexeswith}, and there is evidence for
its interaction with S100P as well; it is involved in cell-cell
interaction as well as vesicle trafficking and von Willibrand Factor
secretion. The expression regulation of another core gene,
\textit{TIMP3}, has been implicated in pre-eclampsia
\cite{Ma.ea2014:Down-regulationofTIMP3leads,Xiang.ea2013:PromoterhypomethylationofTIMP3}.
\Cref{tab:core_placenta_go_terms} lists the significant enrichment of
\ac{GO} terms in the core placenta transcriptome; notable terms
include those involved in cytoskeletal structure, protein transport,
membrane protein structure, and inflammation resolution.


% SPINT2 -- Serine Peptidase inhibitor

% SLC3A2 -- this is probably actually a housekeeping gene; it's
% involved in amino acid transport

% SQSTM1 -- NFÎºB signaling (immune supression?)

% KRTCAP2  N-glycosylation (probably housekeeping)

% NRDG1 -- interacts with HSP90

% SDC4 -- proteoglycan (syndecan) cell adhesion

% LAMC1 -- laminin

% LUM -- proteoglycan

% VDAC1 -- mitochondrial gene?

% F11R -- tight junctions

% SERPINH1? with TIMP3 
% http://www.proteinatlas.org/ENSG00000149257-SERPINH1/tissue/placenta


\subsubsection{\ac{GSE} Analyses of Individual Transcriptomes}

In order to identify major functional categories of genes expressed in
the individual transcriptomes, we examined \ac{GSE} in 1:1 human
orthologs which had \ac{FPKM} values in the top 1\% of expressed genes
(Supplemental %
\cref{tab:go_terms_enriched_top_one_dasypus_novemcinctus,tab:go_terms_enriched_top_one_equus_caballus,tab:go_terms_enriched_top_one_homo_sapiens,tab:go_terms_enriched_top_one_loxodonta_africana,tab:go_terms_enriched_top_one_monodelphis_domestica,tab:go_terms_enriched_top_one_mus_musculus,tab:go_terms_enriched_top_one_ovis_aries,tab:go_terms_enriched_top_one_pan_paniscus,tab:go_terms_enriched_top_one_sus_scrofa,tab:go_terms_enriched_top_one_ateles_fusciceps,tab:go_terms_enriched_top_one_nannospalax_galili,tab:go_terms_enriched_top_one_spalax_carmeli,tab:go_terms_enriched_top_one_bos_taurus,tab:go_terms_enriched_top_one_canis_familiaris}) using topGO \cite{topGO:2.20.0}. %
%
Significant enrichment in genes ($\mathrm{FDR}\le0.05$) in the top 5\%
of \ac{FPKM} values in at least 12 of the
\Sexpr{length(species.names)} species included clusters such as
membrane organization , locomotion, tissue development, extracellular
matrix organization, regulation of peptidase activity, and various
cell death related pathways among others
(\cref{tab:go_terms_all_species}). If we instead examine the top 100
genes by \ac{FPKM} in each of the \Sexpr{length(species.names)}
species (Supplemental %
\cref{tab:go_terms_enriched_dasypus_novemcinctus,tab:go_terms_enriched_equus_caballus,tab:go_terms_enriched_homo_sapiens,tab:go_terms_enriched_loxodonta_africana,tab:go_terms_enriched_monodelphis_domestica,tab:go_terms_enriched_mus_musculus,tab:go_terms_enriched_ovis_aries,tab:go_terms_enriched_pan_paniscus,tab:go_terms_enriched_sus_scrofa,tab:go_terms_enriched_ateles_fusciceps,tab:go_terms_enriched_nannospalax_galili,tab:go_terms_enriched_spalax_carmeli,tab:go_terms_enriched_bos_taurus,tab:go_terms_enriched_canis_familiaris})
notable enrichments include extracellular matrix, female pregnancy,
integrin and collagen-related processes, and various cell death
related processes.

% \textit{Ateles fuciceps} shows enrichment in extracellular
% matrix and structural molecular activity (Supplemental
% \cref{tab:go_terms_enriched_ateles_fusciceps}). \textit{Homo sapiens}
% placentas are enriched in female pregnancy, estrogen biosynthesis,
% reproductive process, hormone aactivity, oxygen transporter activity,
% and haptoglobin binding (Supplemental
% \cref{tab:go_terms_enriched_homo_sapiens}). \textit{Pan paniscus}
% placentas are enriched in clusters related to stress, apoptosis, and
% response to abiotic stimuli. \textit{Mus musculus} placentas are
% enriched in cell junction, vesicle transport, in utero embryonic
% development, and many cytoskeletal processes. \textit{Nannospalax
%   galili} placentas are enriched in extracellular matrix organization,
% integrin binding, and cell death.
% 
% Cow shows only four
% significantly enriched clusters; however many genes highly expressed
% in the cow are ruminant-specific placenta genes and are not yet
% annotated to specific GO categories. Dog shows enrichment in genes
% associated with RNA, as well as homeostasis, glycoproteins, lyosome,
% and calcium binding. Elephant shows enrichment in genes associated
% with RNA, coagulation and calcium binding. Opossum shows enrichment in
% genes involved in apoptosis, oxidative stress, mitochondrion, hormone
% metabolism, and lipids.


\begin{table}
  \footnotesize
<<go.analysis.all.tissue,echo=FALSE,results="asis">>=
go.analysis.wide <- 
    data.table(dcast(data=go.results,
                     formula=GO.ID+type+Term+Ontology~species,
                     value.var="FDR"))
go.analysis.wide <- 
    go.analysis.wide[,`n(FDR â¤ 0.05)`:=apply(go.analysis.wide[,-(1:4),with=FALSE],1,
                                         function(x){sum(as.numeric(as.character(x)) <= 0.05)})]
go.analysis.wide <- 
    go.analysis.wide[,`max(FDR â¤ 0.05)`:=apply(go.analysis.wide[,-(1:4),with=FALSE],1,
                                           function(x){if (!any(is.finite(x) & x<=0.05)) {
                                                           return(NA)
                                                       } else {
                                                           max(x[is.finite(x) & x<=0.05],na.rm=TRUE)
                                                       }})]
go.analysis.wide.table <- 
    go.analysis.wide[!is.na(`n(FDR â¤ 0.05)`) & `n(FDR â¤ 0.05)` >= 12 &
                         grepl("Top 1% No Housekeeping",type),
                     list(GO.ID,Term,Ontology,`n(FDR â¤ 0.05)`,`max(FDR â¤ 0.05)`)]
go.analysis.wide.table <- 
    go.analysis.wide.table[order(Ontology,-`n(FDR â¤ 0.05)`,`max(FDR â¤ 0.05)`)]
setnames(go.analysis.wide.table,"GO.ID","GO ID")

print(xtable(go.analysis.wide.table,
             display=c("d","s","s","s","d","e"),
             align=  c("c","c","c","c","c","c")
             ),
      math.style.exponents=TRUE,
      include.rownames=FALSE,
      booktabs=TRUE,
      floating=FALSE
      )

@ 
\caption{GO terms with significant enrichment ($\mathrm{FDR}\le0.05$)
  in 12 or more of the species analyzed from genes in the top 1\% of
  expression. $n(\mathrm{FDR}\le0.05)$ is the number of species with
  significant enrichment ($\mathrm{FDR}\le 0.05$) in the GO term
  listed. $max(\mathrm{FDR}\le0.05)$ is the maximum FDR value in all
  species with an FDR of less than $0.05$.}
\label{tab:go_terms_all_species}
\end{table}


% Not sure if this is actually worthwhile; I've mentioned some of this
% already for some of the lineages

% \subsection{Comparative Analysis of Gene Expression}
% 
% We performed a comparative analysis of gene expression by
% reconstructing the evolution of expression values and detecting
% significant shifts in expression along specific lineages. Likelihood
% values for each model for each of the X genes reconstructed, as well
% as likelihood ratio tests, P- and Q-values (after correcting for
% multiple testing) are shown in Supplemental Table X. We identified
% between 4 (Boreoeutheria) and 550 (Mouse) significant shifts (Q <
% 0.01) in expression on specific lineages (Table 10). The top 10 genes
% with significant shifts (if applicable) per lineage are shown in Table
% 10 and all genes for each branch are shown in Supplemental Tables
% X-XX. Genes that showed shifts on the human lineage were
% overwhelmingly identified as placenta-specific by the
% tissue-specificity index (TSI), with 56 out of 279 genes (20\%) with
% both TSI>0.75 and placenta as one the top two tissues with the highest
% expression. Unsurpisingly, this was not the case with other lineages,
% since we used expression values from a human-derived microarray to
% generate TSI values for each gene. However, this does underscore the
% dynamic nature of expression patterns in the placenta and indeed all
% tissues across mammals.
% 
% We then performed DAVID functional annotation clustering analyses on
% each list of genes from each lineage. The top three clusters for each
% lineage are shown in Table 10 and a full list of clusters in
% Supplemental Table X. The top enrichment cluster in the human lineage
% is blood vessel development, followed by extracellular matrix and
% metal ion binding (Table 10). Other significant clusters include those
% related to cell communication/signaling, cytokines, defense response,
% tryptophan metabolism, apoptosis, and skeletal system development
% (Supplemental Table X). The bonobo lineage was enriched in only three
% clusters: apoptosis, Toll-like receptor signalling and apoptosis. The
% mouse displayed enrichment in clusters involved in epigenetic
% modifications such as methyltransferase activity and histone
% modification, as well embryonic morphogeneisis, lipid biosynthesis,
% among others (Supplemental Table X). The top enriched cluster in the
% cow lineage was the response to vitamin A, as well as many clusters
% related to the immune system. The dog showed enrichment in skeletal
% system development, as well as lipid metabolic processes and blood
% circulation. The elephant lineage had only one enriched cluster:
% carbohydrate binding. Genes that showed enrichment on the opossum
% lineage (essentially representing the difference between placental
% mammals and marsupials) were clustered in many categories not seen in
% the other lineages, including clusters related to the mitochondria,
% urea cycle, peroxisome, glycolysis, and transport across the cell
% membrane (Table 10 and Supplemental Table X).

\clearpage

\section{Discussion}

\subsection{Evolution of Placental Hormones}

Reproductive hormones are central to the regulation of pregnancy and
their source, timing, and quantity can vary widely from species to
species \cite{Carter2012:Evolutionofplacentalfunction}. Expression and synthesis of hormones is
perhaps the most well-documented physiological process in the placenta
in many species. Here we quantified the extreme disparity in genes
with hormone products or enzymatic products that directly produce
hormones in multiple mammalian species (\cref{fig:hormone_genes}).

\begin{figure}
<<hormone.genes.plot,echo=FALSE,fig.width=8,fig.height=8,warning=FALSE,message=FALSE>>=

gene.expression.nozero <- function(genes) {
    temp <-
        data.matrix.dt(dcast(combined.fpkm[human_name %in% genes,],
                             human_name~species,
                             value.var="FPKM",
                             fun.aggregate=function(x){x[1]}))
    temp <-
        temp[apply(temp,1,function(x){sum(x,na.rm=TRUE)})>5,]
    return(temp)
}

### hormone genes
hormone.genes <- c("CGA","CGB","CGB1","CGB2","CGB5","CGB8","CSH1","CSH2","CSHL1","CYP19A1",
                   "GH2","GHRH","HSD3B1","HSD3B2","HSD17B2","HSD17B1","NPPB","NPPC","PRL","RLN1")
hormone.expression.nozero <-
    gene.expression.nozero(hormone.genes)

hor.exp.long <- melt(log10(hormone.expression.nozero+1)[rev(1:nrow(hormone.expression.nozero)),])
colnames(hor.exp.long)[1:3] <- c("gene","species","fpkm")
hor.exp.long <- hor.exp.long[hor.exp.long$gene!="IGF2",]

### galectin genes
galectin.genes <- unique(grep("^LGALS",combined.fpkm[,human_name],value=TRUE))

galectin.expression.nozero <-
    gene.expression.nozero(galectin.genes)

gal.exp.long <- melt(log10(galectin.expression.nozero+1)[rev(1:nrow(galectin.expression.nozero)),])
colnames(gal.exp.long)[1:3] <- c("gene","species","fpkm")

### IGF related genes

igfrelated.genes <- unique(grep("^IGF",combined.fpkm[,human_name],value=TRUE))

igfrelated.expression.nozero <-
    gene.expression.nozero(igfrelated.genes)

igf.exp.long <- melt(log10(igfrelated.expression.nozero+1)[rev(1:nrow(igfrelated.expression.nozero)),])
colnames(igf.exp.long)[1:3] <- c("gene","species","fpkm")

igf.exp.long$type <- "IGF Related"
gal.exp.long$type <- "Galectin"
hor.exp.long$type <- "Hormone"

### scalerange <- range(c(igf.exp.long$fpkm,gal.exp.long$fpkm,hor.exp.long$fpkm),na.rm=TRUE)
### gradientends <- scalerange + rep(c(0,100,200), each=2)
### colorends <- c("white", "red", "white", "green", "white", "blue")
### 
### gal.exp.long$fpkm <- gal.exp.long$fpkm+100
### hor.exp.long$fpkm <- hor.exp.long$fpkm+200

combined.long <-
    rbind(igf.exp.long,
          gal.exp.long,
          hor.exp.long)
combined.long$species <-
    factor(gsub("^([^ ])[^ ]* +([^ ]*)","\\1. \\2",capfirst(combined.long$species)),
           levels=gsub("^([^ ])[^ ]* +([^ ]*)","\\1. \\2",capfirst(names(species.names))))

common.plot.options <- 
    list(geom_tile(aes(fill = fpkm), colour = "white"),
##         scale_fill_gradientn(colours = colorends, values = rescale(gradientends)),
         scale_fill_gradient(low = grad.def[["low"]],
                             high = grad.def[["high"]],
                             space = grad.def[["space"]],
                             na.value = grad.def[["na.value"]],
                             guide=guide_colorbar(title=expression(log[10](FPKM)))),
         scale_x_discrete("", expand = c(0, 0)),
         scale_y_discrete("", expand = c(0, 0)),
         theme_grey(base_size = 9),
         theme(legend.position = "top",
               axis.ticks = element_blank(), 
               axis.text.x = element_text(angle = 300, hjust = 0, vjust=1,face="italic")),
         facet_grid(type~.,scale="free_y",space="free_y")
         )
print(ggplot(combined.long, aes(y=gene, x=species))
      + common.plot.options
      )
@ 
\caption{Expression of Hormone Genes, Galectins, and IGF related genes}
\label{fig:hormone_genes}
\end{figure}

\subsubsection{Estrogen and Progesterone}

In both ruminants and primates, the placenta is responsible for the
production of estrogen and progesterone. High relative expression of
\textit{CYP19A1} (Supplemental \cref{fig:estrogen_evolution_tree}),
the enzyme that synthesizes estrogen from androstenedione is seen in
\human, \bonobo, \cow, and \horse, though not in \sheep, \smonkey. %
%
\textit{HSD3B1}, one of two enzymes that synthesizes progesterone from
pregnalone
(\cite{Simard.ea2005:Molecularbiologyof3beta-hydroxysteroid},
Supplemental \cref{fig:progesterone_evolution_tree}) is also highly
expressed in \human, \bonobo, \cow, \horse, and \dog, but is not
highly expressed in \smonkey. The diversity of \textit{CYP19A1} and
\textit{HSD3B1} expression levels in in primates and ungulates seen
here points to the need for more complete sampling of placentas at
different timepoints and in different clades.


\subsubsection{Prolactin}

The elephant is strikingly the only mammal analyzed that shows high
expression of the primarily pituitary hormone prolactin (\textit{PRL})
in the placenta, although we see high expression of genes which
resulted from duplications of \textit{PRL} in the \mouse, \carmeli,
and \cow (Supplemental \cref{fig:prolactin_evolution_tree}). All afrotherians yet
analyzed contain only one prolactin gene, and prolactin has undergone
adaptive evolution on the \elephant lineage
\cite{Wallis2009:ProlactininAfrotheriacharacterization}.
\citet{Yamamoto.ea2011:PlacentationinAfricanelephant} noted the
presence of a prolactin in the trophoblast of the elephant placenta
using immunohistological staining and human derived antibodies. A
previous analysis of \elephant placenta also detected the expression
of prolactin \cite{Hou.ea2012:Elephanttranscriptomeprovidesinsights}.
Here we confirm that the sole prolactin in \elephant is not only
expressed but highly expressed in the placenta, and likely functions
both in the maintenance of the corpus luteum and as a placental
lactogen \cite{Yamamoto.ea2011:PlacentationinAfricanelephant}.
\textit{PRL} is not expressed in \opossum, even though there is
evidence that it is expressed in the placenta of another marsupial,
\textit{Macropus eugenii} (tammar wallaby)
\cite{Menzies.ea2011:Placentalexpressionofpituitary}. This suggests
that placental prolactin expression in marsupials may be as diverse as
in eutherian mammals. One possible explanation of the diversity of
expression is the presence of endogenous retroviruses such as MER39 in
primates, MER77 in rodents, and L1-2\_LA in \elephant
\cite{Emera.ea2012:Convergentevolutionofendometrial} although this
does not explain the diversity of expression of different isoforms of
\textit{PRL} in \mouse or their tissue specificity in primates. An
alternative, that \textit{HOXA11} expression is responsible for
\textit{PRL} expression
\cite{Lynch.ea2008:Adaptivechangesintranscription} seems not to be the
case in placenta, as the rodent with the highest \textit{HOXA11}
expression, \galili
(\Sexpr{gene.expression("nannospalax_galili","Hoxa11",complete=TRUE)})
is the rodent with the lowest expression of \textit{PRL}
(\Sexpr{gene.expression("nannospalax_galili","Prl",complete=TRUE)},
\cref{fig:prolactin_evolution_tree}), and \elephant also does not
express \textit{HOXA11} highly either
(\Sexpr{gene.expression("loxodonta_africana","HOXA11",complete=TRUE)},
Supplemental \cref{fig:hoxa11_evolution_tree}).

\subsubsection{\Ac{GH} family}

The \ac{GH} family has expanded in anthropoid primates, resulting in
approximately 5 genes in \human (\textit{GH1}, \textit{GH2},
\textit{CSH1}, \textit{CSH2}, \textit{CSHL1}) and at least three in
\bonobo (two \textit{GH2} and \textit{GH1}; owing to the similarity
between the two \textit{GH1} in \textit{Pan troglodytes} we cannot
reliably distinguish transcripts between the two). All show
substantial expression in the \human placenta
(\cref{fig:csh1_evolution_tree}). The expression of the \ac{GH} family
in placenta appears to be primate specific, and likely existed at the
time of the last common ancestor between all extant anthropoids
\cite{Papper.ea2009:Ancientoriginofplacental}. There is some evidence
that increased expression of these genes may be correlated with
overall birth weight of offspring in \human
\cite{MÃ¤nnik.ea2010:Differentialexpressionprofileof}, and it is
possible that the extreme expression levels of these genes may be due
to high birth weight. \human placental \ac{GH} genes are involved in
increasing blood flow, replacing maternal pituitary expression of
\ac{GH}, as well as trophoblast invasion
\cite{Alsat.ea1998:Physiologicalroleofhuman,Lacroix.ea2005:Stimulationofhumantrophoblast,Schiessl.ea2007:Roleofplacentalgrowth},
and it is likely that they have the same function in \bonobo.

\begin{figure}
<<chorionic.somatomammotropin.alpha.evolution.tree,echo=FALSE,fig.width=12,fig.height=12,warning=FALSE,message=FALSE>>=

tree.id <-
    genes.to.tree[combined.fpkm[gene_short_name=="CGA" & species=="homo sapiens",gene_id],tree]
gene.tree.table <- trees[[tree.id]]$table
gene.tree.expression <- combined.fpkm[gene_id %in% gene.tree.table[,gene_id]]
gene.tree <- read.tree(text=trees[[tree.id]]$tree.dat)
plot.tree.matrix(read.tree(text=trees[[tree.id]]$tree.dat),gene.tree.table,gene.tree.expression)
@ 
\caption{\genetreeexplanation{chorionic gonadotropin, alpha subunit}}
\label{fig:cga_evolution_tree}
\end{figure}

\begin{figure}
<<chorionic.gonadotropin.evolution.tree,echo=FALSE,fig.width=12,fig.height=12,warning=FALSE,message=FALSE>>=

tree.id <-
    genes.to.tree[combined.fpkm[gene_short_name=="CGB1" & species=="homo sapiens",gene_id],tree]
gene.tree.table <- trees[[tree.id]]$table
gene.tree.expression <- combined.fpkm[gene_id %in% gene.tree.table[,gene_id]]
gene.tree <- read.tree(text=trees[[tree.id]]$tree.dat)
plot.tree.matrix(read.tree(text=trees[[tree.id]]$tree.dat),gene.tree.table,gene.tree.expression)
@ 
\caption{\genetreeexplanation{chorionic gonadotropin, beta subunit}}
\label{fig:cgb_evolution_tree}
\end{figure}

The human gene \textit{CGA} codes for the alpha subunit of multiple
hormones including pituitary hormones (luteinizing hormone,
follicle-stimulating hormone, thyroid-stimulating hormone), as well as
numerous chorionic gonadotropins with variable beta subunits
\cite{Maston.ea2002:Chorionicgonadotropinhasrecent}. The function of
the chorionic gonadotropins is to prevent the regression of the corpus
luteum \cite{Hanson.ea1971:EffectsofHCGand}. Among the six placentas
we analyzed here, \textit{CGA} is only expressed in the placentas of
\human and \bonobo (\cref{fig:hormone_genes,fig:cga_evolution_tree}),
which corresponds with the emergence of the chorionic gonadotropins in
anthropoid primates
\cite{Maston.ea2002:Chorionicgonadotropinhasrecent}. However, oddly,
\textit{CGA} is expressed at much higher levels in both species than
the beta subunits with which its product dimerizes (\cref{fig:hormone_genes,fig:cga_evolution_tree,fig:cgb_evolution_tree}).
\textit{CGA} is one of several genes noted by
\citet{Uddin.ea2008:Distinctgenomicsignaturesof} to have undergone
adaptive evolution of the \human{}-\textit{P. troglodytes} stem lineage.

\subsubsection{Relaxin}

Relaxin (\textit{RLN}) is a hormone that has multiple effects during
pregnancy including increasing maternal blood flow, widening the pubic
bone and relaxing uterine musculature in preparation for labor. In
\dog, \elephant, \mouse, \carmeli, and \horse there is high expression
of relaxin in the term placenta, with little evidence in other species
examined here (Supplemental \cref{fig:rln2_evolution_tree}). \textit{RLN} is known
to be produced in the placenta of the pregnant \dog and is used in the
diagnosis of canine pregnancy
\cite{Steinetz.ea1989:Serumrelaxinandprogesterone,Tsutsui.ea1991:Determinationofsourceof},
the luteal ovary and placenta of \pig
\cite{Hisaw.ea1948:Relaxininovaryof}, and the placenta of \horse
\cite{Klonisch.ea1997:Placentallocalizationofrelaxin}. However, the
source of relaxin produced by pregnant \elephant remains unclear
\cite{Meyer.ea2004:Speciesandfetalgender}. The high rise in relaxin in
\elephant has been suggested to be a major signal for the initiation
of labor \cite{Niemuller.ea1998:Plasmaconcentrationsofimmunoreactive}.
Here we show that the placenta of elephants show high expression of
relaxin and is likely a major source of \textit{RLN} at term. In
contrast to the results of
\cite{Evans.ea1993:mouserelaxingenenucleotide}, we also see expression
of \textit{RLN} in the placenta of \mouse and \carmeli.

\subsubsection{\textit{NPPB} and \textit{NPPC}}

Two genes, \textit{NPPB} and \textit{NPPC}, code for peptide hormones
that are important in the inducement of vasodilation, and may be
important in increasing maternal blood flow to the fetus
\cite{Cameron.ea1996:sitesofgeneexpression}. There is some indication
that these proteins are present in the decidua of mice
\cite{Cameron.ea1996:sitesofgeneexpression}, but we find very little
expression in the fetal portion of the mouse placenta
(Supplemental \cref{fig:hormone_genes,fig:nppc_evolution_tree,fig:nppb_evolution_tree}).
However, we do find high expression of at least one of these genes in
\dog, \cow, and \elephant. The lack of expression in the hemochorial
placentas of Euarchontoglires may mean that high expression of these
genes are essential in increasing maternal blood flow in
epitheliochorial and endotheliochorial placentas.

\subsection{The Evolution of Placental Expression of Galectins}

Galectins are recognized as major regulators of pregnancy in both
fetal and maternal tissues
\cite{Than.ea2012:Galectinsguardiansofeutherian}. Indeed, the
evolution of new lineage-specific galectins has occurred in both
primates and ruminants, with the genes \textit{LGALS13},
\textit{LGALS14}, and \textit{LGALS16} recognized as being
placenta-specific in \human and regulating immune responses by
inducing apoptosis of maternal T-cells
\cite{Than.ea2009:primatesubfamilyofgalectins} (Supplemental
\cref{fig:galectins_lgals13_evolution_tree}). Indeed, \textit{LGALS14}
has the highest expression of any galectin in the \human and \smonkey
placenta (\cref{fig:hormone_genes}, Supplemental
\cref{fig:galectins_lgals13_evolution_tree}), and other primate
galectins such as \textit{LGALS13} and \textit{LGALS16} are also
expressed in \human and \smonkey placenta. However, in the closely
related \bonobo, \textit{LGALS14} is not found, and \textit{LGALS1} is
the most abundant galectin. \textit{LGALS1} shows generally high
expression in all placentas analyzed. \textit{LGALS1} is also involved
in mediating maternal-fetal immune tolerance, and is expressed in both
maternal and fetal tissue in primates
\cite{Than.ea2008:Emergenceofhormonaland}. \textit{LGALS1} placental
expression was thought to be confined to eutherian mammals, as there
is evidence of adaptive evolution at this node, as well as the
appearance of estrogen response elements in eutherian cis-regulatory
regions \cite{Than.ea2008:Emergenceofhormonaland}. It was unclear as
to whether galectins such as \textit{LGALS1} were important in the
regulation of pregnancy in marsupials, as well as eutherians. Here we
show substantial expression of galectins in the opossum, especially
\textit{LGALS1} (Supplemental \cref{fig:galectins_evolution_tree}). In
addition, \textit{LGALS2} is also expressed. The presence of galectins
at the maternal-fetal interface in \opossum implies that it was also
present in the ancestral placenta of therian mammals. Indeed, adaptive
evolution of \textit{LGALS1} on the eutherian stem is likely due to
the increased invasive nature of eutehrian placentation rather than
the origination of maternal-fetal tolerance. Although the function of
\textit{LGALS1} in the marsupial placenta needs to be confirmed, it is
likely that the mediation of maternal-fetal immune tolerance by
galectins is an ancient feature of mammalian viviparity.

\subsection{The Opossum Placenta and the Evolution of Viviparity}

The placenta of \opossum is distinctly different from that of the term
placenta of eutherian mammals in that it is derived from both the
chorion and yolk sac. Other differences between the placentas of
marsupials and eutherians include lack of invasive extravillous
trophoblasts in marsupials, the retention of a permeable shell coat
for 2/3 of marsupial gestation, and the greater degree of imprinting
in placenta-expressed genes in eutherians
\cite{Renfree2010:ReviewMarsupialsplacentalmammals,Renfree.ea2013:originandevolutionof}.
In addition, the placenta of \opossum is also distinct from other
marsupials, in that it is considered one of the least specialized, and
thought be closer to the original marsupial condition
\cite{Renfree2010:ReviewMarsupialsplacentalmammals}. In \opossum, the
duration of direct connection between maternal and fetal tissues is as
short as two days in a gestational period of 13-15 days
\cite{Harder.ea1993:Gestationandplacentationin}. During this period,
the placenta is directly engaged in the exchange of urea with maternal
tissues, owing to the lack of a vesicular allantois
\cite{Freyer.ea2007:Placentalfunctionintwo}. This differs drastically
with the situation in another well-studied marsupial, the tammar
wallaby \cite{Freyer.ea2007:Placentalfunctionintwo}. Therefore, we
must interpret differences between the opossum and eutherian placentas
with caution, as highly expressed genes and functional annotation
clusters may represent didelphid or \opossum specific adaptations.

% Genes that show a significant shift on the \opossum lineage include
% functional annotation clusters that correspond to active generation of
% energy, transport, and lipid breakdown via the peroxisome (Table 10;
% Supplemental Table X). The choriovitelline placenta of both didelphids
% and the tammar wallaby are known to be involved in glycolysis and
% protein synethsis (Freyer et al., 2003). This implies that the
% choriovitelline placenta in marsupials is an active site of metabolism
% and molecular synthesis, as well as the transport of nutrients from
% the maternal tissues. This is a distinct difference that is not seen
% in significant shifts on other lineages, and may be due to the
% formation of the placenta in the opossum from the yolk sac. In
% addition, the urea cycle is a highly enriched functional annotation
% cluster which directly correlates with the high levels of urea in the
% Monodelphis placenta (Freyer et al., 2007). This is potentially a
% Monodelphis-specific characteristic as discussed above.

The \opossum placenta shows expression of genes that either do not
exist in eutherians or are not found in the term eutherian placenta
(\cref{tab:top50_opossum_placenta}). The existence of embryonic forms
of hemoglobin in fetal tissues at term, as evidenced by the expression
of \textit{HBE} and \textit{HBZ} is a feature of marsupials that is
generally due to their very short gestation periods. Embryonic
hemoglobins in marsupials have a lower affinity for oxygen and allow
for a lower rate of oxygen uptake from maternal tissues
\cite{Holland.ea1997:Characterizationofembryonicglobin}. One novel
uncharacterized gene that shows high expression (\textit{LOC10001195}) is not
found in eutherians; Ensembl gene trees reveal that homologs of this
gene exist in other amniotes such as birds and turtles, but are not
present in eutherian mammals. The homolog of this gene in chicken is
found to expressed in reproductive tract
\cite{Froman.ea2006:expressedsequencetaganalysis}, but its function is
unknown. \textit{WFDC2} is a member of the whey acidic protein (WAP)
family and involved in innate immunity. It is expressed in the mammary
gland of the tammar wallaby during pregnancy and early lactation
\cite{Sharp.ea2007:Molecularevolutionofmonotreme}, and present in
relative abundance in \opossum placenta.

\subsection{Novel Gene Expansions in the Elephant Placenta}

\begin{figure}
<<sslp1.evolution.tree,echo=FALSE,fig.width=12,fig.height=12,warning=FALSE,message=FALSE>>=

tree.id <-
    genes.to.tree[combined.fpkm[gene_id=="ENSCAFG00000010638" & species=="canis familiaris",gene_id],tree]
gene.tree.table <- trees[[tree.id]]$table
gene.tree.expression <- combined.fpkm[gene_id %in% gene.tree.table[,gene_id]]
gene.tree <- read.tree(text=trees[[tree.id]]$tree.dat)
plot.tree.matrix(read.tree(text=trees[[tree.id]]$tree.dat),gene.tree.table,gene.tree.expression)
@ 
\caption{\genetreeexplanation{\textit{SSLP1}}}
\label{fig:sslp1_evolution_tree}
\end{figure}

We found evidence of high expression in two novel expansions of genes
in the \elephant genome. One family includes 11 genes that are related
to the \textit{PRSS16} gene, a thymus-specific serine protease that
may be involved regulating the presentation of self-peptides in CD4+
T-lymphocytes \cite{Fornari.ea2012:TranscriptionprofilingofPrss16}.
All 11 of these genes are expressed in the elephant placenta, three of
which have very abundant mRNAs
(\cref{tab:top50_elephant_placenta,fig:prss16_evolution_tree}). It is
possible that this elephant-specific gene family may be involved in
immunity in the elephant placenta and the mediation of maternal-fetal
immune tolerance; however, future work will need to test these
hypotheses. Three other genes abundantly expressed in the elephant
placenta are homologues of \textit{SSLP1}, a gene with predominant
expression in the seminal vesicle in mouse
\cite{Li.ea2006:SSLP-1secretedLy-6protein}. Homologues of
\textit{SSLP1} are also expressed in \cow, \dog, \horse, and \sheep.
\textit{SSLP1} also contains an Alu retrotransposon and may be
involved in the development of villi via cell proliferation
\cite{Ushizawa.ea2009:Characterizationandexpressionanalysis}. This
gene is not found in primates. Expansion of these genes with high
expression in \elephant may indicate an elaboration of this system.

\subsection{Annexin Complex}

\subsection{Human Specific Expression Shifts and the Evolution of Preeclampsia}

The \human placenta is distinctive from many other primates in that it
possesses a deeply invasive trophoblast that penetrates the inner
myometrium of the uterus
\cite{Carter.ea2011:Evolutionofinvasiveplacentation}. It is
hypothesized that this feature is related to the increased demand for
resources by the developing fetus to feed the expanded size of the
human brain
\cite{Carter.ea2011:Evolutionofinvasiveplacentation,Robillard.ea2002:Evolutionaryadaptationstopre-eclampsiaeclampsia}.
\human have the largest brain in terms of relative size and this
differs markedly from closely related great apes
\cite{Boddy.ea2012:Comparativeanalysisofencephalization}. Deep
penetration of the myometrium occurs in chimpanzees and gorillas, but
this penetration may be accelerated in humans
\cite{Carter.ea2011:Evolutionofinvasiveplacentation}.

The near exclusivity of deep trophoblast invasion in humans has also
led many to speculate on the origin of various disorders, which are
almost exclusively symptomatic of human pregnancy. For example,
pre-eclampsia is a human disease that occurs in 4\% of pregnancies in
the United States
\cite{Chaiworapongsa.ea2014:Pre-eclampsiapart1current}. Preeclampsia
can be attributed to the lack of adequate invasion of maternal tissue,
and is noted by the onset of hypertension and proteinuria
\cite{Chaiworapongsa.ea2014:Pre-eclampsiapart1current}. Some cases of
pre-eclampsia can develop into eclampsia, a more severe condition
resulting in seizures during pregnancy and death. Lack of invasion in
pre-eclampsia may be attributed to the inefficiency of growth
regulators as well as the inability to sufficiently suppress the
maternal immune response. Although there are speculative cases of
eclampsia in great apes
\cite{Baird1981:Eclampsiainlowlandgorilla,Stout.ea1969:Glomerularcapillaryendothelialswelling,Thornton.ea1992:Convulsionsinpregnancyin},
it is exceedingly rare in these species
\cite{Carter.ea2011:Evolutionofinvasiveplacentation}. It is possible
that a comparative evolutionary approach to gene expression can
identify genes that have divergent expression patterns from other
mammals as well as association with pre-eclampsia.

Functional annotation clusters significantly enriched on the human
lineage (blood vessel development, extracellular matrix, tube
morphogenesis) seem to suggest involvement in remodeling of tissues
and increased blood flow, both of which may correlate with the
invasion of the trophoblast (\cref{tab:go_terms_all_species}). In
addition, clusters such as cytokines and defense response
(\cref{tab:go_terms_all_species}) also indicate an a significant
change in expression of the immune response that may be involved in
immunosupression at the maternal-fetal boundary to further allow
invasion of fetal tissues.

% I haven't actually done this analysis, so commented out

% Many genes that both show expression shifts on the human lineage and
% have very high expression (FPKM>200) are directly related to the
% highly invasive nature of the human placenta. For example, GPC3,
% glypican 3, is an imprinted gene and cell growth regulator, associated
% with Simpson-Golabi-Behmel fetal overgrowth syndrome, and involved in
% intrauterine growth restriction (Monk and Moore, 2004; Pilia et al.,
% 1996). GPC3 has the highest expression in humans (Table 2), but is
% also expressed in other mammals to a lesser degree, especially those
% with invasive hemochorial placentas like bonobos (\ac{FPKM}= 199.924),
% spider monkeys (\ac{FPKM}= 96.24), and the mouse (\ac{FPKM}= 80.7545)
% (Supplemental Tables X-XX). ADAM12 increases the availability of
% insulin-like growth factors that lead to the promotion of cell growth
% and placental invasion (Laigaard et al., 2005). (Hou et al., 2009)
% found evidence of adaptive evolution in ADAM12 on the primate lineage,
% and like GPC3 it is expressed in the bonobo (\ac{FPKM}= 46.1553), spider
% monkey (\ac{FPKM}=30.41) and mouse (\ac{FPKM}=2.25391) as well, although to a
% lesser degree (Supplemental Tables X-XX). RHOBTB3 is imprinted and
% associated with trophoblast invasion as well (Bilban et al., 2010;
% Yuen et al., 2011). In addition, one other gene, KISS1, shows
% exceptional abundance in the human placenta (Table 2); however due to
% its absence in elephant Ensmebl gene annotations, it was not part of
% the 1:1 orthologues examined in the comparative study. Here, it is
% only expressed in the term placentas of humans (Table 2) and bonobos
% (\ac{FPKM}=17.6394; Supplemental Table X). KISS1 is thought to be involved
% in the regulation of trophoblast invasion, and shows highest
% expression in the first trimester of pregnancy (Bilban et al., 2004;
% Reynolds et al., 2009).
% 
% In addition, several other genes that show expression shifts on the
% human lineage also have major effects on placental function. Genes
% such as TXK and EBI3 are intimately involved in the immune system in
% the placenta. TXK is involved in natural killer cell differentiation
% (Bezman et al., 2012). EBI3 is implicated in natural killer cell
% function an maternal-fetal immunomodulation in the syncytiotrophoblast
% (Devergne et al., 2001). KYNU and KMO are interacting genes that are
% involved in tryptophan metabolism, which may be involved in
% immunosupression and the prevention of fetal rejection (Takikawa,
% 2005). THSD7A is a thrombospondin, which inhibits endothelial cell
% migration and tube morphogenesis (Wang et al., 2010).
% 
% Many genes that show evidence of a shift on the human lineage show
% evidence of implication in pre-eclampsia, including ADAM12 (Gack et
% al., 2005; Laigaard et al., 2005) MAOA (Carrasco et al., 2000;
% Bottalico et al., 2004), BCL2 (Ishihara et al., 2002), GPC3 (Gui et
% al., 2012; Ogawa, 2007), TMEM8A, CLIC3, EBI3, FLT1, BCL6, SYDE1, and
% MAN1C1 (Enquobahrie et al., 2008; Winn et al., 2009; Nishizawa, 2011).
% In addition KISS1 is also involved in pre-eclampsia as well (Sado et
% al., 2011; Qiao et al, 2012). Due to the association of many genes
% that show human-specific shifts as well as high expression, we
% conclude that pre-eclampsia is possibly a symptom of recent evolution
% of gene expression on the human lineage. Although many genes such as
% GPC3 and ADAM12 show distinctive relatively high expression in bonobo,
% this expression has intensified on the human lineage. Sequencing of
% more placentas of nonhuman primates, including other individuals of
% the genus Pan will help to further characterize this transition.

\section{Conclusion}
    
We sequenced \Sexpr{star.logs[,length(unique(species))]} placentas
spanning the phylogenetic tree of mammals. This revealed a dynamic
expression pattern of expression of lineage-specific expansions of
gene families and differential gene expression that agreed with the
scenario of an âarms raceâ between maternal and fetal tissues that
have shaped such disparate gene expression profiles. In addition, we
highlighted the broad spectrum of hormone gene expression in multiple
species that suggested that there is a lack of conservation in hormone
signaling during mammalian pregnancy. Major differences between
expression in marsupial and eutherian mammals at term seemed to be
correlated with the differences between choriovitelline and
chorioamniotic placenta, with the marsupial placenta more enriched in
genes the contributed to the active generation of energy. We also
identified that galectin expression, especially that of
\textit{LGALS1}, is a common feature of all mammalian placentas and
was most likely present in the ancestral therian. Also, we identified
two new expansions of genes that are highly expressed in the \elephant
placenta hat may be involved in trophoplast differentiation and immune
system function, mirroring expansions in other groups such as
primates, rodents, and ruminants. We also quantified significant
lineage-specific shifts in gene expression, concentrating on the
identification of genes that showed gene expression changes on the
human lineage. We cautiously propose that many of these genes may be
related to the invasive nature of the trophoblast in \human, although
we note that in many of these genes, there is in some cases
substantial expression in the \bonobo as well. These genes have the
potential to spur further research and may be at the root of
obstetrical disorders such as pre-eclampsia. We propose that more
placentas from diverse lineages, especially those of marsupials and
primates, to further identify genes and expression levels that are
specific to marsupials and eutherians, as well as confirm the identity
of human-specific expression shifts.


\begin{backmatter}
\section*{Competing interests}
  The authors declare that they have no competing interests.

\section*{Author's contributions}
Don Armstrong analyzed all of the data and wrote the paper. 

Michael McGowan wrote the paper and performed initial analyses.

\section*{Acknowledgements}
  Text for this section \ldots

\printaddresses

\end{backmatter}

\printbibliography



\end{document}
