%\documentclass[english,12pt]{article}
%\documentclass[5p]{elsarticle_dla}
\documentclass{elsarticle_dla}
\usepackage{fontspec}
\setmainfont{FreeSerif}
\setsansfont{FreeSans}
\setmonofont{FreeMono}
\setmathrm{FreeSerif}
\setmathsf{FreeSans}
\setmathtt{FreeSerif}
%\usepackage[letterpaper,left=1in,right=1in,top=1in,bottom=1in,headheight=14pt]{geometry}
%\usepackage[letterpaper]{geometry}
%\usepackage{url}
\usepackage{graphicx}
%\usepackage{rotating}
%\usepackage{wrapfig}
\usepackage{booktabs}
%\usepackage{float}
\usepackage{multirow}
%\usepackage{setspace}
\usepackage[macros=false,single=true,single-form=long]{acro}
\usepackage{array}
\usepackage{longtable}
\usepackage{amsmath}
\usepackage[backend=biber,natbib=true,hyperref=true,citestyle=numeric-comp,style=nature,autocite=inline]{biblatex}
\addbibresource{references.bib}
\usepackage[hyperfigures,bookmarks,colorlinks,linkcolor=black,urlcolor=black,citecolor=blue,breaklinks=true]{hyperref}
% Floats at end for journals
% \usepackage[nolists,figuresfirst,nomarkers,heads]{endfloat}
% \renewcommand{\efloatseparator}{\mbox{}}
%\AtBeginFigures{\newpage \renewcommand{\efloatseparator}{\mbox{}}\section*{Figure Legends}}
%\AtBeginTables{\newpage \section*{Tables}\renewcommand{\efloatseparator}{\mbox{}}}
% \renewcommand{\includegraphics}[2][]{}
\usepackage[nomargin,inline,draft]{fixme}
\newcommand{\DLA}[1]{\textcolor{red}{\fxnote{DLA: #1}}}
\newcommand{\DEW}[1]{\textcolor{purple}{\fxnote{DEW: #1}}}
\newcommand{\PP}[1]{\textcolor{orange}{\fxnote{PP: #1}}}
% \usepackage[x11names,svgnames]{xcolor}
% \usepackage{texshade}
% \usepackage{tikz}
\usepackage{nameref}
\usepackage{xspace}
% this package will stretch the font slightly if necessary.
% \usepackage{microtype}
\usepackage{zref-xr,zref-user}
\zxrsetup{toltxlabel}
\zexternaldocument*{supplemental_material}
\usepackage[capitalize]{cleveref}
\def\UrlBreaks{\do\.\do\@\do\\\do\/\do\!\do\_\do\|\do\;\do\>\do\]%
\do\)\do\,\do\?\do\&\do\'\do+\do\=\do\#%
\do\/%
}%
\input{common_latex}
\tolerance=5000
\emergencystretch=0.1pt

\begin{document}
% \raggedright


<<load.libraries,echo=FALSE,results="hide",warning=FALSE,message=FALSE,error=FALSE,cache=FALSE>>=
opts_chunk$set(dev="cairo_pdf",out.width="\\columnwidth",out.height="0.7\\textheight",out.extra="keepaspectratio")
opts_chunk$set(cache=TRUE, autodep=TRUE)
options(device = function(file, width = 8, height = 7, ...) {
  cairo_pdf(tempfile(), width = width, height = height, ...)
})
options(scipen = -2, digits = 1)
library("data.table")
library("ggplot2")
library("reshape2")
library("xtable")
library("grid")
library("gridExtra")
library("ape")
library("ggtree")
library("RSQLite")
library("topGO")
library("org.Hs.eg.db")
library("scales")
library("magrittr")
library("tidyr")
library("phytools")
@ 

<<source.to.latex,echo=FALSE,message=FALSE,results="hide",cache.extra=file.info(paste0("../common_r_code/",c("to_latex.R","plot_tree_matrix.R","utility_functions.R")))[,"mtime"]>>=
for (a in paste0("../common_r_code/",c("to_latex.R","plot_tree_matrix.R","utility_functions.R"))) {
    source(a)
}
@ 

<<load.data,echo=FALSE,results="hide",cache.extra=file.info(paste0("../data/",c("combined_fpkm","gene_trees","housekeeping_genes_superset","placenta_core_transcriptome","placenta_go_results_bp","placenta_go_results_mf","hugo_nomenclature","gene_annotation")))[,"mtime"]>>=
for (a in paste0("../data/",c("combined_fpkm","gene_trees",
                              "housekeeping_genes_superset",
                              "placenta_core_transcriptome",
                              "placenta_go_results_bp",
                              "placenta_go_results_mf",
                              "hugo_nomenclature",
                              "gene_annotation"
                              ))) {
    load(a)
}
combined.fpkm[,species:=factor(species,levels=names(species.names))]
setkey(genes.to.tree,"gene_id")
star.logs[,value:=gsub("%$","",value)]
star.logs.wide <- 
    data.table(dcast(star.logs,species+file~field))
setnames(star.logs.wide,
         "Average input read length",
         "input.read.length")
setnames(star.logs.wide,
         "Number of input reads",
         "input.reads")
setnames(star.logs.wide,
         "Uniquely mapped reads number",
         "unique.map.reads")
star.logs.wide[,`% bad reads`:=(as.numeric(`% of reads mapped to too many loci`)+
                            as.numeric(`% of reads unmapped: other`)+
                            as.numeric(`% of reads unmapped: too many mismatches`)+
                            as.numeric(`% of reads unmapped: too short`))]
star.logs.wide[,`% good reads`:=100-`% bad reads`]
star.logs.wide[,input.reads:=as.numeric(input.reads)]
star.logs.wide[,input.read.length:=as.numeric(input.read.length)]
setkey(hugo.nomenclature,"Approved Symbol")
@ 

<<load.go.results,echo=FALSE,results="hide",cache.extra=tools::md5sum("../go_annotation/go_tissue_analysis")>>=
load("../go_annotation/go_tissue_analysis")

@ 

<<load_plot_ancestral_trees,message=FALSE,echo=FALSE,results="hide",cache.extra=file.info(paste0("../common_r_code/",c("plot_ancestral_state_tree.R")))[,"mtime"]>>=
for (a in paste0("../common_r_code/",c("plot_ancestral_state_tree.R"))) {
    source(a)
}
@ 

<<plot.defaults,echo=FALSE,results="hide">>=
#grad.def <- list(low = "#132B43", high = "#56B1F7", space = "Lab", na.value = NA)
grad.def <- list(low = "gray90", high = "blue", space = "Lab", na.value = NA)

@ 

<<placental_classification,echo=FALSE,results="hide",cache.extra=file.info(dir("../data","placenta.*classification.*",full.names=TRUE))[,"mtime"]>>=
placenta_classification <- fread("../data/placental_classification.txt")
load("../data/placenta_classification_p")
for (mf.bp in c("mf","bp")) {
    for (class in c("interdigitation","intimacy","shape")) {
        load(paste0("../data/placenta_classification_go_",mf.bp,"_",class))
    }
}

@ 

<<placenta_go_results,echo=FALSE,results="hide">>=
placgo <- function(goid,group="mf") {
    if (group=="mf") {
        temp <- placenta_go_results_mf
    } else {
        temp <- placenta_go_results_bp
    }
    paste0("$\\mathrm{FDR}=",
           number_to_latex(temp[GO.ID==goid,`FDR`]),
           "$"
           )
}
@ 

<<pathway_analysis,echo=FALSE,results="asis",cache.extra=file.info(dir("../data","placenta_gsea.*",full.names=TRUE))[,"mtime"]>>=
pathway <- list()
for (database in c("kegg","reactome","biocarta","netpath")) {
    load(paste0("../data/placenta_gsea_",database))
    eval(parse(text=paste0("gsea.table <- placenta_gsea_",database)))
    gsea.table[,Database:=gsub("^(.)","\\U\\1",database,perl=TRUE)]
    pathway[[database]] <-
        gsea.table
}
pathway.analysis <-
    rbindlist(pathway)[fdr < 0.1,]
rm(pathway)
pathway.analysis[,Pathway:=gsub("^(KEGG|BIOCARTA|REACTOME) ","",
                                gsub("_+"," ",
                                     Pathway))]
pathway.analysis[,Pathway:=gsub(".+\\%","",Pathway)]

setnames(pathway.analysis,"p_values","p")
setorder(pathway.analysis,"p")

placpath <- function(pathway,database) {
    paste0("$\\mathrm{FDR}=",
           number_to_latex(pathway.analysis[grepl(pathway,Pathway) & Database==Database,
                                            fdr]),
           "$")
}

@ 

% Acronyms
\input{acronyms}

\title{The core transcriptome of mammalian placentas and the
  divergence of expression with placental shape}%
%
\author[affigb]{Don L. Armstrong\fnref{n1}}
\ead{don@donarmstrong.com}
\author[afflon]{Michael R. McGowen\fnref{n1}}
\ead{1mmcgowen1@gmail.com}
\author[affigb]{Amy Weckle}
\ead{aweckle@illinois.edu}
\author[affigb]{Priyadarshini Pantham}
\ead{pantham@illinois.edu}
\author[affwayne]{Jason Caravas}
\ead{jacaravas@gmail.com}
\author[affmich]{Dalen Agnew}
\ead{agnewd@dcpah.msu.edu}
\author[affuscd]{Kurt Benirschke}
\ead{kbenirschke@ucsd.edu}
\author[affapetrust]{Sue Savage-Rumbaugh}
\ead{suerumbaugh@gmail.com}
\author[affhaifa]{Eviatar Nevo}
\ead{nevo@research.haifa.ac.il}
\author[affnih,affwaynepath]{Chong J Kim}
\ead{cjkim@med.wayne.edu}
\author[affyale]{GÃ¼nter P. Wagner}
\ead{gunter.wagner@yale.edu}
\author[affnih]{Roberto Romero}
\ead{prbchiefstaff@med.wayne.edu}
\author[affigb,affapetrust,affmip]{Derek E. Wildman\corref{cor1}}
\ead{wildmand@illinois.edu}
\cortext[cor1]{Corresponding author}
\address[affigb]{Institute for Genomic Biology, University of Illinois at Urbana-Champaign, Urbana, IL, USA}
\address[afflon]{School of Biological and Chemical Sciences, Queen Mary, University of London, London, UK}
\address[affwayne]{Center for Molecular Medicine and Genetics, Wayne State University School of Medicine, Detroit, MI, USA}
\address[affmich]{Diagnostic Center for Population and Animal Health, College of Veterinary Medicine, Michigan State University, East Lansing, MI, USA}
\address[affuscd]{Department of Pathology, University of California San Diego, La Jolla, CA, USA}
\address[affapetrust]{Great Ape Trust/Bonobo Hope Sanctuary, Des Moines, IA, USA}
\address[affnih]{Perinatology Research Branch, Eunice Kennedy Shriver National Institute of Child Health and Human Development (NICHD), National Institutes of Health (NIH), Detroit, MI, USA}
\address[affwaynepath]{Department of Pathology, Wayne State University School of Medicine, Detroit, MI, USA}
\address[affyale]{Department of Ecology and Evolutionary Biology and Yale Systems Biology Institute, Yale University, New Haven, CT, USA}
\address[affmip]{Department of Molecular and Integrative Physiology, University of Illinois at Urbana-Champaign, Urbana, IL, USA}
\address[affhaifa]{The Institute of Evolution, University of Haifa, Haifa, Israel}

\begin{abstract}
  \textbf{Introduction} The placenta is arguably the most anatomically
  variable organ in mammals even though its primary function is
  conserved.

  \textbf{Method} Using RNA-seq, we measured the expression profiles
  of \Sexpr{star.logs[,length(unique(file))]} term placentas of
  \Sexpr{star.logs[,length(unique(species))]} species of mammals
  representing all major eutherian superordinal clades and marsupials,
  and compared the evolution of expression across clades.
  
  \textbf{Results} We identified a set of
  \Sexpr{core.placenta.transcriptome.long[housekeeping==FALSE,length(unique(human_name))]}
  core genes which are expressed (FPKM $\ge 10$) in all eutherian placentas, %
  %
  including genes with immune-modulating properties (\ac{ANXA2},
  \ac{ANXA1}, \ac{S100A11}, \ac{S100A10}, and \ac{LGALS1}), cell-cell
  interactions (\ac{LAMC1}, \ac{LUM}, and \ac{LGALS1}),
  invasion (\ac{GRB2} and \ac{RALB}) and syncytialization
  (\ac{ANXA5} and \ac{ANXA1}). %
  We also identified multiple pre-eclampsia associated genes which are
  differentially expressed in \textit{Homo sapiens} when compared to
  the other %
  \Sexpr{star.logs[,length(unique(species))-1]} species. %
  %
  Multiple genes are significantly associated with placenta
  morphology, including \ac{EREG} and \ac{WNT5A} which are both
  associated with placental shape.

  \textbf{Discussion} %
  \Sexpr{core.placenta.transcriptome.long[housekeeping==FALSE,length(unique(human_name))]}
  genes are important for the core functions of the placenta in all
  eutherian species analyzed. The molecular functions and pathways
  enriched in the core placenta align with the evolutionarily
  conserved functionality of the placenta.

 % \DEW{The discussion and results seem redundant. Given the title
 %   shouldn't something about shape be mentioned?}
  
\end{abstract}

\begin{keyword}
mammal \sep  gene expression \sep evolution \sep RNA-seq \sep placenta
\end{keyword}

\maketitle

% Reset all acronyms
\acresetall

\listoffixmes

\section{Introduction}

Even though the placenta's primary function is conserved among
mammals, it varies widely in terms of shape, degree
of intimacy between fetal tissues and maternal tissues, and degree of
interdigitation of maternal and fetal tissues in different species of
mammals
\cite{Mossman.1987:vertebratefetalmembranes} %,Moffett.ea2006:Immunologyofplacentationin}
(Supplemental \cref{tab:placental_classification}). Mammals also
differ drastically in the number and size of their offspring, as well
as the length of gestation; these traits are often associated with
specific aspects of placental morphology
\cite{Capellini.ea2011:Placentationandmaternalinvestment}
(Supplemental \cref{tab:gestational_periods}). Although the morphology
of the mammalian placenta has been well characterized in a number of
species \cite{Mossman.1987:vertebratefetalmembranes}, the diversity of
the molecular environment is only beginning to be understood from a
genetic perspective \cite{Carter2012:Evolutionofplacentalfunction}.

These observations lead to two related questions: 1) what are the
genes which are responsible for the \textbf{core functionality of the
  placenta} and 2) which genes are associated with \textbf{changes in
  placenta morphology}?

Previous studies of some species (i.e., \human, \mouse, \cow, and
\elephant) have provided some preliminary answers to these questions.
They revealed a stunning divergence in the genetics of the placenta,
especially in later stages of pregnancy
\cite{Carter2012:Evolutionofplacentalfunction,Knox.ea2008:Genomicevolutionofplacenta,Su.ea2002:Large-scaleanalysisofhuman,Hou.ea2012:Elephanttranscriptomeprovidesinsights}}.
For example, each superorder has evolved expansive placenta-specific gene
families with expression patterns that can vary in space and time
\cite{Green.ea2000:Pregnancy-associatedbovineandovine}. These include
independent expansions of the prolactins in \mouse, \carmeli, \galili,
and \cow, \acp{PSG} in \human and \mouse, \acp{PAG} in \cow, genes
related to the thymus-specific serine protease \ac{PRSS16} in
\elephant (in this work), and the expansion of the growth hormone and
chorionic gonadotropin beta genes in anthropoid primates
\cite{Haig2008:Placentalgrowthhormone-relatedproteins,Rawn.ea2008:evolutionregulationandfunction}.
Expression patterns of specific genes also can vary widely between
species. For example, the expression of orthologous genes in the
placenta have been found to vary substantially between \human and
\mouse \cite{Su.ea2002:Large-scaleanalysisofhuman}.

To address our questions, we 
sequenced and quantified the total RNA of mammalian placentas at term,
including \human, \bonobo (the sister
species to \textit{Pan troglodytes}), \smonkey,
\mouse, \galili, \carmeli, \dog, \cow, \elephant,
\armadillo, and \opossum.
These species represent four major branches of eutherian mammals
(Euarchontoglires, Laurasiatheria, Afrotheria, and Xenartha) as well
as marsupials. These placentas also represent a wide array of
placental morphologies as well as traits such as number and size of
offspring and length of gestation (\cref{tab:gestational_periods}). We
use this data, along with existing RNA-Seq data from \mouse
\cite{Wang.ea2011:surveyfornovelimprinted}, \human
\cite{Hughes.ea2015:Evaluatingintra-andinter-individual},
\horse
\cite{Coleman.ea2010:Structuralannotationofequine}, \sheep
\cite{Archibald.ea2010:sheepgenomereferencesequence}, and
\pig \cite{Ahanda.ea2012:Predictionofaltered3-} to: 1)
identify the non-housekeeping genes which are expressed in the
placentas of all eutherian species analyzed and therefore represent the
\emph{core functionality of the placenta}; 2) identify genes whose
expression level changes significantly in each of the morphology
types.

\section{Methods}

\subsection{Collection of Placental Tissue and RNA Extraction}

Fetal tissue was collected from the placenta of nine mammalian
species (\human [human], \bonobo [bonobo], \smonkey [spider monkey], \dog [domestic dog], \cow
[domestic cow], \elephant [African savanna elephant], \opossum [gray
short-tailed opossum], \galili [blind mole rat from Upper Galilee],
\carmeli [blind mole rat from northern Israel]
\cite{Nevo2001:adaptiveradiationofblindsubterraneanmole}). Information
on specimen ID, and collector is shown for each sample in Supplemental
\cref{tab:srr_numbers}. In the case of \bonobo, \cow, and \elephant,
the placenta was collected at term upon expulsion from the mother
following delivery. Placental tissue from \human was collected upon
delivery via a cesarean section. Near-term placental tissue from \dog
was collected upon the death of the mother and fetus from a car accident.
Placental tissue from \opossum was sampled at fetal stage 32, roughly
one day prior to birth. Only the fetal portion of the placenta was
sampled at the maternal-fetal interface (i.e. villous portion for
\human, \bonobo, \dog, and \elephant; cotyledon for \cow; diffuse
placenta for \opossum). All tissues were collected and stored in
RNAlater (Life Technologies, Carlsbad, CA) until RNA isolation could
be performed. Tissues were homogenized in Trizol, and total RNA was
isolated following the manufacturer's instructions. RNA was further
purified using RNeasy Midi Kit (Qiagen, Valencia, CA), in conjunction
with the RNase-Free DNase Set (Qiagen, Valencia, CA), following the
manufacturerâs protocol to remove any contaminating DNA from the
sample. Total RNA was then evaluated on a Nanodrop 1000 (Thermo
Scientific, Wilmington, DE) and an Agilent Bioanalyzer 2100 (Santa
Clara, CA) to confirm concentration and quality of the isolation. RNA
quality was assessed using A260/A280 and A260/A230 absorbance values.

\subsection{Sequencing}

All RNA-Seq mRNA isolation, library construction, and Illumina
sequencing was performed by the Applied Genomics Technology Center
(AGTC) at Wayne State University School of Me\-di\-cine using 2.5-5 Âµg of
total RNA and the standard Illumina protocol. Paired-end sequencing
was performed using the Illumina Genome Analyzer II with an insert
size of 230 base pairs (bp). Read lengths for species are shown in
Supplemental \cref{tab:ref_genome_aligned_species}.

\subsubsection{Additional Sequences}

For more complete evolutionary comparisons additional sequences of
placenta in \mouse [mouse]
(n=\Sexpr{star.logs[grepl("musculus",species),length(unique(file))]})
\cite{Wang.ea2011:surveyfornovelimprinted}, \human
(n=\Sexpr{star.logs[grepl("sapiens",species),length(unique(file))]})
\cite{Hughes.ea2015:Evaluatingintra-andinter-individual} \horse [horse]
\cite{Coleman.ea2010:Structuralannotationofequine}, \sheep [sheep] 
\cite{Archibald.ea2010:sheepgenomereferencesequence}, and \pig [pig]
\cite{Ahanda.ea2012:Predictionofaltered3-} were obtained from the
\ac{SRA} (Supplemental \cref{tab:srr_numbers}) and analyzed using the
same methodology as RNAs sequenced specifically for this study.

\subsection{Transcriptome Assembly, Alignment, and Quantification}

\begin{sloppypar}
Reads quality was evaluated using FASTQC v0.11.2
(\url{http://www.bioinformatics.babraham.ac.uk/projects/fastqc/}).
Transcriptomes without a suitable reference genome (\smonkey, \galili,
\carmeli; italic reference in Supplemental
\cref{tab:ref_genome_aligned_species}) were assembled using Trinity
v2014-07-17 \cite{Haas.ea2013:Denovotranscriptsequence} and annotated
using Diamond \cite{Buchfink.ea2015:Fastandsensitiveprotein} against
either the \human (\smonkey) or \mouse (\galili and \carmeli) transcriptome (Supplemental
\cref{tab:ref_genome_aligned_species}), and the
the match with the highest bitscore was chosen as the annotation.
Transcripts where the highest scoring match had a bitscore below 40
were left un-annotated.
% 
All transcriptomes were aligned to the reference (or
assembled) genome using STAR v2.4.2a~%
\cite{Dobin.ea2013:STARultrafastuniversalRNA-seq} (Supplemental
\cref{fig:transcriptome_statistics,fig:transcriptome_reads_vs_read_length})
and expression values in \ac{FPKM}
% \cite{Trapnell.ea2010:Transcriptassemblyandquantification}
were
quantified using Cufflinks v2.2.1
\cite{Trapnell.ea2009:TopHatdiscoveringsplicejunctions} (Supplemental
\cref{tab:all_ateles_fusciceps_placenta,tab:all_homo_sapiens_placenta,tab:all_pan_paniscus_placenta,tab:all_mus_musculus_placenta,tab:all_spalax_galili_placenta,tab:all_spalax_carmeli_placenta,tab:all_bos_taurus_placenta,tab:all_ovis_aries_placenta,tab:all_sus_scrofa_placenta,tab:all_equus_caballus_placenta,tab:all_canis_familiaris_placenta,tab:all_loxodonta_africana_placenta,tab:all_dasypus_novemcinctus_placenta,tab:all_monodelphis_domestica_placenta}).
%
The genome of the closely-related \textit{P. troglodytes} (separation
$\approx 1$ million years
\cite{PrÃ¼fer.ea2012:bonobogenomecomparedwith}) was used as a reference
for \bonobo %
.
\end{sloppypar}

\begin{figure}
<<species.tree,fig.width=8,fig.height=6,out.width="90mm",echo=FALSE,message=FALSE,warning=FALSE>>=
species.tree <- 
    read.tree(text="(Monodelphis_domestica,((Dasypus_novemcinctus,Loxodonta_africana)Atlantogenata,((Canis_familiaris,(Equus_caballus,(Sus_scrofa,(Ovis_aries,Bos_taurus))))Laurasiatheria,(((Spalax_carmeli,Spalax_galili),Mus_Musculus)Rodentia,((Pan_paniscus,Homo_sapiens),Ateles_fuciceps)Primates)Euarchontoglires)Boreoeutheria)Eutheria);")
pp <- ggtree(species.tree)+
    geom_text(aes(label=gsub("_"," ",label)),hjust=0,size=5,fontface="italic")+
    geom_text(data=data.frame(x=1,y=1),label="Metatheria",hjust=0,vjust=0,size=5,fontface="italic")+
    scale_x_continuous(limits=c(0,10))
print(pp %>%
      ## monodelphis
      phylopic("91324e57-b3f1-42e0-abe3-43e5bc8aa4c6", color="#86B875",alpha=0.3,node=1,width=0.2)
      ## Armadillo
      %>% phylopic("5d59b5ce-c1dd-40f6-b295-8d2629b9775e", color="#44EEBB",alpha=0.3,node=2,width=0.15)
      ## elephant
      %>% phylopic("80db1004-bc9f-4318-84e7-cdd9639a1f3e",color="#BB1111",alpha=0.3,node=3,width=0.15)
      ## dog
      %>% phylopic("4d83a0cd-cf06-4a32-9a5a-0a6b644158c1",color="#FFAACC",alpha=0.3,node=4,width=0.15)
      ## horse
      %>% phylopic("071ee517-a0f1-4d19-aa29-812b9f86cb53",color="#22BBBB",alpha=0.3,node=5,width=0.15)
      ## pig
      %>% phylopic("008d6d88-d1be-470a-8c70-73625c3fb4fb",color="#33AA11",alpha=0.3,node=6,width=0.12)
      ## sheep -- this is a male, the sample is female, but there aren't any females in phylopic
      %>% phylopic("cbea1964-89bf-4ce1-99b0-58fe0f9170b3",color="#004411",alpha=0.3,node=7,width=0.18)
      ## cow
      %>% phylopic("415714b4-859c-4d1c-9ce0-9e1081613df7",color="#881111",alpha=0.3,node=8,width=0.12)
      ## s carmeli
      %>% phylopic("287c58ad-009b-4878-9a9e-59fd7e0fe66f",color="#881188",alpha=0.3,node=9,width=0.1)
      ## n galili
      %>% phylopic("287c58ad-009b-4878-9a9e-59fd7e0fe66f",color="#888888",alpha=0.3,node=10,width=0.1)
      ## mouse
      %>% phylopic("6b2b98f6-f879-445f-9ac2-2c2563157025",color="#BB11CC",alpha=0.3,node=11,width=0.18)
      ## bonobo
      %>% phylopic("f06b4dd0-094e-4f37-9105-25fdb4eb1b02",color="#1111FF",alpha=0.3,node=12,width=0.15)
      ## spider monkey
      %>% phylopic("aceb287d-84cf-46f1-868c-4797c4ac54a8",color="#FF11FF",alpha=0.3,node=14,width=0.25)
      ## human
      %>% phylopic("acf1cbec-f6ef-4a82-8ab5-be2f963b93f5",color="#AAAA11",alpha=0.3,node=13,width=0.4)
      )
@ 
\caption{Species tree for organisms studied showing the relationship
  between species. Silhouettes of species were obtained from
  \url{http://phylopic.org}. }
\label{fig:species_tree}
\end{figure}

\subsection{Comparative Analysis of Gene Expression}
\label{sec:comp-analys-gene}

1:1 orthologs in were determined using
Ensembl v.80 gene trees via Biomart using the biomaRt R package
\cite{Durinck.ea2009:Mappingidentifiersforintegration}. %
A total of \Sexpr{combined.fpkm[species!="homo sapiens" & mean_fpkm >=
  1,length(unique(human_name))]} 1:1 orthologous genes were expressed
in at least one species.

Significant shifts in expression levels between groups of species were
calculated using Student's t test on 1:1 orthologs using the mean of
each species to avoid large biases towards samples with $n \gg 1$ and
considered significant if the \ac{FDR} as estimated by the Benjamini
and Hochberg procedure \cite{Benjamini1995:bh_procedure} was
$\le 0.05$. For \human and \mouse lineage specific calculations, a
type-III anova was calculated using
lme4~\cite{Bates.ea2015:FittingLinearMixed-EffectsModels} with each
species having separate random effects. Shifts in expression levels
are reported with $\log_2$ Fold-Change (FC), rounded to the nearest
integer.

Significant shifts in expression levels which correlate with placenta
morphology (shape, interdigitation, and intimacy) were identified
using ANOVA on generalized linear models with the $\log$ \ac{FPKM} as
the response variable and the morphology as the predictor variable.
Placenta morphology with a definable order (interdigitation and
intimacy) were additionally analyzed using proportional odds logistic
regression using the MASS package in R~\cite{Venables2002:MASS}.


\subsection{Gene Set and Pathway Enrichment Analysis}

Gene sets and pathways in were obtained from MSigdb v5.1
\cite{Liberzon.ea2011:MolecularsignaturesdatabaseMSigDB} and Gene
Ontology to form a superset of available pathway databases. We
calculated the over-representation of gene lists in these gene sets
using Fisher's exact test and corrected for multiple testing by
estimating the \ac{FDR} using the Benjamini and Hochberg procedure
\cite{Benjamini1995:bh_procedure}.

\subsection{Sequences, FPKMs, and Code}

All of the code used to produce every analysis presented in this paper
is available at
\url{https://github.com/uiuc-cgm/placenta-viviparous.git}. All of the
sequences used are deposited in SRA, and new sequences for this work
can be obtained under the series
\href{http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE79121}{GSE79121}
(Supplemental \cref{tab:srr_numbers}). The mean \ac{FPKM} for gene in each
species with ortholog annotations are located in the supplemental data
file all\_species\_mean\_fpkm.txt.xz and the per-sample \ac{FPKM} values are in
all\_species\_per\_sample\_fpkm.txt.xz.

\section{Results}

\subsection{Transcriptome Statistics}

Supplemental \cref{fig:transcriptome_statistics} shows the RNAseq read and
alignment statistics for all species. For transcriptomes sequenced
with 76 bp %
%
(\Sexpr{species(star.logs.wide[input.read.length > 70 & input.read.length < 80,unique(species)])}),%
%
the number of trimmed reads ranged from %
\Sexpr{star.logs.wide[input.read.length > 70 & input.read.length < 80,min(as.numeric(input.reads))]} 
%
to %
\Sexpr{star.logs.wide[input.read.length > 70 & input.read.length < 80,max(as.numeric(input.reads))]}
with an average of \Sexpr{star.logs.wide[input.read.length > 70 & input.read.length < 80,mean(`% good reads`)]}\% of reads 
mapping to the respective reference genome (\cref{tab:ref_genome_aligned_species}). 
%
For transcriptomes sequenced with 150 bp %
% The mismatch here is because we're reporting trimmed reads here, not
% full length reads
(\Sexpr{species(star.logs.wide[input.read.length > 120 & input.read.length < 160,unique(species)])}),
%
the number of trimmed reads ranged from %
\Sexpr{star.logs.wide[input.read.length > 120 & input.read.length < 160,min(as.numeric(input.reads))]} 
%
to %
\Sexpr{star.logs.wide[input.read.length  > 120 & input.read.length < 160,max(as.numeric(input.reads))]}
%
with an average of \Sexpr{star.logs.wide[input.read.length > 120 & input.read.length < 160,mean(`% good reads`)]}\% %
of reads mapping to the
respective reference genome (Supplemental \cref{tab:ref_genome_aligned_species}).
%
%
For transcriptomes sequenced with 100 bp %
% The mismatch here is because we're reporting trimmed reads here, not
% full length reads
(\Sexpr{species(star.logs.wide[input.read.length > 80 & input.read.length < 120,unique(species)])}),
%
the number of trimmed reads ranged from %
\Sexpr{star.logs.wide[input.read.length > 80 & input.read.length < 120,min(as.numeric(input.reads))]} 
%
to %
\Sexpr{star.logs.wide[input.read.length  > 80 & input.read.length < 120,max(as.numeric(input.reads))]}
%
with an average of \Sexpr{star.logs.wide[input.read.length > 80 & input.read.length < 120,mean(`% good reads`)]}\% %
of reads mapping to the
respective reference genome (Supplemental \cref{tab:ref_genome_aligned_species}).
%
%
For transcriptomes sequenced with 35 bp %
% The mismatch here is because we're reporting trimmed reads here, not
% full length reads
(\Sexpr{species(star.logs.wide[input.read.length < 40 ,unique(species)])}),
%
the number of trimmed reads ranged from %
\Sexpr{star.logs.wide[input.read.length < 40 ,min(as.numeric(input.reads))]} 
%
to %
\Sexpr{star.logs.wide[input.read.length  < 40  ,max(as.numeric(input.reads))]}
%
with an average of %
\Sexpr{star.logs.wide[input.read.length < 40, mean(`% good reads`)]}\%
%
of reads mapping to the respective reference genome (Supplemental
\cref{tab:ref_genome_aligned_species}). Read length was not a
significant predictor of unmapped read percentage (Supplemental
\cref{fig:transcriptome_reads_vs_read_length}). The expression of
all genes with \ac{FPKM} $\ge 50$ is shown in Supplemental
\cref{tab:all_ateles_fusciceps_placenta,tab:all_homo_sapiens_placenta,tab:all_pan_paniscus_placenta,tab:all_mus_musculus_placenta,tab:all_spalax_galili_placenta,tab:all_spalax_carmeli_placenta,tab:all_bos_taurus_placenta,tab:all_ovis_aries_placenta,tab:all_sus_scrofa_placenta,tab:all_equus_caballus_placenta,tab:all_canis_familiaris_placenta,tab:all_loxodonta_africana_placenta,tab:all_dasypus_novemcinctus_placenta,tab:all_monodelphis_domestica_placenta}.

<<top.genes.function,echo=FALSE,results="hide">>=
top.genes.table <- function(species.name,n=50) {
    alignment.species <- species.name
    if (alignment.species=="pan paniscus") {
        alignment.species <- "pan troglodytes"
    } else if (alignment.species=="ateles fusciceps") {
        alignment.species <- "homo sapiens"
    } else if (grepl("spalax",alignment.species)) {
        alignment.species <- "mus musculus"
    }
    temp <- 
        combined.fpkm[species==species.name &
                      !(!is.na(human_name) & 
                        human_name %in% housekeeping.genes.superset[,gene_short_name]) &
                      ! (gene_biotype %in% c("Mt_tRNA","Mt_rRNA","rRNA")) &
                      !grepl("((RNA\\d|MT)-|^RP[SL]P?\\d+[ABCD]?[L]?$)",human_name) &
                      !grepl("((RNA\\d|MT)-|^RP[SL]P?\\d+[ABCD]?[L]?$)",human_alignment_symbol) &
                      !grepl("(^[Mm][Tt]-|^R[pP][lsLS]\\d+[abcdABCD]?[lL]?(?:-.+)?$|rRNA$)",
                             gene_short_name)
                      ][order(-mean_fpkm)][,list(name_or_id,
                                                 human_alignment_symbol,
                                                 human_name,mean_fpkm,sd_fpkm,n_fpkm)]
    ## remove any genes which have a null name_or_id; they are probably from unassembled contigs
    temp <- temp[!(is.na(name_or_id)|name_or_id==""),]
    temp <- temp[!duplicated(name_or_id)][1:n]
    temp[,used_alignment:=FALSE]
    temp[is.na(human_name) & !is.na(human_alignment_symbol),
                    used_alignment:=TRUE]
    temp[is.na(human_name),human_name:=human_alignment_symbol]
    setkey(temp,"human_name")
    temp <- 
        hugo.nomenclature[temp][order(-mean_fpkm),
                                list(name_or_id,used_alignment,`Approved Symbol`,`Approved Name`,
                                     mean_fpkm,sd_fpkm,n_fpkm)]
    temp[!is.na(name_or_id),
         name_or_id:=paste0("\\href{http://www.ensembl.org/",
                             capfirst(gsub(" ","_",alignment.species)),
                             "/Gene/Summary?g=",name_or_id,"}{\\textit{",
                             xtable.sanitize.text(name_or_id),"}}")]
    temp[!is.na(`Approved Name`),
         `Approved Name`:=paste0("\\href{http://www.ensembl.org/Homo_sapiens/Gene/Summary?g=",`Approved Symbol`,"}{",xtable.sanitize.text(`Approved Name`),"}")]
    temp[!is.na(`Approved Symbol`),
         `Approved Symbol`:=paste0("\\href{http://www.ensembl.org/Homo_sapiens/Gene/Summary?g=",`Approved Symbol`,"}{\\textit{",xtable.sanitize.text(`Approved Symbol`),"}}")]
    temp[used_alignment==TRUE,`Approved Symbol`:=paste0("(",`Approved Symbol`,")")]
    temp[,used_alignment:=NULL]
    temp[,sem_fpkm:=as.numeric(NA)]
    temp[n_fpkm>1,sem_fpkm:=sd_fpkm/sqrt(n_fpkm)]
    temp[,sd_fpkm:=NULL]
    temp[,n_fpkm:=NULL]
    setnames(temp,c("Gene symbol","Human Symbol","Human Name","FPKM","SEM"))
    if (temp[,all(is.na(`SEM`))]) {
        temp[,`SEM`:=NULL]
    }
    if (species.name=="homo sapiens" || species.name=="ateles fusciceps") {
        temp[,`Human Symbol`:=NULL]
    }
    temp[,`FPKM`:=round(`FPKM`)]
    return(temp)
}
@ 

<<gene.expression,results="hide",echo=FALSE>>=
mentioned.genes <- load_cache(label="save_mentioned_genes",object="saved_mentioned_genes",list())
gene.expression <- function(species.name,gene,complete=FALSE,units=TRUE,mention.only=FALSE) {
    species.name <- gsub("_"," ",species.name)
    mentioned.genes[[paste0(species.name,"_",gene)]] <<-
        list("species.name"=species.name,
             "gene"=gene)
    if (mention.only) {
        return("")
    }
    fpkm <- combined.fpkm[species==species.name &
                              (gene_short_name==gene | 
                               gene_id==gene),
                          mean_fpkm][1]
    if (is.na(fpkm)) {
        stop("FPKM is NA!!!!")
    }
    if (units || complete)
        fpkm <- formatC(fpkm,digits=0,format="f")
    if (units) {
        fpkm <- paste0(fpkm,
                       "\\ \\text{FPKM}")
    }
    if (complete) {
        if (any(grepl("^ENS.+G\\d+$",gene))) {
            fpkm <- paste0(gsub("(ENS)(.+)(G)0*(\\d+)","\\1\\\\linebreak[0]\\2\\\\linebreak[0]\\3...\\4",gene)," = ",
                           fpkm)
        } else {
            fpkm <- paste0("\\textit{",gene,"} = ",
                           fpkm)
        }
    }
    ## if (units || complete) {
    ##     fpkm <- paste0("$",fpkm,"$")
    ## }
    return(fpkm)
}
    
@ 

<<load.placenta.lineage.specific,echo=FALSE,results="hide",cache.extra=tools::md5sum(paste0("../data/",c("placenta_lineage_specific","human_mouse_lineage_specific")))>>=
load("../data/placenta_lineage_specific")
load("../data/human_mouse_lineage_specific")
source("/home/don/projects/xtable/pkg/R/print.xtable.R")

placenta.lineage.specific.table <- function(analysis,output=TRUE,lineage.table=placenta.lineage.specific.analysis,members=5,orderbyfc=FALSE,fdr.cutoff=1) {
    temp.table <- 
        lineage.table[analysis.name==analysis &
                      p.value <= 0.05 &
                      fdr <= fdr.cutoff][order(p.value),
                                       list(human_name,p.value,fdr,fc),]
    temp.table <- data.table(temp.table)
    setkey(temp.table,"human_name")
    temp.table <- 
        hugo.nomenclature[temp.table][,list(`Approved Symbol`,
                                            `Approved Name`,p.value,fdr,fc)]
    setnames(temp.table,
              c("Human Symbol","Human Name","p (t-test)","FDR","Fold Change")
              )
    setkey(temp.table,"Human Symbol")
    if (output) {
        display.table <- temp.table[`FDR`<=fdr.cutoff,][order(`p (t-test)`),]
        if (orderbyfc) {
            display.table <- temp.table[order(-abs(`Fold Change`)),]
        }
        display.table <- display.table[1:members,]

        display.table <-
            display.table[,`Human Symbol`:=paste0("\\textit{",xtable.sanitize.text(`Human Symbol`),
                                                  "}")]
        display.table[,`Human Name`:=NULL]
        # setnames(display.table,"Human Symbol","\\multicolumn{1}{p{0.5in}}{Human Symbol}")6
        setnames(display.table,"Fold Change","$\\text{log}_2$ FC")
        print(xtable(display.table,
                     display=c("s","s","e","g","f"),
                     align=c("c","p{0.5in}","r","r","r")
                     ),
              booktabs=TRUE,
              floating=FALSE,
              math.style.exponents=TRUE,
              sanitize.text.function=genes.table.sanitize,
              include.rownames=FALSE)
        
    }
    return(invisible(temp.table[order(`p (t-test)`),]))
}
output.top.genes.table <- function(data,output=TRUE) {
    if (output) {
        table.display <- c("s","s","s","d")
        table.align <- c("c","c","p{3in}","r")
        if (any(colnames(data) %in% "SEM"))  {
            table.display <- c(table.display,"d")
            table.align <- c(table.align,"r")
        }
        if (any(colnames(data) %in% "Human Symbol"))  {
            table.align <- c("c",table.align)
            table.display <- c("s",table.display)
        }
        print(xtable(data,
                     display=table.display,
                     align=table.align
                     ),
              booktabs=TRUE,
              floating=FALSE,
              sanitize.text.function=genes.table.sanitize,
              include.rownames=FALSE)
    }
}

@ 

<<mentioned.genes.table.functions,echo=FALSE,results="asis">>=
mentioned.genes.table <- list()
if (length(mentioned.genes)==0) {
    mentioned.genes.table[["No genes"]] <-
        data.table(species="No genes",
                   name_or_id="No genes",
                   human_alignment_symbol="NOGENE",
                   human_name="NONAME",
                   mean_fpkm = 0,
                   sd_fpkm = 1,
                   n_fpkm = 2)
}
for (entry in names(mentioned.genes)) {
    mentioned.genes.table[[entry]] <-
        combined.fpkm[species==mentioned.genes[[entry]][["species.name"]] &
                      (gene_short_name==mentioned.genes[[entry]][["gene"]] | 
                       gene_id==mentioned.genes[[entry]][["gene"]]),
                      list(species,
                           name_or_id,
                           human_alignment_symbol,
                           human_name,
                           mean_fpkm,
                           sd_fpkm,
                           n_fpkm)]
}
mentioned.genes.table <-
    rbindlist(mentioned.genes.table)
mentioned.genes.table[,used_alignment:=FALSE]
mentioned.genes.table[is.na(human_name) & !is.na(human_alignment_symbol),
                      used_alignment:=TRUE]
mentioned.genes.table[is.na(human_name),human_name:=human_alignment_symbol]
setkey(mentioned.genes.table,"human_name")
mentioned.genes.table <- 
    hugo.nomenclature[mentioned.genes.table][,
                                             list(species,
                                                  name_or_id,
                                                  used_alignment,
                                                  `Approved Symbol`,
                                                  `Approved Name`,
                                                  mean_fpkm,sd_fpkm,n_fpkm)]
mentioned.genes.table[,alignment.species:=species]
mentioned.genes.table[alignment.species=="pan paniscus",alignment.species:="pan troglodytes"]
mentioned.genes.table[alignment.species=="ateles fusciceps",alignment.species:="homo sapiens"]
mentioned.genes.table[grepl("spalax",alignment.species),alignment.species:="mus musculus"]
mentioned.genes.table[!is.na(name_or_id),
                      name_or_id:=paste0("\\href{http://www.ensembl.org/",
                                         capfirst(gsub(" ","_",alignment.species)),
                                         "/Gene/Summary?g=",name_or_id,"}{\\textit{",
                                         gsub("(ENS[^0-9]+)","\\1\\\\linebreak[0]",
                                              xtable.sanitize.text(name_or_id)),"}}")]
mentioned.genes.table[!is.na(`Approved Name`),
                      `Approved Name`:=paste0("\\href{http://www.ensembl.org/",
                                              "Homo_sapiens/Gene/Summary?g=",
                                              `Approved Symbol`,"}{",
                                              xtable.sanitize.text(`Approved Name`),
                                              "}")]
mentioned.genes.table[!is.na(`Approved Symbol`),
                      `Approved Symbol`:=paste0("\\href{http://www.ensembl.org/",
                                                "Homo_sapiens/Gene/Summary?g=",
                                                `Approved Symbol`,
                                                "}{\\textit{",
                                                xtable.sanitize.text(`Approved Symbol`),
                                                "}}")]
mentioned.genes.table[used_alignment==TRUE,`Approved Symbol`:=paste0("(",`Approved Symbol`,")")]
mentioned.genes.table[,sem_fpkm:=as.numeric(NA)]
mentioned.genes.table[n_fpkm>1,sem_fpkm:=sd_fpkm/sqrt(n_fpkm)]
mentioned.genes.table[,used_alignment:=NULL]
mentioned.genes.table[,alignment.species:=NULL]
mentioned.genes.table[,sd_fpkm:=NULL]
mentioned.genes.table[,n_fpkm:=NULL]
mentioned.genes.table[,`Approved Name`:=NULL]
### order by species and then by FPKM
setorder(mentioned.genes.table,`Approved Symbol`,species,-mean_fpkm)
### italicize species names
mentioned.genes.table[,species:=paste0("\\textit{",gsub("^(.)\\S+","\\U\\1.",species,perl=TRUE),"}")]
setnames(mentioned.genes.table,c("Species","Gene Symbol",
                                 "Human Symbol",
                                 ## "Human Name",
                                 "FPKM","SEM"))
mentioned.genes.table[,`FPKM`:=round(`FPKM`)]

output.mentioned.genes.table <- function(subset.table=quote(TRUE),
                                         human.symbol=TRUE) {
    mgt.subset <- mentioned.genes.table[eval(subset.table)]
    mgt.display <- c("s","s","s","s","d","d")
    mgt.align <- c("c","c",
                   ">{\\raggedright \\centering}p{0.5in}",
                   ">{\\raggedright \\centering}p{0.5in}",
                   ##">{\\raggedright}p{4in}",
                   "r","r")
    if (!human.symbol) {
        mgt.subset[,`Human Symbol`:=NULL]
        mgt.display <- mgt.display[-4]
        mgt.align <- mgt.align[-4]
        mgt.align[3] <- ">{\\raggedright \\centering}p{1.2in}"
    }
        
    print(xtable(mgt.subset,
                 display=mgt.display,
                 align=mgt.align
                 ),
          math.style.exponents=TRUE,
          booktabs=TRUE,
          floating=FALSE,
          sanitize.text.function=genes.table.sanitize,
          include.rownames=FALSE)
}
@ 

% \subsubsection{Per-species Gene Set and Pathway Enrichment}

We performed \ac{gsea} on both the top 1\% of transcripts by \ac{FPKM}
(Supplemental \cref{tab:go_terms_enriched_top_one_ateles_fusciceps,tab:go_terms_enriched_top_one_homo_sapiens,tab:go_terms_enriched_top_one_pan_paniscus,tab:go_terms_enriched_top_one_mus_musculus,tab:go_terms_enriched_top_one_spalax_galili,tab:go_terms_enriched_top_one_spalax_carmeli,tab:go_terms_enriched_top_one_bos_taurus,tab:go_terms_enriched_top_one_ovis_aries,tab:go_terms_enriched_top_one_sus_scrofa,tab:go_terms_enriched_top_one_equus_caballus,tab:go_terms_enriched_top_one_canis_familiaris,tab:go_terms_enriched_top_one_loxodonta_africana,tab:go_terms_enriched_top_one_dasypus_novemcinctus,tab:go_terms_enriched_top_one_monodelphis_domestica}) and the top
100 transcripts by \ac{FPKM} (Supplemental
\cref{tab:go_terms_enriched_ateles_fusciceps,tab:go_terms_enriched_homo_sapiens,tab:go_terms_enriched_pan_paniscus,tab:go_terms_enriched_mus_musculus,tab:go_terms_enriched_spalax_galili,tab:go_terms_enriched_spalax_carmeli,tab:go_terms_enriched_bos_taurus,tab:go_terms_enriched_ovis_aries,tab:go_terms_enriched_sus_scrofa,tab:go_terms_enriched_equus_caballus,tab:go_terms_enriched_canis_familiaris,tab:go_terms_enriched_loxodonta_africana,tab:go_terms_enriched_dasypus_novemcinctus,tab:go_terms_enriched_monodelphis_domestica}); common processes
include protein transport and viral transcription/expression
(Supplemental \cref{tab:go_terms_all_species}).

\subsection{Pre-eclampsia \human Lineage Specific Genes}

\begin{table}[tbh]
  \footnotesize
<<human.lineage.specific,echo=FALSE,results="asis">>=
hum.all.lin.spec <-
    placenta.lineage.specific.table("hum.all",
                                    lineage.table=human.mouse.lineage.specific.analysis,
                                    members=40,
                                    orderbyfc=TRUE,
                                    fdr.cutoff=0.001
                                    )
setkey(hum.all.lin.spec,"Human Symbol")
@ 
\caption{\lineagetableexplanationtypeiii{\human}{all other analyzed species}}
\label{tab:human_lineage_specific}
\end{table}

% \DEW{Add gene names}

Using a type-III ANOVA to account for repeated testing of
\human and \mouse (see \cref{sec:comp-analys-gene} for details), we identified multiple genes which are
differentially expressed in the \human lineage in comparison to all
other lineages (\cref{tab:human_lineage_specific}, Supplemental
\cref{tab:human_lineage_specific_all}). These genes include multiple
genes which are thought to be involved in pre-eclampsia. Notably,
\ac{CRH} (FDR=\Sexpr{hum.all.lin.spec["CRH",`FDR`]},
FC=\Sexpr{hum.all.lin.spec["CRH",`Fold Change`]}, Supplemental
\cref{fig:crh_evolution_tree}), has been shown to be more highly
expressed in placentas from pre-eclamptic pregnancies
\cite{Petsas.ea2012:Aberrantexpressionofcorticotropin-releasing,Paiva.ea2011:MeasurementofmRNAtranscripts},
where it is hypothesized to activate decidual macrophages. Likewise,
\ac{KISS1} (\Sexpr{hum.all.lin.spec["KISS1",`FDR`]},
FC=\Sexpr{hum.all.lin.spec["KISS1",`Fold Change`]}, Supplemental
\cref{fig:kiss1_evolution_tree}) also shows increased expression in
pre-eclamptic placentas, and is
involved in modulating trophoblast invasion~%
\cite{Vazquez-Alaniz.ea2011:Comparativeexpressionprofilesfor}. Two other genes,
\ac{PAPPA} (\Sexpr{hum.all.lin.spec["PAPPA",`FDR`]},
FC=\Sexpr{hum.all.lin.spec["PAPPA",`Fold Change`]} Supplemental
\cref{fig:pappa_evolution_tree}) and \ac{ADAM12}
(\Sexpr{hum.all.lin.spec["ADAM12",`FDR`]},
FC=\Sexpr{hum.all.lin.spec["ADAM12",`Fold Change`]}, Supplemental
\cref{fig:adam12_evolution_tree}) are also differentially expressed
but are negatively correlated with pre-eclampsia~%
\cite{Spencer.ea2008:ADAM12sinmaternalserum}.


\subsection{Core Placenta Transcriptome}

\begin{figure}[tbh]
<<core.placenta.transcriptome,fig.width=9/2.5,fig.height=8,out.width="90mm",out.height="0.8\\textheight",echo=FALSE>>=
cptl <-
    core.placenta.transcriptome.long[!grepl("(min|median|max|all)_expression|all_but_",
                                            Species) & housekeeping==FALSE,]
cptl <- cptl[,Species:=factor(gsub("^([^ ])[^ ]* +([^ ]*)","\\1. \\2",capfirst(Species)),
                              levels=gsub("^([^ ])[^ ]* +([^ ]*)","\\1. \\2",
                                          capfirst(names(species.names))))]

print(ggplot(cptl[FPKM > 0.1,],aes(x=human_name,y=FPKM))+
      geom_boxplot(outlier.shape=NA)+
      scale_y_log10(labels=trans_format("log10", math_format(10^.x)))+
      geom_point(aes(color=Species,shape=Species))+
      scale_shape_manual("Species",
                         guide=guide_legend(#nrow=1,
                                            keywidth=unit(3,"mm"),
                                            keyheight=unit(3,"mm"),
                                            label.theme=element_text(face="italic",
                                                                     angle=0,
                                                                     size=7),
                                            title.position="top"
                                            ),
                         values=0:18)+
      # scale_color_discrete(NULL)+
      coord_flip()+
      xlab("Human Ortholog Name")+
      ylab("Gene Expression (FPKM)")+
      theme_bw() +
      theme(text=element_text(size=9),
            legend.position=c(0.99,0.41),
            axis.text.y=element_text(face="italic",size=8),
            legend.margin=unit(0,"mm"),
            legend.justification=c(1,1),
            plot.margin=unit(c(0,0,0,0),"mm")
            ))
@ 
\caption{Core Placenta Transcriptome.
  \Sexpr{core.placenta.transcriptome.long[housekeeping==FALSE,length(unique(human_name))]}
  1:1 \human ortholog genes expressed (\ac{FPKM}$ \ge 10$) in the
  placenta of all 13 Eutherians
  studied with housekeeping genes
  \cite{Eisenberg.ea2013:Humanhousekeepinggenesrevisited} removed
  ordered (from top to bottom) by median expression. Supplemental
  \cref{fig:core_placenta_transcriptome_shape} shows the shape of the
  distribution of the number of genes in the core placenta
  transcriptome if different FPKM and species restriction criteria are
  utilized.}
\label{fig:core_placenta_transcriptome}
\end{figure}


\begin{figure}
<<core.placenta.transcriptome.distribution,fig.width=9/2.5,fig.height=9/2.5,out.width="90mm",out.height="0.8\\textheight",echo=FALSE,message=FALSE>>=
num.species <- star.logs.wide[,length(unique(species))]
calculate.distribution.shape <- 
    function(min.vector) {
        min.vector <- sort(min.vector,decreasing=TRUE)
        temp <- data.table(FPKM=min.vector,num_genes=1:length(min.vector))
        temp <- temp[order(-num_genes)][!duplicated(FPKM)]
        return(temp)
    }

cptg.nh <- core.placenta.transcriptome.genes.shape[!(human_name %in% housekeeping.genes.superset[,gene_short_name])]

cpt.distribution.shape <- 
    lapply(c("all_expression",
             "all_but_one_expression",
             "all_but_two_expression",
             "all_but_three_expression",
             "all_but_four_expression",
             "all_but_metatherian_expression"
             ),
           function(x){
               calculate.distribution.shape(cptg.nh[,x,with=FALSE][[1]])[,`Min. Exp. in`:=x]
           })
fix.expression.names <- function(x){
    x <- gsub("^all_but_metatherian_expression$",
              paste0("All ",num.species-1," Eutherians"),
              x
              )
    x <- gsub("^all_expression$",
              paste0(num.species," of ",num.species," Species"),
              x
              )
    nc.levels <- c(paste0("All ",num.species-1," Eutherians"),
                   paste0(num.species," of ",num.species," Species")
                   )
    ## number coutner
    nc <- c("one"=1,"two"=2,"three"=3,"four"=4)
    for (a in names(nc)) {
        x <- gsub(paste0("^all_but_",a,"_expression$"),
              paste0(num.species-nc[a]," of ",num.species," Species"),
              x
              )
        nc.levels <-
            c(nc.levels,
              paste0(num.species-nc[a]," of ",num.species," Species"))
    }
    return(factor(x,levels=nc.levels))
}

cpt.distribution.shape <- 
    rbindlist(cpt.distribution.shape)

cpt.distribution.shape <- 
    cpt.distribution.shape[,`Min. Exp. in`:=
                                fix.expression.names(`Min. Exp. in`)]
setnames(cpt.distribution.shape,
         "FPKM",
         "Expression (FPKM)")

print(ggplot(cpt.distribution.shape[`Expression (FPKM)` > 0.1,],
             aes(x=`Expression (FPKM)`,color=`Min. Exp. in`,y=num_genes))
      + geom_line(lwd=1)
      + xlab("Minimum Expression cutoff (FPKM)")
      + ylab("Number of Genes Meeting FPKM Cutoff")
      + scale_y_log10(breaks = trans_breaks("log10", function(x) 10^x),
                      labels = trans_format("log10", math_format(10^.x)))
      + scale_x_log10(breaks = trans_breaks("log10", function(x) 10^x),
                      labels = trans_format("log10", math_format(10^.x)))
      + geom_vline(xintercept=10,linetype="dotted")
      + geom_segment(mapping=aes(y=V1-ifelse(V1<80,20,0),
                         yend=V1,
                         xend=10,
                         x=sapply(V1,
                                  function(x){
                                      cpt.distribution.shape[num_genes > x,
                                                             max(`Expression (FPKM)`)*4]
                                  }),
                         color=`Min. Exp. in`),
                     data=cpt.distribution.shape[`Expression (FPKM)` > 10,
                                              max(num_genes),`Min. Exp. in`],
                     arrow=arrow(length=unit(0.5,"cm")),
                     show.legend=FALSE
                     )
      + geom_text(mapping=aes(x=sapply(V1,
                                       function(x){
                                           cpt.distribution.shape[num_genes > x,
                                                                  max(`Expression (FPKM)`)*4.1]
                                       }),
                      y=V1-ifelse(V1<80,20,0),
                      label=paste0(V1," genes"),override=TRUE),
                  data=cpt.distribution.shape[`Expression (FPKM)` > 10,
                      max(num_genes),`Min. Exp. in`],
                  show.legend=FALSE,
                  hjust=0,
                  size=8*(5/14)
                  )
      + geom_segment(mapping=aes(yend=V1,
                         y=100,
                         xend=1,
                         x=1,
                         color=`Min. Exp. in`),
                     data=cpt.distribution.shape[`Expression (FPKM)` > 1 &
                                                 `Min. Exp. in` == 'All 13 Eutherians',
                                                 max(num_genes),`Min. Exp. in`],
                     arrow=arrow(length=unit(0.5,"cm")),
                     show.legend=FALSE
                     )
      + geom_text(mapping=aes(x=1,
                      y=100,
                      label=paste0(V1," genes"),
                      override=TRUE),
                  angle=90,
                  data=cpt.distribution.shape[`Expression (FPKM)` > 1 &
                                              `Min. Exp. in` == 'All 13 Eutherians',
                                              max(num_genes),`Min. Exp. in`],
                  show.legend=FALSE,
                  hjust=1,
                  size=8*(5/14)
                  )
      + guides(color=guide_legend(title=paste0("Minimum\nExpression in"),
                                  keywidth=unit(3,"mm"),
                                  keyheight=unit(3,"mm"),
                                  title.hjust=0.5,
                                  label.position="right",
                                  label.theme=element_text(size=7,angle=0)
                                  ))
      + theme_bw()
      + theme(text=element_text(size=9),
              legend.position=c(0.99,0.99),
              legend.margin=unit(0,"mm"),
              legend.justification=c(1,1),
              plot.margin=unit(c(0,0,0,0),"mm")
              )
      )
@ 
\caption{Distribution shape of placenta core genes. The vertical axis
  shows the number of 1:1 orthologous genes included in the core
  placenta transcriptome if %
  \Sexpr{cpt.distribution.shape[,levels(`Min. Exp. in`)[1]]}
  %
  (or %
  \Sexpr{cpt.distribution.shape[,levels(`Min. Exp. in`)[2]]},
  % 
  etc.) have a minimum expression in FPKM equal to the horizontal axis.
  For example, if the cutoff is set to 10 (vertical black line), 
  % 
  \Sexpr{cpt.distribution.shape[`Expression (FPKM)` > 10 
    & `Min. Exp. in` == 'All 13 Eutherians',max(num_genes)]}
  %
  genes would be included if the expression of a set of 1:1 orthologs of all 13 Eutherians must be at
  least 10 FPKM, %
  \Sexpr{cpt.distribution.shape[`Expression (FPKM)` > 10 
    & `Min. Exp. in` == '13 of 14 Species',max(num_genes)]}
  %
  1:1 orthologs would be included if 13 of 14 species have an
   expression of at least 10 FPKM, and 
  \Sexpr{cpt.distribution.shape[`Expression (FPKM)` > 10 
    & `Min. Exp. in` == '10 of 14 Species',max(num_genes)]}
  genes would be included if 10 of 14 species
  must have an FPKM of at least 10.}
\label{fig:core_placenta_transcriptome_shape}
\end{figure}


The
\Sexpr{core.placenta.transcriptome.long[housekeeping==FALSE,length(unique(human_name))]}
genes comprising the core placenta transcriptome are shown in
\cref{fig:core_placenta_transcriptome} ordered by their median
transcription level in \ac{FPKM}. We considered a gene to be core to
the placenta transcriptome if a 1:1 ortholog of that gene was
expressed with $\text{FPKM} \ge 10$ in all eutherian species studied
(excluding the metatherian, \opossum) the and was not a human housekeeping
gene according to
\citet{Eisenberg.ea2013:Humanhousekeepinggenesrevisited}. Genes which
meet this criteria but are housekeeping genes are depicted in
Supplemental \cref{fig:core_placenta_transcriptome_housekeeping}.
\Cref{fig:core_placenta_transcriptome_shape} shows the number of genes
which would be included in the core placenta set if the \ac{FPKM}
cutoff was set differently; for example, choosing an \ac{FPKM} cutoff
of 1 yields %
\Sexpr{cpt.distribution.shape[`Expression (FPKM)`>=1 & `Min. Exp. in`
  == '14 of 14 Species',max(num_genes)]} %
genes in the core transcriptome. %
In contrast to the shape of the number of genes included in the core
placenta transcriptome, the set of \human housekeeping genes expressed
all placentas is flatter (Supplemental
\cref{fig:core_housekeeping_transcriptome_shape}), and includes more
of the total fraction of genes. However, there is still a large
difference between the set of housekeeping genes which are expressed
in \human and those which are expressed in all placentas.

Supplemental \cref{tab:core_placenta_go_terms} lists the significant
enrichment of \ac{GO} terms in the core placenta transcriptome;
notable terms include those involved in cytoskeletal structure,
protein transport, membrane protein structure, and inflammation
resolution. Increasing the permissiveness of the core placenta
transcriptome by only requiring 10 of 14 species to express 1:1
orthologs with FPKM $>10$ and results in the same terms also being
significantly enriched, along with additional terms (Supplemental
\cref{tab:core_placenta_go_terms}). Supplemental
\cref{tab:core_placenta_pathway_analysis} shows the pathways with
significant and suggestive enrichment in the core placenta
transcriptome.

\subsubsection{Annexin Complexes}

Multiple components of annexin complexes are included in the core
transcriptome, including \ac{ANXA2}, \ac{ANXA1},
\ac{S100A11}, \ac{S100A10}. ANXA1 is involved in the
resolution of the inflammation response through inhibition of
phospholipase A2 activity and signaling through the formyl peptide
receptor family, and has previously been hypothesized to be important
for the maintenance of the anti-inflammatory state during pregnancy
\cite{Hutchinson.ea2011:Molecularregulatorsofresolution}. ANXA2 is
known to interact with S100A10, S100A6, S100A11, and S100A4
\cite{Liu.ea2015:AnnexinA2complexeswith}, and there is evidence for
its interaction with S100P as well; it is involved in cell-cell
interaction as well as vesicle trafficking and von Willibrand Factor
secretion. The expression regulation of another core gene,
\ac{TIMP3}, has been implicated in pre-eclampsia
\cite{Ma.ea2014:Down-regulationofTIMP3leads,Xiang.ea2013:PromoterhypomethylationofTIMP3}.

Another annexin which is highly expressed in all term placentas is
\ac{ANXA5}, which was originally identified as a an anti-coagulant
protein, but is also involved in syncytium formation, membrane repair,
and reduction of spontaneous
abortion~\cite{Ueki.ea2012:Lossofmaternalannexin}. Other notable
syncytiotrophoblast fusion proteins, Syncytin-1 (\ac{ERVW-1}) and -2
(\ac{ERVFRD-1}) were not present in the core placenta, as they (or
genes similar to them) are not highly expressed in \dog, \bonobo, or \sheep,
nor was the Syncytin-2 receptor, \ac{MFSD2A} (\ac{FPKM} $\ge 10$
only in \human, \mouse, and \dog).

\subsubsection{Invasion: EGF and Actin Y in the Core Placenta}

The core placenta transcriptome is significantly enriched in genes
which are involved in multiple biological pathways, including multiple
pathways related to translation (ribosome %
%
(\Sexpr{placpath("^RIBOSOME","Kegg")})
, translation %
(\Sexpr{placpath("^TRANSLATION","Reactome")})
%
, signal
recognition particle targeting to the rough endoplasmic reticulum %
(\Sexpr{placpath("^SRP DEPENDENT","Reactome")})
%
,
etc.) (Supplemental \cref{tab:core_placenta_pathway_analysis}). Beyond
translation, the next most abundant set of significantly enriched
molecular pathways are related to the migration of the trophoblast
into the endometrium, including the EGFR1 pathway %
(\Sexpr{placpath("^EGFR1","Netpath")})
%
and the Actin Y pathway %
(\Sexpr{placpath("^ACTINY","Biocarta")})
%
. Two other significant pathways, the mPR/Progesterone pathway
and Rho cell motility pathways, feed into the Actin Y pathway as well.
Supplemental \cref{fig:egf_kegg_pathway} shows the EGF pathway in
humans with the median expression in FPKM of genes in the pathway
shown. Notably, many of the genes on the GRB2âMYC pathway are strongly
expressed; their activation leads to an increase in migration and
invasion in response to EGF binding to
EGFR1~\cite{Gupta.ea2016:CellSignalingPathwaysInvolved}. The Actin
cytoskeleton pathway is depicted in Supplemental
\cref{fig:kegg_actin}, which shows that \ac{ITGB1} and its binding
protein \ac{ITGB1BP1} as well as \ac{FN1} are highly expressed, which
activates Rac. This potentially leads to the formation of focal adhesions
and lamellipodia formation through inhibition of GSN and
HSPC300. Other components of cell-cell adhesion, including \ac{F11R}
which forms tight junctions in epithelia, the proteoglycans \ac{SDC4} and
\ac{LUM}, and \ac{LAMC1}.

% SPINT2 -- Serine Peptidase inhibitor

% SLC3A2 -- this is probably actually a housekeeping gene; it's
% involved in amino acid transport

% SQSTM1 -- NFÎºB signaling (immune supression?)

% KRTCAP2  N-glycosylation (probably housekeeping)

% NRDG1 -- interacts with HSP90

% SDC4 -- proteoglycan (syndecan) cell adhesion

% LAMC1 -- laminin

% LUM -- proteoglycan

% VDAC1 -- mitochondrial gene?

% F11R -- tight junctions

% SERPINH1? with TIMP3 
% http://www.proteinatlas.org/ENSG00000149257-SERPINH1/tissue/placenta

\subsection{Placenta Morphology and Genetics}

We correlated different aspects of placenta morphology, including
shape, intimacy, and interdigitation with the mRNA transcription
profiles of all 1:1 human orthologs with \ac{FPKM} greater than 10 in
one species, and \ac{FPKM} greater than 1 at least two species.
Supplemental \cref{tab:placental_classification} lists the shape,
intimacy, and interdigitation of each species analyzed.

\subsubsection{Placenta Shape}

Supplemental \cref{tab:shape_genes} shows the 50 genes which are most
significantly associated with the shape of the placenta, as
well as the shape in which the difference is most significant. The
most significant genes which were associated with diffuse placenta
shape include \ac{EREG}, a ligand of EGFR, \ac{UNC93A}, a
gene similar to \ac{UNC93B1} which is involved in the balancing between
Tlr7 and Tlr9 immune response in
mice~\cite{Brinkmann.ea2007:interactionbetweenERmembrane}, and
\ac{WNT5A}, which is a member of the Wnt ligand family whose
signaling pathway has been implicated in motility in human trophoblast
cells~\cite{Gupta.ea2016:CellSignalingPathwaysInvolved}.

\subsubsection{Degree of Intimacy and Interdigitation}

For the degree of intimacy and interdigitation, we saw multiple genes
which were associated with different levels of interdigitation and
intimacy (Supplemental
\cref{tab:intimacy_genes,tab:interdigitation_genes}) using an
unordered factor approach, but when we used an ordered factor
analysis, none of these genes were retained (Supplemental
\cref{tab:intimacy_genes_polr,tab:interdigitation_genes_polr}). We
suspect that this is due to the fact that the ancestral state of all
eutherians is a labyrinthine, hemochorial
placenta~\cite{Wildman.ea2006:Evolutionofmammalianplacenta}. This
implies that the transcriptome of intermediately invasive or
interdigitated placentas need not be intermediate between the
transcriptomes of less intimate (folded/lamellar) or interdigitated
placentas (epitheliochorial) and most intimate (hemochorial) or
interdigitated placentas (labyrinthine) because these morphologies
have evolved independently. An alternative is that the limited number
of species with placentas of intermediate interdigitation and intimacy
in this study limits the ability to detect genes which whose
expression correlates with degree of intimacy and interdigitation.
Future analyses of additional placentas with intermediate
interdigitation and intimacy should better illuminate the pathways
which are responsible for these placental morphologies.

\PP{Introduce the characteristics earlier in the introduction}

\DLA{They're in the first paragraph of the introduction, but I've also
  referred to them now in the first subsection here.}


\section{Discussion}


\subsection{\human Specific Expression Shifts and the Evolution of Pre-eclampsia}

The \human placenta is distinctive from many other primates in that it
possesses a deeply invasive trophoblast that penetrates the inner
myometrium of the uterus
\cite{Carter.ea2011:Evolutionofinvasiveplacentation}. It is
hypothesized that this feature is related to the increased demand for
resources by the developing fetus to feed the expanded size of the
human brain
\cite{Carter.ea2011:Evolutionofinvasiveplacentation,Robillard.ea2002:Evolutionaryadaptationstopre-eclampsiaeclampsia}.

The near exclusivity of deep trophoblast invasion in humans has also
led many to speculate on the origin of various disorders, which are
almost exclusively symptomatic of human pregnancy. For example,
pre-eclampsia is a human disease that occurs in 4\% of pregnancies in
the United States
\cite{Chaiworapongsa.ea2014:Pre-eclampsiapart1current}. Pre-eclampsia
can be attributed to the lack of adequate invasion of maternal tissue,
and is noted by the onset of hypertension and proteinuria (or in the
absence of proteinuria, thrombocytopenia, impaired liver function,
renal insufficiency, pulmonary edema, or cerebral or visual
disturbances) %
\cite{Chaiworapongsa.ea2014:Pre-eclampsiapart1current}. Some cases of
pre-eclampsia can develop into eclampsia, a more severe condition
resulting in seizures during pregnancy and death. Lack of invasion in
pre-eclampsia may be attributed to the inefficiency of growth
regulators as well as the inability to sufficiently suppress the
maternal immune response~\cite{Hutchinson.ea2011:Molecularregulatorsofresolution}. %
Although there are speculative cases of eclampsia in great apes, 
%\cite{Baird1981:Eclampsiainlowlandgorilla,Stout.ea1969:Glomerularcapillaryendothelialswelling,Thornton.ea1992:Convulsionsinpregnancyin},
it is exceedingly rare in these species
\cite{Carter.ea2011:Evolutionofinvasiveplacentation}.

The identification of multiple genes (\ac{CRH}, \ac{ADAM12},
\ac{KISS1}, \ac{PAPPA}, \ac{IL1RL1}, and \ac{PLAC1};
\cref{tab:human_lineage_specific}) which are associated with
pre-eclampsia which have increased expression on the \human lineage
may explain the larger proportion of pre-eclampsia observed in humans.
However, there are still genes which are associated with pre-eclampsia
which are also more highly expressed in the \mouse lineage such as
\ac{Afp} and \ac{Gjb2}. Furthermore, the onset of
pre-eclampsia is well before term; all of our placentas are term
placentas and provide limited information about the transcriptome
during the onset of pre-eclampsia.


\subsection{Core placenta transcriptome}

The core placenta transcriptome is enriched in multiple genes which
are critical for the function of the placenta, including vesicular
transport, immune regulation, cell fusion and membrane repair, and
invasion.

\subsubsection{Immune Regulation and Annexins}
The transcriptomes from multiple different species enabled us to
identify the set of expressed genes which are common to all placentas.
Components of the annexin complexes, including
\ac{ANXA2}, \ac{ANXA1}, \ac{S100A11}, and \ac{S100A10}
are expressed in all placentas examined, and argue for the evolution
importance of annexins in placenta function, where they likely
establish maternal-fetal
tolerance\cite{Liu.ea2015:AnnexinA2complexeswith,Hutchinson.ea2011:Molecularregulatorsofresolution,Aarli.ea1997:Suppressiveeffectonlymphoproliferation}.
We suggest that further research into molecular role of these annexin
complexes may illuminate how immune tolerance is maintained during
placentation because these appear to the the major class of immune
modulatory transcripts which are expressed in all placentas analyzed.

\DEW{Why?} \DLA{Clarified why I think this is important to study.}

\subsubsection{Formation of the syncytiotrophoblast}

While Syncytin-1 and -2 (\ac{ERVW-1} and \ac{ERVFRD-1}, respectively) are implicated
in the formation of the syncytiotrophoblast in
\human~\cite{Rawn.ea2008:evolutionregulationandfunction} and genes which are similar to 
\ac{ERVFRD-1} and \ac{ERVW-1} are widely expressed in most of the
placentas analyzed here, they are not expressed in \bonobo, \sheep,
and \dog, even though these mammals form
syncytiotrophoblasts~\cite{Mess.ea2006:Evolutionarytransformationsoffetal}.
Because the formation of syncytiotrophoblast is the stem state of
Eutheria, Afrotheria, Euarchontoglires, and
Laurasiatheria~\cite{Mess.ea2006:Evolutionarytransformationsoffetal},
and syncytiotrophoblast must be continuously replenished
the gene(s) responsible for cytotrophoblast fusion to form
syncytiotrophoblast should be highly expressed in the core
placenta. \ac{ANXA5} is known to be expressed at multiple stages of
placentation, and is capable of repairing membrane defects and
promoting the fusion of
cells~\cite{Carmeille.ea2015:Annexin-A5promotesmembraneresealing} in
the presence of Ca$^{2+}$. While there may be other genes which
are responsible for the positioning of cytotrophoblast cells in close
proximity to start the fusion process, \ac{ANXA5} is likely to
be essential for the completion of fusion and the maintenance of
membrane repair during the massive flux of nutrients and waste in the
placenta \cite{Carmeille.ea2015:Annexin-A5promotesmembraneresealing,Ueki.ea2012:Lossofmaternalannexin}.

\PP{Add refs for fusion}

\DLA{Added the Carmeille.ea2015 and Ueki.ea2012 I have other
  references
  (\cite{Ota.ea2013:ContributionoffetalANXA5,Bouter.ea2015:ReviewAnnexin-A5andcell})
  but I'd exceed the limit for placenta (45)}

% \DLA{Possibility of membrane repair due to maternal immune response as
%   well during this process, as well as repair of membrane due to high
%   secretion and adsoroption of nutrients/waste in the placenta.}

% Cows have BERV-K1 and -K2, but it's not annotated in the ENSEMBL
% build of the cow genome that we have

\subsubsection{Invasion: EGF and CD9 in the Core Placenta}

% \DEW{Write out FDR for enrichment described}

The high expression of components of the EGFR signaling pathway %
(Enrichment \Sexpr{placpath("^EGFR1","Netpath")}) %
in the
core placenta transcriptome (including \ac{GRB2}, \ac{RALB}) as well
as annexins which are involved in the spatio-temporal regulation of
EGF receptor signal transduction (\ac{ANXA1}, \ac{ANXA2}, and
\ac{S100A11} \cite{Poeter.ea2013:DisruptionofannexinA1S100A11}) in the
core placenta is consistent with previous findings showing that
alterations to the EGF signaling system change invasion of trophoblast
cells into the spiral
arteries~\cite{Armant.ea2015:Reducedexpressionofepidermal}. Another
component of the core placenta transcriptome, \ac{CD9}, has been shown
to suppress \human extravillous trophoblast
invasion~\cite{Matsumoto.ea2016:CD9suppresseshumanextravillous}, and
we hypothesize that it is responsible for maintaining the balance
between invasion and attachment in the other placentas examined here
as well.

% \PP{EGF interacts with S100 from the core placenta as well}

% TAGLN2 -- likely to be produced by the placenta as well; possibly
% due to the smooth muscle transition?

\subsection{Conclusion}
    
We analyzed the sequenced transcriptome of
\Sexpr{star.logs[,length(unique(file))]} placentas from
\Sexpr{star.logs[,length(unique(species))]} species spanning the
phylogenetic tree of mammals. This revealed both
lineage-specific differential expression of gene
families as well as a set of core, non-housekeeping genes which are
critical for the proper organization and functioning of the placenta.

The core placenta transcriptome contains genes which are involved in
maternal-fetal tolerance, such as the annexins, and genes responsible
for cell-cell and cell-matrix interactions, such as \ac{LGALS1}.
These genes were almost certainly present in the placenta of the
common mammalian ancestor. We hypothesize that modifications to their
expression or function will compromise the development and
organization of the placenta.

We also identified genes differentially expressed on the \human
lineage which are implicated in pre-eclampsia, including
\ac{ADAM12}, \ac{CRH}, and \ac{KISS1}.

This study has identified many tantalizing molecular pathways which
illuminate the vast diversity of placental morphology and may explain
some placental malfunctions in \human{}. However, the strength of our
hypotheses are limited by our limited data; more placentas from
multiple members of diverse clades and different timepoints are still
needed to come to a more complete understanding of the genomic and
molecular basis for placenta morphology and function. We look forward
to collecting and analyzing such a dataset.



%\begin{backmatter}
  \section*{Competing interests}
  The authors declare that they have no competing interests.
  
  \section*{Author's contributions}
  Don Armstrong analyzed all of the data and wrote the paper. 

  Michael McGowan wrote the paper and performed initial analyses.

  Amy Weckle performed all of the RNA extractions and managed the
  samples.

  Priyadarshini Pantham wrote the paper and performed gene enrichment
  analyses.

  Jason Caravas performed initial analyses.
  
  Dalen Agnew provided samples.
  
  Kurt Benirschke provided samples.
  
  Sue Savage-Rumbaugh provided samples.

  Eviatar Nevo provided samples.

  Chong J. Kim provided samples.

  GÃ¼nter P. Wagner provided samples and wrote the paper.

  Roberto Romero provided samples and conceived the study.

  Derek E. Wildman conceived the study, wrote the paper, and performed
  analyses.

  \section*{Acknowledgements}
  The authors would like to specially acknowledge those groups whose
  publicly available data contributed to the completeness of this
  work.

  \section*{Funding}
  Partial support for this research came from the \textit{Eunice
    Kennedy Shriver} National Institute of Child Health and Human
  Development (NICHD), National Institutes of Health, U.S. Department
  of Health and Human Services (N01-HD-3342), the National Science
  Foundation (BCS-0827546 and BCS-0751508), and a seed grant from the
  Institute for Genomic Biology at the University of Illinois,
  Urbana-Champaign.

\DEW{Need to cite NWM grants, IGB, etc.}

  \DLA{Unclear what the IGB grant is; Derek: which grant did you want
    me to cite? Mentioned seed grant.}

  
%  \printaddresses

%\end{backmatter}

\printbibliography

<<save_mentioned_genes,echo=FALSE,results="hide">>=
saved_mentioned_genes <- mentioned.genes
@ 

\end{document}
