\documentclass[english,12pt]{article}
\usepackage{fontspec}
\setmainfont{FreeSerif}
\setsansfont{FreeSans}
\setmonofont{FreeMono}
\usepackage[letterpaper,left=1in,right=1in,top=1in,bottom=1in,headheight=14pt]{geometry}
\usepackage{url}
\usepackage{fancyhdr}
\usepackage{graphicx}
\usepackage[bf]{caption}
\usepackage{rotating}
\usepackage{wrapfig}
\usepackage{booktabs}
\usepackage{float}
\usepackage{multirow}
\usepackage{setspace}
\usepackage{wrapfig}
% \usepackage{txfonts}
\usepackage{acronym}
\usepackage{array}
\usepackage{dcolumn}
\usepackage{adjustbox}
\usepackage{longtable}
\usepackage{pdflscape}
\usepackage[backend=biber,natbib=true,hyperref=true,style=reading,autocite=inline]{biblatex}
\addbibresource{references.bib}
\usepackage[noblocks,auth-sc]{authblk}
\usepackage[hyperfigures,bookmarks,colorlinks,linkcolor=black,urlcolor=black]{hyperref}
%\usepackage[hyperfigures,bookmarks,colorlinks,citecolor=black,filecolor=black,linkcolor=black,urlcolor=black]{hyperref}
% Floats at end for journals
% \usepackage[nolists,figuresfirst,nomarkers,heads,nofighead,notabhead]{endfloat}
% \AtBeginFigures{\section*{Figure Legends}\renewcommand{\efloatseparator}{\hbox{}}}
% \AtBeginTables{\section*{Tables}\renewcommand{\efloatseparator}{\mbox{}}}
\usepackage[nomargin,inline,draft]{fixme}
\newcommand{\DLA}[1]{\textcolor{red}{\fxnote{DLA: #1}}}
\usepackage[x11names,svgnames]{xcolor}
\usepackage{texshade}
\usepackage{tikz}
\usepackage{nameref}
\usepackage{zref-xr,zref-user}
\usepackage{cleveref}
\newenvironment{narrow}[2]{%
  \begin{list}{}{%
      \setlength{\topsep}{0pt}%
      \setlength{\leftmargin}{#1}%
      \setlength{\rightmargin}{#2}%
      \setlength{\listparindent}{\parindent}%
      \setlength{\itemindent}{\parindent}%
      \setlength{\parsep}{\parskip}}%
  \item[]}{\end{list}}
\def\newblock{\hskip}
\newenvironment{paperquote}{%
  \begin{quote}%
     \it
  }%
  {\end{quote}}
\renewcommand{\textfraction}{0.15}
\renewcommand{\topfraction}{0.85}
\renewcommand{\bottomfraction}{0.65}
\renewcommand{\floatpagefraction}{0.60}
%\renewcommand{\baselinestretch}{1.8}
\newenvironment{enumerate*}%
  {\begin{enumerate}%
    \setlength{\itemsep}{0pt}%
    \setlength{\parskip}{0pt}}%
  {\end{enumerate}}
\newenvironment{itemize*}%
  {\begin{itemize}%
    \setlength{\itemsep}{0pt}%
    \setlength{\parskip}{0pt}}%
  {\end{itemize}}
\oddsidemargin 0.0in 
\textwidth 6.5in
\raggedbottom
\clubpenalty = 10000
\widowpenalty = 10000
% make the afill smaller
\renewcommand\Affilfont{\itshape\small}
\renewcommand\cite[1]{\autocite{#1}}
\pagestyle{fancy}

\newcommand{\smonkey}[0]{\textit{A. fusciceps}\xspace}
\newcommand{\cow}[0]{\textit{B. taurus}\xspace}
\newcommand{\dog}[0]{\textit{C. familiaris}\xspace}
\newcommand{\armadillo}[0]{\textit{D. novemcinctus}\xspace}
\newcommand{\horse}[0]{\textit{E. caballus}\xspace}
\newcommand{\elephant}[0]{\textit{L. africana}\xspace}
\newcommand{\human}[0]{\textit{H. sapiens}\xspace}
\newcommand{\opossum}[0]{\textit{M. domestica}\xspace}
\newcommand{\mouse}[0]{\textit{M. musculus}\xspace}
\newcommand{\galili}[0]{\textit{N. galili}\xspace}
\newcommand{\sheep}[0]{\textit{O. aries}\xspace}
\newcommand{\bonobo}[0]{\textit{P. paniscus}\xspace}
\newcommand{\carmeli}[0]{\textit{S. carmeli}\xspace}
\newcommand{\pig}[0]{\textit{S. scrofa}\xspace}

\newcommand{\genetreeexplanation}[1]{Gene tree and heatmap of
  expression of genes related to #1. The gene tree was obtained from
  Ensembl~\cite{Cunningham.ea2015:Ensembl2015} and genes from species
  not included in this study or having expression by \ac{FPKM} less
  than 1 were excluded from the plotted tree. Heatmap cells in grey
  are from species where there was no expression of the relevant
  transcript (or a species where that transcript does not apply).}


\begin{document}

\makeatletter 
\renewcommand{\thefigure}{S\@arabic\c@figure} 
\makeatother

\makeatletter 
\renewcommand{\thetable}{S\@arabic\c@table} 
\makeatother

\title{Supplemental Material for: The evolution of gene expression in the term placenta of viviparous mammals}

% \maketitle

\listoftables

\listoffigures


<<load.libraries,echo=FALSE,results="hide",warning=FALSE,message=FALSE,error=FALSE,cache=FALSE>>=
opts_chunk$set(dev="cairo_pdf",out.width="\\textwidth",out.height="0.7\\textheight",out.extra="keepaspectratio")
opts_chunk$set(cache=TRUE, autodep=TRUE)
options(device = function(file, width = 8, height = 7, ...) {
  cairo_pdf(tempfile(), width = width, height = height, ...)
})
options(scipen = -2, digits = 2)
library("data.table")
library("ggplot2")
library("reshape2")
library("xtable")
library("grid")
library("gridExtra")
library("ape")
library("ggtree")
library("topGO")
library("org.Hs.eg.db")


@ 

<<source.to.latex,echo=FALSE,results="hide",cache.extra=tools::md5sum(paste0("../common_r_code/",c("to_latex.R","plot_tree_matrix.R","utility_functions.R")))>>=
for (a in paste0("../common_r_code/",c("to_latex.R","plot_tree_matrix.R","utility_functions.R"))) {
    source(a)
}
@ 

<<load.data,echo=FALSE,results="hide",cache.extra=tools::md5sum(paste0("../data/",c("combined_fpkm","gene_trees","housekeeping_genes_superset","placenta_core_transcriptome","placenta_go_results_bp","palcenta_go_results_mf","hugo_nomenclature","gene_annotation")))>>=
for (a in paste0("../data/",c("combined_fpkm","gene_trees",
                              "housekeeping_genes_superset",
                              "placenta_core_transcriptome",
                              "placenta_go_results_bp",
                              "placenta_go_results_mf",
                              "hugo_nomenclature",
                              "gene_annotation"
                              ))) {
    load(a)
}
setkey(genes.to.tree,"gene_id")
star.logs[,value:=gsub("%$","",value)]
star.logs.wide <- 
    data.table(dcast(star.logs,species+file~field))
setnames(star.logs.wide,
         "Average input read length",
         "input.read.length")
setnames(star.logs.wide,
         "Number of input reads",
         "input.reads")
setnames(star.logs.wide,
         "Uniquely mapped reads number",
         "unique.map.reads")
setkey(hugo.nomenclature,"Approved Symbol")
@ 

<<load.go.results,echo=FALSE,results="hide",cache.extra=tools::md5sum("../go_annotation/go_tissue_analysis")>>=
load("../go_annotation/go_tissue_analysis")

@ 

\section{Species}

\subsection{Sequence Source}

\DLA{Add in SRR and GSE numbers for everything here once I have them.}

\begin{table}
  \begin{tabular}{c c p{3in}}
    \toprule
    Species                       & Collector & Experiment Number & Sample Number(s) \\
    \midrule
    \textit{Ateles fusciceps}     &  & & \\
    \textit{Bos taurus}           & Bob Kreft & & \\
    \textit{Canis familiaris}     &  & & \\
    \textit{Dasypus novemcinctus} &  & & \\
    \textit{Equus caballus}       & \citet{Coleman.ea2010:Structuralannotationofequine} & SRR850299,  SRR850300 \\
    \textit{Loxodonta africana}   & Kurt Benirschke & & \\
    \textit{Homo sapiens}         & Karina Richani, \citet{Hughes.ea2015:Evaluatingintra-andinter-individual} & GSE66622 
                                             & SRR1850945,
                                               SRR1850946, 
                                               SRR1850951, 
                                               SRR1850952, 
                                               SRR1850953, 
                                               SRR1850954, 
                                               SRR1850957, 
                                               SRR1850958, 
                                               SRR1850965, 
                                               SRR1850966, 
                                               SRR1850975, 
                                               SRR1850976, 
                                               SRR1850983, 
                                               SRR1850984, 
                                               SRR1850987, 
                                               SRR1850988, 
                                               SRR1850991, 
                                               SRR1850992, 
                                               SRR1850993, 
                                               SRR1850994, 
                                               SRR1850995, 
                                               SRR1850996, 
                                               SRR1851003, 
                                               SRR1851004, 
                                               SRR1851011, 
                                               SRR1851012 \\
    \textit{Monodelphis domestica} & K. Smith & & \\
    \textit{Mus musculus}         & J. Gonzalez, \cite{Wang.ea2011:surveyfornovelimprinted} & GSE29690 
                                     & SRR243594,
                                       SRR243595,
                                       SRR243596,
                                       SRR243597,
                                       SRR243598,
                                       SRR243599,
                                       SRR243600,
                                       SRR243601,
                                       SRR243602,
                                       SRR243586,
                                       SRR243587,
                                       SRR243588,
                                       SRR243589,
                                       SRR243590,
                                       SRR243591,
                                       SRR243592,
                                       SRR243593\\
    \textit{Nannospalax galili}   &  & GSE68763 & SRR2016467 \\
    \textit{Ovis aries}           & \cite{Archibald.ea2010:sheepgenomereferencesequence} & ERR489144, ERR489145\\
    \textit{Pan paniscus}         &  & \\
    \textit{Spalax carmeli}       &  & GSE68763 & SRR2016467 \\
    \textit{Sus scrofa}           & \cite{Ahanda.ea2012:Predictionofaltered3-} &  SRP014902 & SRR543893 \\
    \bottomrule
  \end{tabular}
  \caption{Species, experiment and species numbers used in the analyses presented in this paper}
  \label{tab:srr_numbers}
\end{table}

\subsection{Species Alignment}

\begin{table}
  \begin{tabular}{c c c c}
    \toprule
    Species                       & Common Name        & Reference & Read Length \\
    \midrule
    \textit{Ateles fusciceps}     & Spider monkey      & \textit{Homo sapiens} & 100 \\
    \textit{Bos taurus}           & Cow                & UMD3.1     & 75 \\
    \textit{Canis familiaris}     & Dog                & CanFam3.1  & 150 \\
    \textit{Dasypus novemcinctus} & 9-Banded Armadillo & DASNOV3    & 100 \\
    \textit{Equus caballus}       & Horse              & EquCab2    & 100 \\ 
    \textit{Loxodonta africana}   & Elephant           & loxAfr3    & 75 \\
    \textit{Homo sapiens}         & Human              & GRCh38     & 75, 150 \\
    \textit{Monodelphis domestica} & Opossum           & BROAD05    & 75 \\
    \textit{Mus musculus}         & Mouse              & GRCm38     & 35,44,150 \\
    \textit{Nannospalax galili}   & Blind mole rat     & \textit{Mus musculus} & 100 \\
    \textit{Ovis aries}           & Sheep              & Oar\_v3.1   & 151 \\
    \textit{Pan paniscus}         & Bonobo             & CHIMP2.1.4 & 150 \\
    \textit{Spalax carmeli}       & Blind mole rat     & \textit{Mus musculus} & 100 \\
    \textit{Sus scrofa}           & Duroc (pig)        & Sscrofa10.2 & 51 \\
    \bottomrule
  \end{tabular}
  \caption{Reference genomes for aligned species}
  \label{tab:ref_genome_aligned_species}
\end{table}

\section{Transcriptome statistics}



\begin{figure}
<<transcriptome.mapping.figure,fig.width=8,fig.height=8,echo=FALSE>>=
star.logs.unmapped <- 
    star.logs[grepl("%",field) & 
                  grepl("unmapped",field) & 
                      !grepl("too many mismatches",field),]
star.logs.unmapped <- 
    star.logs.unmapped[,field:=capfirst(gsub("\\% of reads unmapped: ","",field))]
pushViewport(viewport(layout=grid.layout(ncol=1,nrow=2,heights=c(2,1))))
print(ggplot(star.logs.unmapped,
             aes(x=capfirst(species),y=as.numeric(gsub("%","",value))))
      + geom_boxplot()
      + facet_grid(field~.,scales="free")
      + ylab("Percent of Reads")
      + xlab(NULL)
      + theme_bw()
      + theme(legend.position="top",
              plot.margin=unit(c(0,0,0,1.1),"strwidth",data=list("","","","e8")),
              axis.title.x=element_blank(),
              axis.text.x = element_blank()),#element_text(angle = 30, hjust = 1, vjust=1,face="italic")),
      vp=viewport(layout.pos.col=1,layout.pos.row=1)
      )
pushViewport(viewport(layout.pos.col=1,layout.pos.row=1))
grid.text("A",
          y=unit(1,"npc")-unit(0.1,"cm"),
          x=unit(0,"npc")+unit(0.1,"cm"),
          gp=gpar(fontsize=24),
          just=c("left","top"))
popViewport()          
print(ggplot(star.logs[grepl("input reads$",field),],
             aes(x=capfirst(species),y=as.numeric(value)))
      + geom_boxplot()
      + scale_y_log10()
      + ylab("Reads")
      + xlab("Species")
      + theme_bw()
      + theme(legend.position="top",
              plot.margin=unit(c(0,1.6,0,0),"strheight",data=list("","TO","","")),
              axis.text.x = element_text(angle = 30, hjust = 1, vjust=1,face="italic")),
      vp=viewport(layout.pos.col=1,layout.pos.row=2))
pushViewport(viewport(layout.pos.col=1,layout.pos.row=2))
grid.text("B",
          y=unit(1,"npc")-unit(0.1,"cm"),
          x=unit(0,"npc")+unit(0.1,"cm"),
          gp=gpar(fontsize=24),
          just=c("left","top"))
popViewport(2)
@ 
\caption{RNAseq read and alignment statistics for all species. A)
  Boxplot of percentages of unmapped reads in each species; 0\% of
  reads in all species were unmapped because of mismatches. B) Boxplot
  of number of RNAseq reads in each species.}
\label{fig:transcriptome_statistics}
\end{figure}


\begin{figure}
<<reads.vs.read.length,fig.width=8,fig.height=8,echo=FALSE>>=
print(ggplot(star.logs.wide,
             aes(x=as.numeric(input.read.length),
                 y=as.numeric(input.reads),
                 color=capfirst(species),
                 shape=capfirst(species)
                 ))
      + scale_shape_manual(values=1:nlevels(as.factor(star.logs.wide[,species])),
                           guide=guide_legend(title="Species")
                           )
      + scale_color_discrete(guide=guide_legend(title="Species")
                             )
      + geom_point(size=5)
      + xlab("Trimmed Read Length")
      + ylab("Read number")
      + scale_y_log10()
      + theme(legend.text=element_text(face="italic")
              )
      )
@ 
\caption{RNA Read number as a function of trimmed read length}
\label{fig:transcriptome_reads_vs_read_length}
\end{figure}


\section{Gene Ontology Analyses}

\begin{table}
  \footnotesize
<<go.analysis.all.tissue,echo=FALSE,results="asis">>=
go.analysis.wide <- 
    data.table(dcast(data=go.results,
                     formula=GO.ID+type+Term+Ontology~species,
                     value.var="FDR"))
go.analysis.wide <- 
    go.analysis.wide[,`n(FDR ≤ 0.05)`:=apply(go.analysis.wide[,-(1:4),with=FALSE],1,
                                         function(x){sum(as.numeric(as.character(x)) <= 0.05)})]
go.analysis.wide <- 
    go.analysis.wide[,`max(FDR ≤ 0.05)`:=apply(go.analysis.wide[,-(1:4),with=FALSE],1,
                                           function(x){if (!any(is.finite(x) & x<=0.05)) {
                                                           return(NA)
                                                       } else {
                                                           max(x[is.finite(x) & x<=0.05],na.rm=TRUE)
                                                       }})]
go.analysis.wide.table <- 
    go.analysis.wide[!is.na(`n(FDR ≤ 0.05)`) & `n(FDR ≤ 0.05)` >= 12 &
                         grepl("Top 1% No Housekeeping",type),
                     list(GO.ID,Term,Ontology,`n(FDR ≤ 0.05)`,`max(FDR ≤ 0.05)`)]
go.analysis.wide.table <- 
    go.analysis.wide.table[order(Ontology,-`n(FDR ≤ 0.05)`,`max(FDR ≤ 0.05)`)]
setnames(go.analysis.wide.table,"GO.ID","GO ID")

print(xtable(go.analysis.wide.table,
             display=c("d","s","s","s","d","e"),
             align=  c("c","c","c","c","c","c")
             ),
      math.style.exponents=TRUE,
      include.rownames=FALSE,
      booktabs=TRUE,
      floating=FALSE
      )

@ 
\caption{GO terms with significant enrichment ($\mathrm{FDR}\le0.05$)
  in 12 or more of the species analyzed from genes in the top 1\% of
  expression. $n(\mathrm{FDR}\le0.05)$ is the number of species with
  significant enrichment ($\mathrm{FDR}\le 0.05$) in the GO term
  listed. $max(\mathrm{FDR}\le0.05)$ is the maximum FDR value in all
  species with an FDR of less than $0.05$.}
\label{tab:go_terms_all_species}
\end{table}


\subsection{Top 100 Transcripts Enrichment}

<<go.analysis.all.species,echo=FALSE,results="asis">>=
for (this.species in names(species.names)) {
    cat(paste0("\\subsubsection{Signficantly enriched ($\\mathrm{FDR}\\le0.05$) GO terms in ",
                           " the top 100 \\textit{",
                           capfirst(this.species),"} expressed transcripts}\n\n"))
    this.caption <- paste0("Signficantly enriched ($\\mathrm{FDR}\\le0.05$) GO terms in ",
                           " the top 100 \\textit{",capfirst(this.species),"} expressed transcripts")
    this.label <- paste0("tab:go_terms_enriched_",gsub(" ","_",this.species))
    print(xtable(go.results[grepl("100 No Housekeeping",type) & FDR <= 0.05 & species == this.species,
                            list(GO.ID,Term,FDR,Ontology)
                            ][order(FDR)],
                 caption=this.caption,
                 label=this.label,
                 display=c("s","s","s","e","s")
                 ),
          size="\\small",
          booktabs=TRUE,
          tabular.environment="longtable",
          caption.placement="top",
          floating=FALSE)
}
@ 

\subsection{Top 1\% Transcripts Enrichment}

<<go.analysis.top.5p.all.species,echo=FALSE,results="asis">>=
for (this.species in names(species.names)) {
    cat(paste0("\\subsubsection{Signficantly enriched ($\\mathrm{FDR}\\le0.05$) GO terms in ",
                           " the top 1\\% of \\textit{",
                           capfirst(this.species),"} expressed transcripts}\n\n"))
    this.caption <- paste0("Signficantly enriched ($\\mathrm{FDR}\\le0.05$) GO terms in ",
                           " the top 1\\% of \\textit{",
                           capfirst(this.species),"} expressed transcripts.")
    print(xtable(go.results[grepl("Top 1% No Housekeeping",type) & FDR <= 0.05 & species == this.species,
                            list(GO.ID,Term,FDR,Ontology)
                            ][order(FDR)],
                 caption=this.caption,
                 label=paste0("tab:go_terms_enriched_top_one_",gsub(" ","_",this.species)),
                 display=c("s","s","s","e","s")
                 ),
          size="\\small",
          booktabs=TRUE, 
          tabular.environment="longtable",
          caption.placement="top",
          floating=FALSE)
}
@ 

<<genes.expression,echo=FALSE,results="hide">>=
genes.expression.table <- function(species.name,min.fpkm=50) {
    alignment.species <- species.name
    if (alignment.species=="pan paniscus") {
        alignment.species <- "pan troglodytes"
    } else if (alignment.species=="ateles fusciceps") {
        alignment.species <- "homo sapiens"
    } else if (grepl("spalax",alignment.species)) {
        alignment.species <- "mus musculus"
    }

    temp <- 
        combined.fpkm[species==species.name
                      ][order(-mean_fpkm)
                        ][,list(name_or_id,
                                human_alignment_symbol,
                                human_name,mean_fpkm)]
    temp <- temp[!duplicated(name_or_id)][mean_fpkm>=min.fpkm]
    temp[,used_alignment:=FALSE]
    temp[is.na(human_name) & !is.na(human_alignment_symbol),
         used_alignment:=TRUE]
    temp[is.na(human_name),human_name:=human_alignment_symbol]
    setkey(temp,"human_name")
    temp <- 
        hugo.nomenclature[temp][order(-mean_fpkm),
                                list(name_or_id,used_alignment,`Approved Symbol`,`Approved Name`,mean_fpkm)]
    temp[!is.na(name_or_id),
         name_or_id:=paste0("\\href{http://www.ensembl.org/",
                            capfirst(gsub(" ","_",alignment.species)),
                            "/Gene/Summary?g=",name_or_id,"}{",
                            xtable.sanitize.text(name_or_id),"}")]
    temp[!is.na(`Approved Name`),
         `Approved Name`:=paste0("\\href{http://www.ensembl.org/Homo_sapiens/Gene/Summary?g=",`Approved Symbol`,"}{",xtable.sanitize.text(`Approved Name`),"}")]
    temp[!is.na(`Approved Symbol`),
         `Approved Symbol`:=paste0("\\href{http://www.ensembl.org/Homo_sapiens/Gene/Summary?g=",`Approved Symbol`,"}{",xtable.sanitize.text(`Approved Symbol`),"}")]
    temp[used_alignment==TRUE,`Approved Symbol`:=paste0("(",`Approved Symbol`,")")]
    temp[,used_alignment:=NULL]
    setnames(temp,c("Gene symbol","Human Symbol","Human Name","FPKM"))
    if (species.name=="homo sapiens" || species.name=="ateles fusciceps") {
        temp[,`Human Symbol`:=NULL]
    }
    return(temp)
}
@ 

\section{Gene Expression}

\subsection{Gene Evolution Trees}

\begin{figure}
<<fstl1.evolution.tree,echo=FALSE,fig.width=12,fig.height=12,warning=FALSE,message=FALSE>>=

tree.id <-
    genes.to.tree[combined.fpkm[gene_short_name=="FSTL1" & species=="homo sapiens",gene_id],tree]
gene.tree.table <- trees[[tree.id]]$table
gene.tree.expression <- combined.fpkm[gene_id %in% gene.tree.table[,gene_id]]
gene.tree <- read.tree(text=trees[[tree.id]]$tree.dat)
plot.tree.matrix(read.tree(text=trees[[tree.id]]$tree.dat),gene.tree.table,gene.tree.expression,min.fpkm=5)
@ 
\caption{\genetreeexplanation{follistatin-like 1 \textit{FSTL1}}}
\label{fig:fstl1_evolution_tree}
\end{figure}

\begin{figure}
<<serpine.evolution.tree,echo=FALSE,fig.width=12,fig.height=12,warning=FALSE,message=FALSE>>=

tree.id <-
    genes.to.tree[combined.fpkm[gene_short_name=="SERPINE2" & species=="homo sapiens",gene_id],tree]
gene.tree.table <- trees[[tree.id]]$table
gene.tree.expression <- combined.fpkm[gene_id %in% gene.tree.table[,gene_id]]
gene.tree <- read.tree(text=trees[[tree.id]]$tree.dat)
plot.tree.matrix(read.tree(text=trees[[tree.id]]$tree.dat),gene.tree.table,gene.tree.expression,min.fpkm=1)
@ 
\caption{\genetreeexplanation{\textit{SERPINE2}}}
\label{fig:serpine_evolution_tree}
\end{figure}

\begin{figure}
<<sslp1.evolution.tree,echo=FALSE,fig.width=12,fig.height=12,warning=FALSE,message=FALSE>>=

tree.id <-
    genes.to.tree[combined.fpkm[gene_id=="ENSCAFG00000010638" & species=="canis familiaris",gene_id],tree]
gene.tree.table <- trees[[tree.id]]$table
gene.tree.expression <- combined.fpkm[gene_id %in% gene.tree.table[,gene_id]]
gene.tree <- read.tree(text=trees[[tree.id]]$tree.dat)
plot.tree.matrix(read.tree(text=trees[[tree.id]]$tree.dat),gene.tree.table,gene.tree.expression)
@ 
\caption{\genetreeexplanation{\textit{SSLP1}}}
\label{fig:sslp1_evolution_tree}
\end{figure}

\begin{figure}
<<estrogen.evolution.tree,echo=FALSE,fig.width=12,fig.height=12,warning=FALSE,message=FALSE>>=

tree.id <-
    genes.to.tree[combined.fpkm[gene_short_name=="CYP19A1" & species=="homo sapiens",gene_id],tree]
gene.tree.table <- trees[[tree.id]]$table
gene.tree.expression <- combined.fpkm[gene_id %in% gene.tree.table[,gene_id]]
gene.tree <- read.tree(text=trees[[tree.id]]$tree.dat)
plot.tree.matrix(read.tree(text=trees[[tree.id]]$tree.dat),gene.tree.table,gene.tree.expression)
@ 
\caption{\genetreeexplanation{estrogen synthesis genes}}
\label{fig:estrogen_evolution_tree}
\end{figure}

\begin{figure}
<<progesterone.evolution.tree,echo=FALSE,fig.width=12,fig.height=12,warning=FALSE,message=FALSE>>=

tree.id <-
    genes.to.tree[combined.fpkm[gene_short_name=="HSD3B1" & species=="homo sapiens",gene_id],tree]
gene.tree.table <- trees[[tree.id]]$table
gene.tree.expression <- combined.fpkm[gene_id %in% gene.tree.table[,gene_id]]
gene.tree <- read.tree(text=trees[[tree.id]]$tree.dat)
plot.tree.matrix(read.tree(text=trees[[tree.id]]$tree.dat),gene.tree.table,gene.tree.expression,
                 subtree=c("ENSMODG00000005385","ENSLAFG00000027896")
                 )
@ 
\caption{\genetreeexplanation{progesterone synthesis genes}}
\label{fig:progesterone_evolution_tree}
\end{figure}

\begin{figure}
<<chorionic.somatomammotropin.evolution.tree,echo=FALSE,fig.width=12,fig.height=12,warning=FALSE,message=FALSE>>=

tree.id <-
    genes.to.tree[combined.fpkm[gene_short_name=="CSH1" & species=="homo sapiens",gene_id],tree]
gene.tree.table <- trees[[tree.id]]$table
gene.tree.expression <- combined.fpkm[gene_id %in% gene.tree.table[,gene_id]]
gene.tree <- read.tree(text=trees[[tree.id]]$tree.dat)
plot.tree.matrix(read.tree(text=trees[[tree.id]]$tree.dat),gene.tree.table,gene.tree.expression)
@ 
\caption{\genetreeexplanation{chorionic somatomammotropin}}
\label{fig:csh1_evolution_tree}
\end{figure}

\begin{figure}
<<prolactin.evolution.tree,echo=FALSE,fig.width=12,fig.height=12,warning=FALSE,message=FALSE>>=

tree.id <-
    genes.to.tree[combined.fpkm[gene_short_name=="PRL" & species=="homo sapiens",gene_id],tree]
gene.tree.table <- trees[[tree.id]]$table
gene.tree.expression <- combined.fpkm[gene_id %in% gene.tree.table[,gene_id]]
gene.tree <- read.tree(text=trees[[tree.id]]$tree.dat)
plot.tree.matrix(read.tree(text=trees[[tree.id]]$tree.dat),gene.tree.table,gene.tree.expression)
@ 
\caption{\genetreeexplanation{prolactin}}
\label{fig:prolactin_evolution_tree}
\end{figure}

\begin{figure}
<<rln2.evolution.tree,echo=FALSE,fig.width=12,fig.height=12,warning=FALSE,message=FALSE>>=

tree.id <-
    genes.to.tree[combined.fpkm[gene_short_name=="RLN2" & species=="homo sapiens",gene_id],tree]
gene.tree.table <- trees[[tree.id]]$table
gene.tree.expression <- combined.fpkm[gene_id %in% gene.tree.table[,gene_id]]
gene.tree <- read.tree(text=trees[[tree.id]]$tree.dat)
plot.tree.matrix(read.tree(text=trees[[tree.id]]$tree.dat),gene.tree.table,gene.tree.expression)
@ 
\caption{\genetreeexplanation{\textit{RLN2}}}
\label{fig:rln2_evolution_tree}
\end{figure}

\begin{figure}
<<ovos2.evolution.tree,echo=FALSE,fig.width=12,fig.height=12,warning=FALSE,message=FALSE>>=

tree.id <-
    genes.to.tree["ENSDNOG00000033320",tree]
gene.tree.table <- trees[[tree.id]]$table
gene.tree.expression <- combined.fpkm[gene_id %in% gene.tree.table[,gene_id]]
gene.tree <- read.tree(text=trees[[tree.id]]$tree.dat)
plot.tree.matrix(read.tree(text=trees[[tree.id]]$tree.dat),gene.tree.table,gene.tree.expression,
                 subtree=c("ENSDNOG00000033320","ENSMODG00000017988"))
@ 
\caption{\genetreeexplanation{\cow \textit{OVOS2}}}
\label{fig:ovos2_evolution_tree}
\end{figure}

\begin{figure}
<<prolactin.receptor.evolution.tree,echo=FALSE,fig.width=12,fig.height=12,warning=FALSE,message=FALSE>>=

tree.id <-
    genes.to.tree[combined.fpkm[gene_short_name=="PRLR" & species=="homo sapiens",gene_id],tree]
gene.tree.table <- trees[[tree.id]]$table
gene.tree.expression <- combined.fpkm[gene_id %in% gene.tree.table[,gene_id]]
gene.tree <- read.tree(text=trees[[tree.id]]$tree.dat)
plot.tree.matrix(read.tree(text=trees[[tree.id]]$tree.dat),gene.tree.table,gene.tree.expression,
                 subtree=c("ENSG00000113494","ENSMODG00000020376"))
@ 
\caption{\genetreeexplanation{prolactin receptor, \textit{PRLR}}}
\label{fig:prolactin_receptor_evolution_tree}
\end{figure}


\begin{figure}
<<hoxa11.receptor.evolution.tree,echo=FALSE,fig.width=12,fig.height=12,warning=FALSE,message=FALSE>>=

tree.id <-
    genes.to.tree[combined.fpkm[gene_short_name=="HOXA11" & species=="homo sapiens",gene_id],tree]
gene.tree.table <- trees[[tree.id]]$table
gene.tree.expression <- combined.fpkm[gene_id %in% gene.tree.table[,gene_id]]
gene.tree <- read.tree(text=trees[[tree.id]]$tree.dat)
plot.tree.matrix(read.tree(text=trees[[tree.id]]$tree.dat),gene.tree.table,gene.tree.expression,
                 subtree=c("ENSBTAG00000014738","ENSECAG00000013229")
                 )
@ 
\caption{\genetreeexplanation{\textit{HOXA11}}}
\label{fig:hoxa11_evolution_tree}
\end{figure}

 
\begin{figure}
<<hbe1.evolution.tree,echo=FALSE,fig.width=12,fig.height=12,warning=FALSE,message=FALSE>>=

tree.id <-
    genes.to.tree[combined.fpkm[gene_short_name=="HBE1" & species=="homo sapiens",gene_id],tree]
gene.tree.table <- trees[[tree.id]]$table
gene.tree.expression <- combined.fpkm[gene_id %in% gene.tree.table[,gene_id]]
gene.tree <- read.tree(text=trees[[tree.id]]$tree.dat)
plot.tree.matrix(read.tree(text=trees[[tree.id]]$tree.dat),gene.tree.table,gene.tree.expression)
@ 
\caption{Expression of \textit{HBE1} and related genes}
\label{fig:hbe1_evolution_tree}
\end{figure}

\subsection{Clade specific Expression}

\begin{table}
  \footnotesize
<<rodentia.lineage.specific,echo=FALSE,results="asis">>=
rod.all.lin.spec <- placenta.lineage.specific.table("rod.all")
setkey(rod.all.lin.spec,"Human Symbol")
@ 
\caption{Genes whose expression changed significantly in the Rodentia
  order as compared to all other analyzed species.}
\label{tab:rodentia_lineage_specific}
\end{table}

\begin{table}
  \footnotesize
<<lau.lineage.specific,echo=FALSE,results="asis">>=
lau.all.lin.spec <- placenta.lineage.specific.table("lau.all")
setkey(lau.all.lin.spec,"Human Symbol")
@ 
\caption{Genes whose expression changed in the Laurasiatheria
  superorder as compared to all other analyzed species.}
\label{tab:laurasiatheria_lineage_specific}
\end{table}

\begin{table}
  \footnotesize
<<eua.lau.lineage.specific,echo=FALSE,results="asis">>=
eua.lau.lin.spec <- placenta.lineage.specific.table("eua.lau")
setkey(eua.lau.lin.spec,"Human Symbol")
@ 
\caption{Genes whose expression changed between the
  Euasiatherian and Laurasiatherian superorder.}
\label{tab:laurasiatheria_eurasiatherian_lineage_specific}
\end{table}


\subsection{Species Expression}

<<top.expressed.genes,echo=FALSE,results="asis">>=
for (this.species in names(species.names)) {
    this.caption <- paste0("\\textit{",
                           capfirst(this.species),"} Placental Gene Expression of all genes with FPKM $\\ge 50$")
    cat(paste0("\\subsection{",this.caption,"}\n\n"))
    print(xtable(genes.expression.table(species.name=this.species),
                 caption=this.caption,
                 label=paste0("tab:all_",gsub(" ","_",this.species),"_placenta")
                 ),
          booktabs=TRUE,
          tabular.environment="longtable",
          floating=FALSE,
          caption.placement="top",
          sanitize.text.function=genes.table.sanitize,
          size="\\tiny"
          )
}
@ 

\begin{figure}
<<nppb.evolution.tree,echo=FALSE,fig.width=12,fig.height=12,warning=FALSE,message=FALSE>>=

tree.id <-
    genes.to.tree[combined.fpkm[gene_short_name=="NPPB" & species=="homo sapiens",gene_id],tree]
gene.tree.table <- trees[[tree.id]]$table
gene.tree.expression <- combined.fpkm[gene_id %in% gene.tree.table[,gene_id]]
gene.tree <- read.tree(text=trees[[tree.id]]$tree.dat)
plot.tree.matrix(read.tree(text=trees[[tree.id]]$tree.dat),gene.tree.table,gene.tree.expression)
@ 
\caption{\genetreeexplanation{\textit{NPPB}}}
\label{fig:nppb_evolution_tree}
\end{figure}

\begin{figure}
<<nppc.evolution.tree,echo=FALSE,fig.width=12,fig.height=12,warning=FALSE,message=FALSE>>=

tree.id <-
    genes.to.tree[combined.fpkm[gene_short_name=="NPPC" & species=="homo sapiens",gene_id],tree]
gene.tree.table <- trees[[tree.id]]$table
gene.tree.expression <- combined.fpkm[gene_id %in% gene.tree.table[,gene_id]]
gene.tree <- read.tree(text=trees[[tree.id]]$tree.dat)
plot.tree.matrix(read.tree(text=trees[[tree.id]]$tree.dat),gene.tree.table,gene.tree.expression)
@ 
\caption{\genetreeexplanation{\textit{NPPC}}}
\label{fig:nppc_evolution_tree}
\end{figure}


\begin{figure}
<<galectins.evolution.tree,echo=FALSE,fig.width=12,fig.height=12,warning=FALSE,message=FALSE>>=

tree.id <-
    genes.to.tree[combined.fpkm[gene_short_name=="LGALS1" & species=="homo sapiens",gene_id],tree]
gene.tree.table <- trees[[tree.id]]$table
gene.tree.expression <- combined.fpkm[gene_id %in% gene.tree.table[,gene_id]]
gene.tree <- read.tree(text=trees[[tree.id]]$tree.dat)
plot.tree.matrix(read.tree(text=trees[[tree.id]]$tree.dat),gene.tree.table,gene.tree.expression)
@ 
\caption{\genetreeexplanation{galectins}}
\label{fig:galectins_evolution_tree}
\end{figure}


\begin{figure}
<<lgals13.galectins.evolution.tree,echo=FALSE,fig.width=12,fig.height=12,warning=FALSE,message=FALSE>>=

tree.id <-
    genes.to.tree[combined.fpkm[gene_short_name=="LGALS13" & species=="homo sapiens",gene_id],tree]
gene.tree.table <- trees[[tree.id]]$table
gene.tree.expression <- combined.fpkm[gene_id %in% gene.tree.table[,gene_id]]
gene.tree <- read.tree(text=trees[[tree.id]]$tree.dat)
plot.tree.matrix(read.tree(text=trees[[tree.id]]$tree.dat),gene.tree.table,gene.tree.expression)
@ 
\caption{\genetreeexplanation{\textit{LGALS13}}}
\label{fig:galectins_lgals13_evolution_tree}
\end{figure}

\section{Core Placenta Transcriptome}

\begin{table}
\footnotesize
<<go.analysis,warning=FALSE,message=FALSE,echo=FALSE,results="asis">>=
placenta.go.results <- rbindlist(list(head(n=20,placenta_go_results_mf[classic.fdr < 0.05,]),
                             head(n=20,placenta_go_results_bp[classic.fdr < 0.05,])))
placenta.go.results <- placenta.go.results[,Annotated:=NULL]
setnames(placenta.go.results,"Significant","# Core")
setnames(placenta.go.results,"classic","p value")
setnames(placenta.go.results,"classic.fdr","FDR")
setnames(placenta.go.results,"GO.ID","GO ID")
setnames(placenta.go.results,"Term","GO Term")
print(xtable(placenta.go.results,
             digits=2,
             display=c("d","s","s","d","f","e","e","s")),
      include.rownames=FALSE,
      booktabs=TRUE,
      floating=FALSE
      )
@ 
\caption{\ac{GO} terms most significantly enriched
  (\Sexpr{placenta.go.results[,sum(Ontology=="MF")]} MF,
  \Sexpr{placenta.go.results[,sum(Ontology=="BP")]} BP)
  in the core placenta transcriptome as compared to all genes
  with 1:1 human orthologs in all species. “\# Core” is the
  number of genes in the core transcriptome for each \ac{GO} term;
  “Expected” is the number of genes expected in the core
  transcriptome for each term; “p-value” are the odds of this
  occurring by chance calculated by the Fisher exact test for
  each term; “FDR” is the false discovery rate as estimated by
  the Benjamini and Hochberg procedure \cite{Benjamini1995:bh_procedure}.}
\label{tab:core_placenta_go_terms}
\end{table}

\begin{figure}
<<core.placenta.transcriptome.distribution,fig.width=8,fig.height=8,out.height="0.8\\textheight",echo=FALSE,message=FALSE>>=
calculate.distribution.shape <- 
    function(min.vector) {
        min.vector <- sort(min.vector,decreasing=TRUE)
        temp <- data.table(FPKM=min.vector,num_genes=1:length(min.vector))
        temp <- temp[order(-num_genes)][!duplicated(FPKM)]
        return(temp)
    }

cptg.nh <- core.placenta.transcriptome.genes.shape[!(human_name %in% housekeeping.genes.superset[,gene_short_name])]

cpt.distribution.shape <- 
    lapply(c("min_expression",
             "second_min_expression",
             "third_min_expression",
             "fourth_min_expression",
             "fifth_min_expression"),
           function(x){
               calculate.distribution.shape(cptg.nh[,x,with=FALSE][[1]])[,`Lowest Expression`:=x]
           })
cpt.distribution.shape <- 
    rbindlist(cpt.distribution.shape)

cpt.distribution.shape <- 
    cpt.distribution.shape[,`Lowest Expression`:=
                               gsub("^min_expression$","First",
                                    `Lowest Expression`)]
cpt.distribution.shape <- 
    cpt.distribution.shape[,`Lowest Expression`:=
                               factor(gsub("^(.)(.+)_min_expression$","\\U\\1\\L\\2",
                                           `Lowest Expression`,perl=TRUE),
                                      levels=c("First","Second","Third","Fourth","Fifth"))
                           ]
setnames(cpt.distribution.shape,
         "FPKM",
         "Expression (FPKM)")
         

print(ggplot(cpt.distribution.shape[`Expression (FPKM)` > 1,],
             aes(x=`Expression (FPKM)`,color=`Lowest Expression`,y=num_genes))
      + geom_line(lwd=1)
      + xlab(expression(FPKM))
      + ylab("Genes with N-th Lowest Expression")
      + scale_y_log10()
      + scale_x_log10()
      + geom_vline(xintercept=10)
      + geom_segment(mapping=aes(y=V1,
                         yend=V1,
                         xend=10,
                         x=100,
                         color=`Lowest Expression`),
                     data=cpt.distribution.shape[`Expression (FPKM)` > 10,
                                              max(num_genes),`Lowest Expression`],
                     arrow=arrow(length=unit(0.5,"cm"))
                   )
      + geom_text(mapping=aes(x=100,
                      y=V1,
                      label=paste0(V1," genes"),override=TRUE),
                  data=cpt.distribution.shape[`Expression (FPKM)` > 10,
                      max(num_genes),`Lowest Expression`],
                  show_guide=FALSE,
                  hjust=0
                  )
      + guides(color=guide_legend(title="Lowest\nExpression"))
      )
@ 
\caption{Distribution shape of placenta core genes. The vertical axis shows
  the number of orthologous gene sets included in the core placenta
  transcriptome if the least expression cutoff (or second-least, etc.)
  was set to the horizontal axis. For example, if the cutoff is set to 10
  (vertical black line), 37 genes would be included if the FPKM of all
  species must be at least 10, 134 genes would be included if all but
  one species must have FPKM of at least 10. 655 genes would be
  included if all but five species must have FPKM of at least 10.}
\label{fig:core_placenta_transcriptome_shape}
\end{figure}


 
\printbibliography



\end{document}
